<!DOCTYPE html>
<html ng-app="ddiApp" id="ng-app" class="js flexbox canvas canvastext webgl no-touch geolocation postmessage no-websqldatabase
      indexeddb hashchange history draganddrop websockets rgba hsla multiplebgs backgroundsize borderimage
      borderradius boxshadow textshadow opacity cssanimations csscolumns cssgradients no-cssreflections
      csstransforms csstransforms3d csstransitions fontface generatedcontent video audio localstorage
      sessionstorage webworkers applicationcache svg inlinesvg smil svgclippaths ng-scope"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
    <meta charset="utf-8">
    <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">

    <meta content="width=device-width" name="viewport">
    <meta content="This SuperSeries is composed of the following subset Series: GSE37194: Gene expression profiling during interference with PPAR gamma signaling in thoracic aorta GSE37195: Gene expression profiling using exon arrays during interference with PPAR gamma signaling in thoracic aorta Refer to individual Series" name="description">
    <title>E-GEOD-37196 - Interference of PPAR gamma signaling in thoracic aorta - OmicsDI</title>

    <!-- base href="https://www.omicsdi.org/" -->

    <!-- favico -->
    <link rel="apple-touch-icon" sizes="57x57" href="https://www.omicsdi.org/img/favico/apple-icon-57x57.png">
    <link rel="apple-touch-icon" sizes="60x60" href="https://www.omicsdi.org/img/favico/apple-icon-60x60.png">
    <link rel="apple-touch-icon" sizes="72x72" href="https://www.omicsdi.org/img/favico/apple-icon-72x72.png">
    <link rel="apple-touch-icon" sizes="76x76" href="https://www.omicsdi.org/img/favico/apple-icon-76x76.png">
    <link rel="apple-touch-icon" sizes="114x114" href="https://www.omicsdi.org/img/favico/apple-icon-114x114.png">
    <link rel="apple-touch-icon" sizes="120x120" href="https://www.omicsdi.org/img/favico/apple-icon-120x120.png">
    <link rel="apple-touch-icon" sizes="144x144" href="https://www.omicsdi.org/img/favico/apple-icon-144x144.png">
    <link rel="apple-touch-icon" sizes="152x152" href="https://www.omicsdi.org/img/favico/apple-icon-152x152.png">
    <link rel="apple-touch-icon" sizes="180x180" href="https://www.omicsdi.org/img/favico/apple-icon-180x180.png">
    <link rel="icon" type="image/png" sizes="192x192" href="https://www.omicsdi.org/img/favico/android-icon-192x192.png">
    <link rel="icon" type="image/png" sizes="32x32" href="https://www.omicsdi.org/img/favico/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="96x96" href="https://www.omicsdi.org/img/favico/favicon-96x96.png">
    <link rel="icon" type="image/png" sizes="16x16" href="https://www.omicsdi.org/img/favico/favicon-16x16.png">
    <!--<link rel="manifest" href="./img/favico/manifest.json">-->
    <meta name="msapplication-TileColor" content="#ffffff">
    <meta name="msapplication-TileImage" content="./img/favico/ms-icon-144x144.png">
    <meta name="theme-color" content="#ffffff">

    <script>
        if (!window.location.href.includes("rosette")) {
            var link = document.createElement('link');
            link.href = './assets/css/bootstrap.min.css';
            link.rel = 'stylesheet';
            link.type = 'text/css';
            document.head.appendChild(link);

            function getParameterByName(name, url) {
                if (!url) url = window.location.href;
                name = name.replace(/[\[\]]/g, "\\$&");
                var regex = new RegExp("[?&]" + name + "(=([^&#]*)|&|#|$)"),
                    results = regex.exec(url);
                if (!results) return null;
                if (!results[2]) return '';
                return decodeURIComponent(results[2].replace(/\+/g, " "));
            }

            var authToken = getParameterByName('AUTH_TOKEN');
            if (authToken && typeof localStorage !== 'undefined') {
                localStorage.setItem('id_token', authToken);
                window.location.href = window.location.href.replace('AUTH_TOKEN=' + authToken, '');
            }

            var galaxyURL = getParameterByName('GALAXY_URL');
            if (galaxyURL && typeof sessionStorage !== 'undefined') {
                sessionStorage.setItem('galaxy_instance', galaxyURL);
            }
        }
    </script><link href="page_with_metadata_files/bootstrap.min.css" rel="stylesheet" type="text/css">
<link rel="stylesheet" href="page_with_metadata_files/styles.8ce41a2a5ce095e11207.css"><script charset="utf-8" src="page_with_metadata_files/5.160b7eac4c93378a75c1.js"></script><script charset="utf-8" src="page_with_metadata_files/common.b80e95026de13a118f43.js"></script><script charset="utf-8" src="page_with_metadata_files/9.980a99276ae10a201127.js"></script><style>.elixir-ribbon[_ngcontent-serverApp-c0] {
                padding: 1rem 0;
                background-color: transparent;
            }

            .elixir-ribbon[_ngcontent-serverApp-c0] {
                font-family: Helvetica, Arial, FreeSans, 'Liberation Sans', sans-serif;
                clear: both;
            }

            .elixir-ribbon[_ngcontent-serverApp-c0]   a[_ngcontent-serverApp-c0], .elixir-ribbon[_ngcontent-serverApp-c0]   a[_ngcontent-serverApp-c0]:active, .elixir-ribbon[_ngcontent-serverApp-c0]   a[_ngcontent-serverApp-c0]:visited, .elixir-ribbon[_ngcontent-serverApp-c0]   a[_ngcontent-serverApp-c0]:hover {
                border-bottom: none;
            }

            .elixir-ribbon[_ngcontent-serverApp-c0]   .row[_ngcontent-serverApp-c0] {
                max-width: 1200px;
                margin: 0 auto 1rem;
            }

            .elixir-ribbon[_ngcontent-serverApp-c0]   .row[_ngcontent-serverApp-c0]::before, .elixir-ribbon[_ngcontent-serverApp-c0]   .row[_ngcontent-serverApp-c0]::after {
                display: table;
                content: ' ';
            }

            .elixir-ribbon[_ngcontent-serverApp-c0]   h5[_ngcontent-serverApp-c0] {
                font-size: 1.3rem;
                padding: 0;
                display: inline-block;
            }

            .elixir-ribbon[_ngcontent-serverApp-c0], .elixir-ribbon[_ngcontent-serverApp-c0]   h5[_ngcontent-serverApp-c0], .elixir-ribbon[_ngcontent-serverApp-c0]   a[_ngcontent-serverApp-c0], .elixir-ribbon[_ngcontent-serverApp-c0]   a[_ngcontent-serverApp-c0]:active, .elixir-ribbon[_ngcontent-serverApp-c0]   a[_ngcontent-serverApp-c0]:visited, .elixir-ribbon[_ngcontent-serverApp-c0]   a[_ngcontent-serverApp-c0]:hover {
                color: #222;
                text-decoration: none;
            }
            .elixir-ribbon[_ngcontent-serverApp-c0]   a[_ngcontent-serverApp-c0]:hover {
                opacity: .8;
            }
            .elixir-ribbon[_ngcontent-serverApp-c0]   .readmore[_ngcontent-serverApp-c0] {
                border-bottom: 1px dotted #222;
            }
            .elixir-ribbon[_ngcontent-serverApp-c0]   h5[_ngcontent-serverApp-c0] {
                margin: 0;
            }
            .elixir-ribbon[_ngcontent-serverApp-c0]   .elixir-logo-kite[_ngcontent-serverApp-c0] {
                background: 80% 58% url("https://ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.2/images/logos/assorted/elixir_kitemark-60px.png") no-repeat;
                position: relative;
                top: -5px;
                margin: 0 1rem -.5rem 0;
                height: 60px;
                width: 60px;
                display: inline-block;
                float: left;
                background-size: 60px;
            }</style><style>ul[_ngcontent-serverApp-c0]   li[_ngcontent-serverApp-c0]{font-size:10pt}.navbar-inverse[_ngcontent-serverApp-c0]   .navbar-nav[_ngcontent-serverApp-c0] > .open[_ngcontent-serverApp-c0] > a[_ngcontent-serverApp-c0], .navbar-inverse[_ngcontent-serverApp-c0]   .navbar-nav[_ngcontent-serverApp-c0] > .open[_ngcontent-serverApp-c0] > a[_ngcontent-serverApp-c0]:focus, .navbar-inverse[_ngcontent-serverApp-c0]   .navbar-nav[_ngcontent-serverApp-c0] > .open[_ngcontent-serverApp-c0] > a[_ngcontent-serverApp-c0]:hover{background-color:#2e6da4}.indent[_ngcontent-serverApp-c0]{margin-left:260px}</style><style>ng-progress{z-index:999999;pointer-events:none}ng-progress[fixed=true] .ng-progress-bar,ng-progress[fixed=true] .ng-spinner{position:fixed}ng-progress[fixed=true] .ng-spinner{top:15px}ng-progress[fixed=true][spinnerPosition=left] .ng-spinner{left:15px}ng-progress[fixed=true][spinnerPosition=right] .ng-spinner{right:15px}ng-progress[thick=true] .ng-spinner-icon{width:24px;height:24px;border-width:3px}ng-progress[thick=true] .ng-bar-placeholder{height:3px!important}ng-progress[dir='ltr+'] .ng-meteor,ng-progress[dir=ltr-] .ng-meteor{-webkit-transform:rotate(3deg);transform:rotate(3deg)}ng-progress[dir='ltr+'][thick=true] .ng-meteor,ng-progress[dir=ltr-][thick=true] .ng-meteor{-webkit-transform:rotate(4deg);transform:rotate(4deg)}ng-progress[dir='ltr+'] .ng-bar,ng-progress[dir='rtl+'] .ng-bar{margin-left:-100%}ng-progress[dir='ltr+'] .ng-meteor,ng-progress[dir='rtl+'] .ng-meteor{right:0}ng-progress[dir='ltr+'] .ng-meteor,ng-progress[dir=rtl-] .ng-meteor{top:-3px}ng-progress[dir='ltr+'][thick=true] .ng-meteor,ng-progress[dir=rtl-][thick=true] .ng-meteor{top:-4px}ng-progress[dir='rtl+'] .ng-meteor,ng-progress[dir=ltr-] .ng-meteor{bottom:-3px}ng-progress[dir='rtl+'][thick=true] .ng-meteor,ng-progress[dir=ltr-][thick=true] .ng-meteor{bottom:-4px}ng-progress[dir='rtl+'] .ng-bar-placeholder,ng-progress[dir=ltr-] .ng-bar-placeholder{-webkit-transform:rotate(180deg);transform:rotate(180deg)}ng-progress[dir='rtl+'] .ng-spinner-icon,ng-progress[dir=ltr-] .ng-spinner-icon{animation-direction:reverse}ng-progress[dir='rtl+'] .ng-meteor,ng-progress[dir=rtl-] .ng-meteor{-webkit-transform:rotate(-3deg);transform:rotate(-3deg)}ng-progress[dir='rtl+'][thick=true] .ng-meteor,ng-progress[dir=rtl-][thick=true] .ng-meteor{-webkit-transform:rotate(-4deg);transform:rotate(-4deg)}ng-progress[spinnerPosition=left] .ng-spinner{left:10px}ng-progress[spinnerPosition=right] .ng-spinner{right:10px}.ng-progress-bar{position:relative;z-index:999999;top:0;left:0;width:100%;zoom:1;opacity:0}.ng-progress-bar.-active{opacity:1;transition:none}.ng-bar-placeholder{position:absolute;height:2px;width:100%}.ng-bar{width:100%;height:100%;-webkit-transform:translate(-100%,0,0);transform:translate(-100%,0,0)}.ng-meteor{display:block;position:absolute;width:100px;height:100%;opacity:1}.ng-spinner{position:absolute;display:block;z-index:1031;top:10px}.ng-spinner-icon{width:18px;height:18px;box-sizing:border-box;-webkit-animation:250ms linear infinite spinner-animation;animation:250ms linear infinite spinner-animation;border:2px solid transparent;border-radius:50%}@-webkit-keyframes spinner-animation{0%{-webkit-transform:rotate(0);transform:rotate(0)}100%{-webkit-transform:rotate(360deg);transform:rotate(360deg)}}@keyframes spinner-animation{0%{-webkit-transform:rotate(0);transform:rotate(0)}100%{-webkit-transform:rotate(360deg);transform:rotate(360deg)}}</style><style>.simple-notification-wrapper {
    position: fixed;
    width: 300px;
    z-index: 1000;
}

.simple-notification-wrapper.left {
    left: 20px;
}

.simple-notification-wrapper.top {
    top: 20px;
}

.simple-notification-wrapper.right {
    right: 20px;
}

.simple-notification-wrapper.bottom {
    bottom: 20px;
}

.simple-notification-wrapper.center {
    left: 50%;
    transform: translateX(-50%);
}

.simple-notification-wrapper.middle {
    top: 50%;
    transform: translateY(-50%);
}

.simple-notification-wrapper.middle.center {
    transform: translate(-50%, -50%);
}

@media (max-width: 340px) {
    .simple-notification-wrapper {
        width: auto;
        left: 20px;
        right: 20px;
    }
}</style><style>div.mat-menu-panel{width:670px!important;max-width:670px;overflow:visible}</style><style>.autocomplete[_ngcontent-serverApp-c6]   input[_ngcontent-serverApp-c6]{font-size:1.1em;padding:6px;text-indent:4px;width:100%;border:0 solid transparent}.autocomplete[_ngcontent-serverApp-c6]   input[_ngcontent-serverApp-c6]:active, .autocomplete[_ngcontent-serverApp-c6]   input[_ngcontent-serverApp-c6]:focus{border-color:transparent;box-shadow:none;outline:transparent solid 0}</style><style>.mat-menu-panel{min-width:112px;max-width:280px;overflow:auto;-webkit-overflow-scrolling:touch;max-height:calc(100vh - 48px);border-radius:4px;outline:0}.mat-menu-panel.ng-animating{pointer-events:none}@media (-ms-high-contrast:active){.mat-menu-panel{outline:solid 1px}}.mat-menu-content:not(:empty){padding-top:8px;padding-bottom:8px}.mat-menu-item{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;cursor:pointer;outline:0;border:none;-webkit-tap-highlight-color:transparent;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;display:block;line-height:48px;height:48px;padding:0 16px;text-align:left;text-decoration:none;max-width:100%;position:relative}.mat-menu-item::-moz-focus-inner{border:0}.mat-menu-item[disabled]{cursor:default}[dir=rtl] .mat-menu-item{text-align:right}.mat-menu-item .mat-icon{margin-right:16px;vertical-align:middle}.mat-menu-item .mat-icon svg{vertical-align:top}[dir=rtl] .mat-menu-item .mat-icon{margin-left:16px;margin-right:0}@media (-ms-high-contrast:active){.mat-menu-item-highlighted,.mat-menu-item.cdk-keyboard-focused,.mat-menu-item.cdk-program-focused{outline:dotted 1px}}.mat-menu-item-submenu-trigger{padding-right:32px}.mat-menu-item-submenu-trigger::after{width:0;height:0;border-style:solid;border-width:5px 0 5px 5px;border-color:transparent transparent transparent currentColor;content:'';display:inline-block;position:absolute;top:50%;right:16px;transform:translateY(-50%)}[dir=rtl] .mat-menu-item-submenu-trigger{padding-right:16px;padding-left:32px}[dir=rtl] .mat-menu-item-submenu-trigger::after{right:auto;left:16px;transform:rotateY(180deg) translateY(-50%)}button.mat-menu-item{width:100%}.mat-menu-item .mat-menu-ripple{top:0;left:0;right:0;bottom:0;position:absolute;pointer-events:none}</style><style>.form-inline[_ngcontent-serverApp-c8]   .form-control[_ngcontent-serverApp-c8]{display:inline-block;width:auto;vertical-align:middle}</style><style>.even[_ngcontent-serverApp-c10]{color:#ff0}.odd[_ngcontent-serverApp-c10]{color:#fff}.citation-button[_ngcontent-serverApp-c10]{background-color:#09c;border-color:#09c;border-bottom-color:#09c!important;padding:8px 16px;text-align:center;border-radius:12px;font-size:10px;transition-duration:.4s;box-shadow:0 2px 4px 0 rgba(0,0,0,.2),0 1px 5px 0 rgba(0,0,0,.03);color:#000}.citation-button[_ngcontent-serverApp-c10]:hover{background-color:#fff;color:#000}.elements[_ngcontent-serverApp-c10] > span[_ngcontent-serverApp-c10]{display:inline-block;margin-right:8px}.btn[_ngcontent-serverApp-c10]{padding:4px 7px;min-width:85px}@media (min-width:768px){.btn[_ngcontent-serverApp-c10]{padding:4px 12px;min-width:100px}}#notfound-background[_ngcontent-serverApp-c10]{position:fixed;width:100%;height:100%;background:#09c;top:0;left:0;z-index:1}#err-title[_ngcontent-serverApp-c10]{text-align:center;font-size:40px;margin-top:40px;margin-bottom:-40px;position:relative;color:#fff}.circles-cut[_ngcontent-serverApp-c10]{position:relative}.circles-cut[_ngcontent-serverApp-c10]:after{content:'';display:inline-block;width:100%;height:100px;background:#fff;position:absolute;top:-50px;left:0;transform:skewY(-5deg);-webkit-transform:skewY(-5deg)}.circles[_ngcontent-serverApp-c10]{background:#fff;text-align:center;position:absolute;width:100%;top:160px;height:70%}#err-message[_ngcontent-serverApp-c10]{margin-top:50px;display:grid}#err-message[_ngcontent-serverApp-c10] > *[_ngcontent-serverApp-c10]{text-align:center}#err-message[_ngcontent-serverApp-c10] > p[_ngcontent-serverApp-c10]{font-size:16px}#err-message[_ngcontent-serverApp-c10]   ul[_ngcontent-serverApp-c10]   li[_ngcontent-serverApp-c10]{list-style:none;font-size:16px}#err-message[_ngcontent-serverApp-c10]   ul[_ngcontent-serverApp-c10]   li[_ngcontent-serverApp-c10]   i[_ngcontent-serverApp-c10]{margin-right:5px}table[_ngcontent-serverApp-c10]{width:100%}</style><style></style><style>.even[_ngcontent-serverApp-c13]{color:#ff0}</style><style>.ddi-next[_ngcontent-serverApp-c22], .ddi-prev[_ngcontent-serverApp-c22]{font-size:3em}</style><style>.ddi-card-panel[_ngcontent-serverApp-c24]{margin:0;border:0;padding:12px 15px;border-bottom:1px solid #ddd;border-radius:unset}</style><style>.mat-form-field{display:inline-block;position:relative;text-align:left}[dir=rtl] .mat-form-field{text-align:right}.mat-form-field-wrapper{position:relative}.mat-form-field-flex{display:inline-flex;align-items:baseline;box-sizing:border-box;width:100%}.mat-form-field-prefix,.mat-form-field-suffix{white-space:nowrap;flex:none;position:relative}.mat-form-field-infix{display:block;position:relative;flex:auto;min-width:0;width:180px}@media (-ms-high-contrast:active){.mat-form-field-infix{border-image:linear-gradient(transparent,transparent)}}.mat-form-field-label-wrapper{position:absolute;left:0;box-sizing:content-box;width:100%;height:100%;overflow:hidden;pointer-events:none}[dir=rtl] .mat-form-field-label-wrapper{left:auto;right:0}.mat-form-field-label{position:absolute;left:0;font:inherit;pointer-events:none;width:100%;white-space:nowrap;text-overflow:ellipsis;overflow:hidden;transform-origin:0 0;transition:transform .4s cubic-bezier(.25,.8,.25,1),color .4s cubic-bezier(.25,.8,.25,1),width .4s cubic-bezier(.25,.8,.25,1);display:none}[dir=rtl] .mat-form-field-label{transform-origin:100% 0;left:auto;right:0}.mat-form-field-can-float.mat-form-field-should-float .mat-form-field-label,.mat-form-field-empty.mat-form-field-label{display:block}.mat-form-field-autofill-control:-webkit-autofill+.mat-form-field-label-wrapper .mat-form-field-label{display:none}.mat-form-field-can-float .mat-form-field-autofill-control:-webkit-autofill+.mat-form-field-label-wrapper .mat-form-field-label{display:block;transition:none}.mat-input-server:focus+.mat-form-field-label-wrapper .mat-form-field-label,.mat-input-server[placeholder]:not(:placeholder-shown)+.mat-form-field-label-wrapper .mat-form-field-label{display:none}.mat-form-field-can-float .mat-input-server:focus+.mat-form-field-label-wrapper .mat-form-field-label,.mat-form-field-can-float .mat-input-server[placeholder]:not(:placeholder-shown)+.mat-form-field-label-wrapper .mat-form-field-label{display:block}.mat-form-field-label:not(.mat-form-field-empty){transition:none}.mat-form-field-underline{position:absolute;width:100%;pointer-events:none;transform:scaleY(1.0001)}.mat-form-field-ripple{position:absolute;left:0;width:100%;transform-origin:50%;transform:scaleX(.5);opacity:0;transition:background-color .3s cubic-bezier(.55,0,.55,.2)}.mat-form-field.mat-focused .mat-form-field-ripple,.mat-form-field.mat-form-field-invalid .mat-form-field-ripple{opacity:1;transform:scaleX(1);transition:transform .3s cubic-bezier(.25,.8,.25,1),opacity .1s cubic-bezier(.25,.8,.25,1),background-color .3s cubic-bezier(.25,.8,.25,1)}.mat-form-field-subscript-wrapper{position:absolute;box-sizing:border-box;width:100%;overflow:hidden}.mat-form-field-label-wrapper .mat-icon,.mat-form-field-subscript-wrapper .mat-icon{width:1em;height:1em;font-size:inherit;vertical-align:baseline}.mat-form-field-hint-wrapper{display:flex}.mat-form-field-hint-spacer{flex:1 0 1em}.mat-error{display:block}.mat-form-field-control-wrapper{position:relative}.mat-form-field._mat-animation-noopable .mat-form-field-label,.mat-form-field._mat-animation-noopable .mat-form-field-ripple{transition:none}</style><style>.mat-form-field-appearance-fill .mat-form-field-flex{border-radius:4px 4px 0 0;padding:.75em .75em 0 .75em}@media (-ms-high-contrast:active){.mat-form-field-appearance-fill .mat-form-field-flex{outline:solid 1px}}.mat-form-field-appearance-fill .mat-form-field-underline::before{content:'';display:block;position:absolute;bottom:0;height:1px;width:100%}.mat-form-field-appearance-fill .mat-form-field-ripple{bottom:0;height:2px}@media (-ms-high-contrast:active){.mat-form-field-appearance-fill .mat-form-field-ripple{height:0;border-top:solid 2px}}.mat-form-field-appearance-fill:not(.mat-form-field-disabled) .mat-form-field-flex:hover~.mat-form-field-underline .mat-form-field-ripple{opacity:1;transform:none;transition:opacity .6s cubic-bezier(.25,.8,.25,1)}.mat-form-field-appearance-fill._mat-animation-noopable:not(.mat-form-field-disabled) .mat-form-field-flex:hover~.mat-form-field-underline .mat-form-field-ripple{transition:none}.mat-form-field-appearance-fill .mat-form-field-subscript-wrapper{padding:0 1em}</style><style>.mat-input-element{font:inherit;background:0 0;color:currentColor;border:none;outline:0;padding:0;margin:0;width:100%;max-width:100%;vertical-align:bottom;text-align:inherit}.mat-input-element:-moz-ui-invalid{box-shadow:none}.mat-input-element::-ms-clear,.mat-input-element::-ms-reveal{display:none}.mat-input-element,.mat-input-element::-webkit-search-cancel-button,.mat-input-element::-webkit-search-decoration,.mat-input-element::-webkit-search-results-button,.mat-input-element::-webkit-search-results-decoration{-webkit-appearance:none}.mat-input-element::-webkit-caps-lock-indicator,.mat-input-element::-webkit-contacts-auto-fill-button,.mat-input-element::-webkit-credentials-auto-fill-button{visibility:hidden}.mat-input-element[type=date]::after,.mat-input-element[type=datetime-local]::after,.mat-input-element[type=datetime]::after,.mat-input-element[type=month]::after,.mat-input-element[type=time]::after,.mat-input-element[type=week]::after{content:' ';white-space:pre;width:1px}.mat-input-element::-webkit-calendar-picker-indicator,.mat-input-element::-webkit-clear-button,.mat-input-element::-webkit-inner-spin-button{font-size:.75em}.mat-input-element::placeholder{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;transition:color .4s .133s cubic-bezier(.25,.8,.25,1)}.mat-input-element::-moz-placeholder{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;transition:color .4s .133s cubic-bezier(.25,.8,.25,1)}.mat-input-element::-webkit-input-placeholder{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;transition:color .4s .133s cubic-bezier(.25,.8,.25,1)}.mat-input-element:-ms-input-placeholder{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;transition:color .4s .133s cubic-bezier(.25,.8,.25,1)}.mat-form-field-hide-placeholder .mat-input-element::placeholder{color:transparent!important;-webkit-text-fill-color:transparent;transition:none}.mat-form-field-hide-placeholder .mat-input-element::-moz-placeholder{color:transparent!important;-webkit-text-fill-color:transparent;transition:none}.mat-form-field-hide-placeholder .mat-input-element::-webkit-input-placeholder{color:transparent!important;-webkit-text-fill-color:transparent;transition:none}.mat-form-field-hide-placeholder .mat-input-element:-ms-input-placeholder{color:transparent!important;-webkit-text-fill-color:transparent;transition:none}textarea.mat-input-element{resize:vertical;overflow:auto}textarea.mat-input-element.cdk-textarea-autosize{resize:none}textarea.mat-input-element{padding:2px 0;margin:-2px 0}select.mat-input-element{-moz-appearance:none;-webkit-appearance:none;position:relative;background-color:transparent;display:inline-flex;box-sizing:border-box;padding-top:1em;top:-1em;margin-bottom:-1em}select.mat-input-element::-ms-expand{display:none}select.mat-input-element::-moz-focus-inner{border:0}select.mat-input-element:not(:disabled){cursor:pointer}select.mat-input-element::-ms-value{color:inherit;background:0 0}@media (-ms-high-contrast:active){.mat-focused select.mat-input-element::-ms-value{color:inherit}}.mat-form-field-type-mat-native-select .mat-form-field-infix::after{content:'';width:0;height:0;border-left:5px solid transparent;border-right:5px solid transparent;border-top:5px solid;position:absolute;top:50%;right:0;margin-top:-2.5px;pointer-events:none}[dir=rtl] .mat-form-field-type-mat-native-select .mat-form-field-infix::after{right:auto;left:0}.mat-form-field-type-mat-native-select .mat-input-element{padding-right:15px}[dir=rtl] .mat-form-field-type-mat-native-select .mat-input-element{padding-right:0;padding-left:15px}.mat-form-field-type-mat-native-select .mat-form-field-label-wrapper{max-width:calc(100% - 10px)}.mat-form-field-type-mat-native-select.mat-form-field-appearance-outline .mat-form-field-infix::after{margin-top:-5px}.mat-form-field-type-mat-native-select.mat-form-field-appearance-fill .mat-form-field-infix::after{margin-top:-10px}</style><style>.mat-form-field-appearance-legacy .mat-form-field-label{transform:perspective(100px);-ms-transform:none}.mat-form-field-appearance-legacy .mat-form-field-prefix .mat-icon,.mat-form-field-appearance-legacy .mat-form-field-suffix .mat-icon{width:1em}.mat-form-field-appearance-legacy .mat-form-field-prefix .mat-icon-button,.mat-form-field-appearance-legacy .mat-form-field-suffix .mat-icon-button{font:inherit;vertical-align:baseline}.mat-form-field-appearance-legacy .mat-form-field-prefix .mat-icon-button .mat-icon,.mat-form-field-appearance-legacy .mat-form-field-suffix .mat-icon-button .mat-icon{font-size:inherit}.mat-form-field-appearance-legacy .mat-form-field-underline{height:1px}@media (-ms-high-contrast:active){.mat-form-field-appearance-legacy .mat-form-field-underline{height:0;border-top:solid 1px}}.mat-form-field-appearance-legacy .mat-form-field-ripple{top:0;height:2px;overflow:hidden}@media (-ms-high-contrast:active){.mat-form-field-appearance-legacy .mat-form-field-ripple{height:0;border-top:solid 2px}}.mat-form-field-appearance-legacy.mat-form-field-disabled .mat-form-field-underline{background-position:0;background-color:transparent}@media (-ms-high-contrast:active){.mat-form-field-appearance-legacy.mat-form-field-disabled .mat-form-field-underline{border-top-style:dotted;border-top-width:2px}}.mat-form-field-appearance-legacy.mat-form-field-invalid:not(.mat-focused) .mat-form-field-ripple{height:1px}</style><style>.mat-form-field-appearance-outline .mat-form-field-wrapper{margin:.25em 0}.mat-form-field-appearance-outline .mat-form-field-flex{padding:0 .75em 0 .75em;margin-top:-.25em;position:relative}.mat-form-field-appearance-outline .mat-form-field-prefix,.mat-form-field-appearance-outline .mat-form-field-suffix{top:.25em}.mat-form-field-appearance-outline .mat-form-field-outline{display:flex;position:absolute;top:.25em;left:0;right:0;bottom:0;pointer-events:none}.mat-form-field-appearance-outline .mat-form-field-outline-end,.mat-form-field-appearance-outline .mat-form-field-outline-start{border:1px solid currentColor;min-width:5px}.mat-form-field-appearance-outline .mat-form-field-outline-start{border-radius:5px 0 0 5px;border-right-style:none}[dir=rtl] .mat-form-field-appearance-outline .mat-form-field-outline-start{border-right-style:solid;border-left-style:none;border-radius:0 5px 5px 0}.mat-form-field-appearance-outline .mat-form-field-outline-end{border-radius:0 5px 5px 0;border-left-style:none;flex-grow:1}[dir=rtl] .mat-form-field-appearance-outline .mat-form-field-outline-end{border-left-style:solid;border-right-style:none;border-radius:5px 0 0 5px}.mat-form-field-appearance-outline .mat-form-field-outline-gap{border-radius:.000001px;border:1px solid currentColor;border-left-style:none;border-right-style:none}.mat-form-field-appearance-outline.mat-form-field-can-float.mat-form-field-should-float .mat-form-field-outline-gap{border-top-color:transparent}.mat-form-field-appearance-outline .mat-form-field-outline-thick{opacity:0}.mat-form-field-appearance-outline .mat-form-field-outline-thick .mat-form-field-outline-end,.mat-form-field-appearance-outline .mat-form-field-outline-thick .mat-form-field-outline-gap,.mat-form-field-appearance-outline .mat-form-field-outline-thick .mat-form-field-outline-start{border-width:2px;transition:border-color .3s cubic-bezier(.25,.8,.25,1)}.mat-form-field-appearance-outline.mat-focused .mat-form-field-outline,.mat-form-field-appearance-outline.mat-form-field-invalid .mat-form-field-outline{opacity:0;transition:opacity .1s cubic-bezier(.25,.8,.25,1)}.mat-form-field-appearance-outline.mat-focused .mat-form-field-outline-thick,.mat-form-field-appearance-outline.mat-form-field-invalid .mat-form-field-outline-thick{opacity:1}.mat-form-field-appearance-outline:not(.mat-form-field-disabled) .mat-form-field-flex:hover .mat-form-field-outline{opacity:0;transition:opacity .6s cubic-bezier(.25,.8,.25,1)}.mat-form-field-appearance-outline:not(.mat-form-field-disabled) .mat-form-field-flex:hover .mat-form-field-outline-thick{opacity:1}.mat-form-field-appearance-outline .mat-form-field-subscript-wrapper{padding:0 1em}.mat-form-field-appearance-outline._mat-animation-noopable .mat-form-field-outline,.mat-form-field-appearance-outline._mat-animation-noopable .mat-form-field-outline-end,.mat-form-field-appearance-outline._mat-animation-noopable .mat-form-field-outline-gap,.mat-form-field-appearance-outline._mat-animation-noopable .mat-form-field-outline-start,.mat-form-field-appearance-outline._mat-animation-noopable:not(.mat-form-field-disabled) .mat-form-field-flex:hover~.mat-form-field-outline{transition:none}</style><style>.mat-form-field-appearance-standard .mat-form-field-flex{padding-top:.75em}.mat-form-field-appearance-standard .mat-form-field-underline{height:1px}@media (-ms-high-contrast:active){.mat-form-field-appearance-standard .mat-form-field-underline{height:0;border-top:solid 1px}}.mat-form-field-appearance-standard .mat-form-field-ripple{bottom:0;height:2px}@media (-ms-high-contrast:active){.mat-form-field-appearance-standard .mat-form-field-ripple{height:0;border-top:2px}}.mat-form-field-appearance-standard.mat-form-field-disabled .mat-form-field-underline{background-position:0;background-color:transparent}@media (-ms-high-contrast:active){.mat-form-field-appearance-standard.mat-form-field-disabled .mat-form-field-underline{border-top-style:dotted;border-top-width:2px}}.mat-form-field-appearance-standard:not(.mat-form-field-disabled) .mat-form-field-flex:hover~.mat-form-field-underline .mat-form-field-ripple{opacity:1;transform:none;transition:opacity .6s cubic-bezier(.25,.8,.25,1)}.mat-form-field-appearance-standard._mat-animation-noopable:not(.mat-form-field-disabled) .mat-form-field-flex:hover~.mat-form-field-underline .mat-form-field-ripple{transition:none}</style><style>.mat-select{display:inline-block;width:100%;outline:0}.mat-select-trigger{display:inline-table;cursor:pointer;position:relative;box-sizing:border-box}.mat-select-disabled .mat-select-trigger{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;cursor:default}.mat-select-value{display:table-cell;max-width:0;width:100%;overflow:hidden;text-overflow:ellipsis;white-space:nowrap}.mat-select-value-text{white-space:nowrap;overflow:hidden;text-overflow:ellipsis}.mat-select-arrow-wrapper{display:table-cell;vertical-align:middle}.mat-form-field-appearance-fill .mat-select-arrow-wrapper{transform:translateY(-50%)}.mat-form-field-appearance-outline .mat-select-arrow-wrapper{transform:translateY(-25%)}.mat-form-field-appearance-standard.mat-form-field-has-label .mat-select:not(.mat-select-empty) .mat-select-arrow-wrapper{transform:translateY(-50%)}.mat-form-field-appearance-standard .mat-select.mat-select-empty .mat-select-arrow-wrapper{transition:transform .4s cubic-bezier(.25,.8,.25,1)}._mat-animation-noopable.mat-form-field-appearance-standard .mat-select.mat-select-empty .mat-select-arrow-wrapper{transition:none}.mat-select-arrow{width:0;height:0;border-left:5px solid transparent;border-right:5px solid transparent;border-top:5px solid;margin:0 4px}.mat-select-panel-wrap{flex-basis:100%}.mat-select-panel{min-width:112px;max-width:280px;overflow:auto;-webkit-overflow-scrolling:touch;padding-top:0;padding-bottom:0;max-height:256px;min-width:100%;border-radius:4px}@media (-ms-high-contrast:active){.mat-select-panel{outline:solid 1px}}.mat-select-panel .mat-optgroup-label,.mat-select-panel .mat-option{font-size:inherit;line-height:3em;height:3em}.mat-form-field-type-mat-select:not(.mat-form-field-disabled) .mat-form-field-flex{cursor:pointer}.mat-form-field-type-mat-select .mat-form-field-label{width:calc(100% - 18px)}.mat-select-placeholder{transition:color .4s .133s cubic-bezier(.25,.8,.25,1)}._mat-animation-noopable .mat-select-placeholder{transition:none}.mat-form-field-hide-placeholder .mat-select-placeholder{color:transparent;-webkit-text-fill-color:transparent;transition:none;display:block}</style><style>mat-table{display:block}mat-header-row{min-height:56px}mat-footer-row,mat-row{min-height:48px}mat-footer-row,mat-header-row,mat-row{display:flex;border-width:0;border-bottom-width:1px;border-style:solid;align-items:center;box-sizing:border-box}mat-footer-row::after,mat-header-row::after,mat-row::after{display:inline-block;min-height:inherit;content:''}mat-cell:first-of-type,mat-footer-cell:first-of-type,mat-header-cell:first-of-type{padding-left:24px}[dir=rtl] mat-cell:first-of-type,[dir=rtl] mat-footer-cell:first-of-type,[dir=rtl] mat-header-cell:first-of-type{padding-left:0;padding-right:24px}mat-cell:last-of-type,mat-footer-cell:last-of-type,mat-header-cell:last-of-type{padding-right:24px}[dir=rtl] mat-cell:last-of-type,[dir=rtl] mat-footer-cell:last-of-type,[dir=rtl] mat-header-cell:last-of-type{padding-right:0;padding-left:24px}mat-cell,mat-footer-cell,mat-header-cell{flex:1;display:flex;align-items:center;overflow:hidden;word-wrap:break-word;min-height:inherit}table.mat-table{border-spacing:0}tr.mat-header-row{height:56px}tr.mat-footer-row,tr.mat-row{height:48px}th.mat-header-cell{text-align:left}[dir=rtl] th.mat-header-cell{text-align:right}td.mat-cell,td.mat-footer-cell,th.mat-header-cell{padding:0;border-bottom-width:1px;border-bottom-style:solid}td.mat-cell:first-of-type,td.mat-footer-cell:first-of-type,th.mat-header-cell:first-of-type{padding-left:24px}[dir=rtl] td.mat-cell:first-of-type,[dir=rtl] td.mat-footer-cell:first-of-type,[dir=rtl] th.mat-header-cell:first-of-type{padding-left:0;padding-right:24px}td.mat-cell:last-of-type,td.mat-footer-cell:last-of-type,th.mat-header-cell:last-of-type{padding-right:24px}[dir=rtl] td.mat-cell:last-of-type,[dir=rtl] td.mat-footer-cell:last-of-type,[dir=rtl] th.mat-header-cell:last-of-type{padding-right:0;padding-left:24px}</style><style>.mat-button .mat-button-focus-overlay,.mat-icon-button .mat-button-focus-overlay{opacity:0}.mat-button:hover .mat-button-focus-overlay,.mat-stroked-button:hover .mat-button-focus-overlay{opacity:.04}@media (hover:none){.mat-button:hover .mat-button-focus-overlay,.mat-stroked-button:hover .mat-button-focus-overlay{opacity:0}}.mat-button,.mat-flat-button,.mat-icon-button,.mat-stroked-button{box-sizing:border-box;position:relative;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;cursor:pointer;outline:0;border:none;-webkit-tap-highlight-color:transparent;display:inline-block;white-space:nowrap;text-decoration:none;vertical-align:baseline;text-align:center;margin:0;min-width:64px;line-height:36px;padding:0 16px;border-radius:4px;overflow:visible}.mat-button::-moz-focus-inner,.mat-flat-button::-moz-focus-inner,.mat-icon-button::-moz-focus-inner,.mat-stroked-button::-moz-focus-inner{border:0}.mat-button[disabled],.mat-flat-button[disabled],.mat-icon-button[disabled],.mat-stroked-button[disabled]{cursor:default}.mat-button.cdk-keyboard-focused .mat-button-focus-overlay,.mat-button.cdk-program-focused .mat-button-focus-overlay,.mat-flat-button.cdk-keyboard-focused .mat-button-focus-overlay,.mat-flat-button.cdk-program-focused .mat-button-focus-overlay,.mat-icon-button.cdk-keyboard-focused .mat-button-focus-overlay,.mat-icon-button.cdk-program-focused .mat-button-focus-overlay,.mat-stroked-button.cdk-keyboard-focused .mat-button-focus-overlay,.mat-stroked-button.cdk-program-focused .mat-button-focus-overlay{opacity:.12}.mat-button::-moz-focus-inner,.mat-flat-button::-moz-focus-inner,.mat-icon-button::-moz-focus-inner,.mat-stroked-button::-moz-focus-inner{border:0}.mat-raised-button{box-sizing:border-box;position:relative;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;cursor:pointer;outline:0;border:none;-webkit-tap-highlight-color:transparent;display:inline-block;white-space:nowrap;text-decoration:none;vertical-align:baseline;text-align:center;margin:0;min-width:64px;line-height:36px;padding:0 16px;border-radius:4px;overflow:visible;transform:translate3d(0,0,0);transition:background .4s cubic-bezier(.25,.8,.25,1),box-shadow 280ms cubic-bezier(.4,0,.2,1)}.mat-raised-button::-moz-focus-inner{border:0}.mat-raised-button[disabled]{cursor:default}.mat-raised-button.cdk-keyboard-focused .mat-button-focus-overlay,.mat-raised-button.cdk-program-focused .mat-button-focus-overlay{opacity:.12}.mat-raised-button::-moz-focus-inner{border:0}._mat-animation-noopable.mat-raised-button{transition:none;animation:none}.mat-stroked-button{border:1px solid currentColor;padding:0 15px;line-height:34px}.mat-stroked-button .mat-button-focus-overlay,.mat-stroked-button .mat-button-ripple.mat-ripple{top:-1px;left:-1px;right:-1px;bottom:-1px}.mat-fab{box-sizing:border-box;position:relative;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;cursor:pointer;outline:0;border:none;-webkit-tap-highlight-color:transparent;display:inline-block;white-space:nowrap;text-decoration:none;vertical-align:baseline;text-align:center;margin:0;min-width:64px;line-height:36px;padding:0 16px;border-radius:4px;overflow:visible;transform:translate3d(0,0,0);transition:background .4s cubic-bezier(.25,.8,.25,1),box-shadow 280ms cubic-bezier(.4,0,.2,1);min-width:0;border-radius:50%;width:56px;height:56px;padding:0;flex-shrink:0}.mat-fab::-moz-focus-inner{border:0}.mat-fab[disabled]{cursor:default}.mat-fab.cdk-keyboard-focused .mat-button-focus-overlay,.mat-fab.cdk-program-focused .mat-button-focus-overlay{opacity:.12}.mat-fab::-moz-focus-inner{border:0}._mat-animation-noopable.mat-fab{transition:none;animation:none}.mat-fab .mat-button-wrapper{padding:16px 0;display:inline-block;line-height:24px}.mat-mini-fab{box-sizing:border-box;position:relative;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;cursor:pointer;outline:0;border:none;-webkit-tap-highlight-color:transparent;display:inline-block;white-space:nowrap;text-decoration:none;vertical-align:baseline;text-align:center;margin:0;min-width:64px;line-height:36px;padding:0 16px;border-radius:4px;overflow:visible;transform:translate3d(0,0,0);transition:background .4s cubic-bezier(.25,.8,.25,1),box-shadow 280ms cubic-bezier(.4,0,.2,1);min-width:0;border-radius:50%;width:40px;height:40px;padding:0;flex-shrink:0}.mat-mini-fab::-moz-focus-inner{border:0}.mat-mini-fab[disabled]{cursor:default}.mat-mini-fab.cdk-keyboard-focused .mat-button-focus-overlay,.mat-mini-fab.cdk-program-focused .mat-button-focus-overlay{opacity:.12}.mat-mini-fab::-moz-focus-inner{border:0}._mat-animation-noopable.mat-mini-fab{transition:none;animation:none}.mat-mini-fab .mat-button-wrapper{padding:8px 0;display:inline-block;line-height:24px}.mat-icon-button{padding:0;min-width:0;width:40px;height:40px;flex-shrink:0;line-height:40px;border-radius:50%}.mat-icon-button .mat-icon,.mat-icon-button i{line-height:24px}.mat-button-focus-overlay,.mat-button-ripple.mat-ripple{top:0;left:0;right:0;bottom:0;position:absolute;pointer-events:none;border-radius:inherit}.mat-button-focus-overlay{opacity:0;transition:opacity .2s cubic-bezier(.35,0,.25,1),background-color .2s cubic-bezier(.35,0,.25,1)}._mat-animation-noopable .mat-button-focus-overlay{transition:none}@media (-ms-high-contrast:active){.mat-button-focus-overlay{background-color:#fff}}@media (-ms-high-contrast:black-on-white){.mat-button-focus-overlay{background-color:#000}}.mat-button-ripple-round{border-radius:50%;z-index:1}.mat-button .mat-button-wrapper>*,.mat-fab .mat-button-wrapper>*,.mat-flat-button .mat-button-wrapper>*,.mat-icon-button .mat-button-wrapper>*,.mat-mini-fab .mat-button-wrapper>*,.mat-raised-button .mat-button-wrapper>*,.mat-stroked-button .mat-button-wrapper>*{vertical-align:middle}.mat-form-field:not(.mat-form-field-appearance-legacy) .mat-form-field-prefix .mat-icon-button,.mat-form-field:not(.mat-form-field-appearance-legacy) .mat-form-field-suffix .mat-icon-button{display:block;font-size:inherit;width:2.5em;height:2.5em}@media (-ms-high-contrast:active){.mat-button,.mat-fab,.mat-flat-button,.mat-icon-button,.mat-mini-fab,.mat-raised-button{outline:solid 1px}}</style><style>.mat-paginator{display:block}.mat-paginator-outer-container{display:flex}.mat-paginator-container{display:flex;align-items:center;justify-content:flex-end;min-height:56px;padding:0 8px;flex-wrap:wrap-reverse;width:100%}.mat-paginator-page-size{display:flex;align-items:baseline;margin-right:8px}[dir=rtl] .mat-paginator-page-size{margin-right:0;margin-left:8px}.mat-paginator-page-size-label{margin:0 4px}.mat-paginator-page-size-select{margin:6px 4px 0 4px;width:56px}.mat-paginator-page-size-select.mat-form-field-appearance-outline{width:64px}.mat-paginator-page-size-select.mat-form-field-appearance-fill{width:64px}.mat-paginator-range-label{margin:0 32px 0 24px}.mat-paginator-range-actions{display:flex;align-items:center}.mat-paginator-icon{width:28px;fill:currentColor}[dir=rtl] .mat-paginator-icon{transform:rotate(180deg)}</style><style>#chord_diagram_input[_ngcontent-serverApp-c23]   h5[_ngcontent-serverApp-c23]{text-align:center}@media (min-width:0px){#shared_molecules_message[_ngcontent-serverApp-c23]{text-align:center}}@media (min-width:800px){#shared_molecules_message[_ngcontent-serverApp-c23]{text-align:left}}</style><style>.mat-option{white-space:nowrap;overflow:hidden;text-overflow:ellipsis;display:block;line-height:48px;height:48px;padding:0 16px;text-align:left;text-decoration:none;max-width:100%;position:relative;cursor:pointer;outline:0;display:flex;flex-direction:row;max-width:100%;box-sizing:border-box;align-items:center;-webkit-tap-highlight-color:transparent}.mat-option[disabled]{cursor:default}[dir=rtl] .mat-option{text-align:right}.mat-option .mat-icon{margin-right:16px;vertical-align:middle}.mat-option .mat-icon svg{vertical-align:top}[dir=rtl] .mat-option .mat-icon{margin-left:16px;margin-right:0}.mat-option[aria-disabled=true]{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;cursor:default}.mat-optgroup .mat-option:not(.mat-option-multiple){padding-left:32px}[dir=rtl] .mat-optgroup .mat-option:not(.mat-option-multiple){padding-left:16px;padding-right:32px}@media (-ms-high-contrast:active){.mat-option{margin:0 1px}.mat-option.mat-active{border:solid 1px currentColor;margin:0}}.mat-option-text{display:inline-block;flex-grow:1;overflow:hidden;text-overflow:ellipsis}.mat-option .mat-option-ripple{top:0;left:0;right:0;bottom:0;position:absolute;pointer-events:none}@media (-ms-high-contrast:active){.mat-option .mat-option-ripple{opacity:.5}}.mat-option-pseudo-checkbox{margin-right:8px}[dir=rtl] .mat-option-pseudo-checkbox{margin-left:8px;margin-right:0}</style><style>@keyframes mat-checkbox-fade-in-background{0%{opacity:0}50%{opacity:1}}@keyframes mat-checkbox-fade-out-background{0%,50%{opacity:1}100%{opacity:0}}@keyframes mat-checkbox-unchecked-checked-checkmark-path{0%,50%{stroke-dashoffset:22.91026}50%{animation-timing-function:cubic-bezier(0,0,.2,.1)}100%{stroke-dashoffset:0}}@keyframes mat-checkbox-unchecked-indeterminate-mixedmark{0%,68.2%{transform:scaleX(0)}68.2%{animation-timing-function:cubic-bezier(0,0,0,1)}100%{transform:scaleX(1)}}@keyframes mat-checkbox-checked-unchecked-checkmark-path{from{animation-timing-function:cubic-bezier(.4,0,1,1);stroke-dashoffset:0}to{stroke-dashoffset:-22.91026}}@keyframes mat-checkbox-checked-indeterminate-checkmark{from{animation-timing-function:cubic-bezier(0,0,.2,.1);opacity:1;transform:rotate(0)}to{opacity:0;transform:rotate(45deg)}}@keyframes mat-checkbox-indeterminate-checked-checkmark{from{animation-timing-function:cubic-bezier(.14,0,0,1);opacity:0;transform:rotate(45deg)}to{opacity:1;transform:rotate(360deg)}}@keyframes mat-checkbox-checked-indeterminate-mixedmark{from{animation-timing-function:cubic-bezier(0,0,.2,.1);opacity:0;transform:rotate(-45deg)}to{opacity:1;transform:rotate(0)}}@keyframes mat-checkbox-indeterminate-checked-mixedmark{from{animation-timing-function:cubic-bezier(.14,0,0,1);opacity:1;transform:rotate(0)}to{opacity:0;transform:rotate(315deg)}}@keyframes mat-checkbox-indeterminate-unchecked-mixedmark{0%{animation-timing-function:linear;opacity:1;transform:scaleX(1)}100%,32.8%{opacity:0;transform:scaleX(0)}}.mat-checkbox-background,.mat-checkbox-frame{top:0;left:0;right:0;bottom:0;position:absolute;border-radius:2px;box-sizing:border-box;pointer-events:none}.mat-checkbox{transition:background .4s cubic-bezier(.25,.8,.25,1),box-shadow 280ms cubic-bezier(.4,0,.2,1);cursor:pointer;-webkit-tap-highlight-color:transparent}._mat-animation-noopable.mat-checkbox{transition:none;animation:none}.mat-checkbox .mat-ripple-element:not(.mat-checkbox-persistent-ripple){opacity:.16}.mat-checkbox-layout{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;cursor:inherit;align-items:baseline;vertical-align:middle;display:inline-flex;white-space:nowrap}.mat-checkbox-label{-webkit-user-select:auto;-moz-user-select:auto;-ms-user-select:auto;user-select:auto}.mat-checkbox-inner-container{display:inline-block;height:16px;line-height:0;margin:auto;margin-right:8px;order:0;position:relative;vertical-align:middle;white-space:nowrap;width:16px;flex-shrink:0}[dir=rtl] .mat-checkbox-inner-container{margin-left:8px;margin-right:auto}.mat-checkbox-inner-container-no-side-margin{margin-left:0;margin-right:0}.mat-checkbox-frame{background-color:transparent;transition:border-color 90ms cubic-bezier(0,0,.2,.1);border-width:2px;border-style:solid}._mat-animation-noopable .mat-checkbox-frame{transition:none}@media (-ms-high-contrast:active){.mat-checkbox.cdk-keyboard-focused .mat-checkbox-frame{border-style:dotted}}.mat-checkbox-background{align-items:center;display:inline-flex;justify-content:center;transition:background-color 90ms cubic-bezier(0,0,.2,.1),opacity 90ms cubic-bezier(0,0,.2,.1)}._mat-animation-noopable .mat-checkbox-background{transition:none}.mat-checkbox-persistent-ripple{width:100%;height:100%;transform:none}.mat-checkbox-inner-container:hover .mat-checkbox-persistent-ripple{opacity:.04}.mat-checkbox.cdk-keyboard-focused .mat-checkbox-persistent-ripple{opacity:.12}.mat-checkbox-persistent-ripple,.mat-checkbox.mat-disabled .mat-checkbox-inner-container:hover .mat-checkbox-persistent-ripple{opacity:0}.mat-checkbox-checkmark{top:0;left:0;right:0;bottom:0;position:absolute;width:100%}.mat-checkbox-checkmark-path{stroke-dashoffset:22.91026;stroke-dasharray:22.91026;stroke-width:2.13333px}.mat-checkbox-mixedmark{width:calc(100% - 6px);height:2px;opacity:0;transform:scaleX(0) rotate(0);border-radius:2px}@media (-ms-high-contrast:active){.mat-checkbox-mixedmark{height:0;border-top:solid 2px;margin-top:2px}}.mat-checkbox-label-before .mat-checkbox-inner-container{order:1;margin-left:8px;margin-right:auto}[dir=rtl] .mat-checkbox-label-before .mat-checkbox-inner-container{margin-left:auto;margin-right:8px}.mat-checkbox-checked .mat-checkbox-checkmark{opacity:1}.mat-checkbox-checked .mat-checkbox-checkmark-path{stroke-dashoffset:0}.mat-checkbox-checked .mat-checkbox-mixedmark{transform:scaleX(1) rotate(-45deg)}.mat-checkbox-indeterminate .mat-checkbox-checkmark{opacity:0;transform:rotate(45deg)}.mat-checkbox-indeterminate .mat-checkbox-checkmark-path{stroke-dashoffset:0}.mat-checkbox-indeterminate .mat-checkbox-mixedmark{opacity:1;transform:scaleX(1) rotate(0)}.mat-checkbox-unchecked .mat-checkbox-background{background-color:transparent}.mat-checkbox-disabled{cursor:default}.mat-checkbox-anim-unchecked-checked .mat-checkbox-background{animation:180ms linear 0s mat-checkbox-fade-in-background}.mat-checkbox-anim-unchecked-checked .mat-checkbox-checkmark-path{animation:180ms linear 0s mat-checkbox-unchecked-checked-checkmark-path}.mat-checkbox-anim-unchecked-indeterminate .mat-checkbox-background{animation:180ms linear 0s mat-checkbox-fade-in-background}.mat-checkbox-anim-unchecked-indeterminate .mat-checkbox-mixedmark{animation:90ms linear 0s mat-checkbox-unchecked-indeterminate-mixedmark}.mat-checkbox-anim-checked-unchecked .mat-checkbox-background{animation:180ms linear 0s mat-checkbox-fade-out-background}.mat-checkbox-anim-checked-unchecked .mat-checkbox-checkmark-path{animation:90ms linear 0s mat-checkbox-checked-unchecked-checkmark-path}.mat-checkbox-anim-checked-indeterminate .mat-checkbox-checkmark{animation:90ms linear 0s mat-checkbox-checked-indeterminate-checkmark}.mat-checkbox-anim-checked-indeterminate .mat-checkbox-mixedmark{animation:90ms linear 0s mat-checkbox-checked-indeterminate-mixedmark}.mat-checkbox-anim-indeterminate-checked .mat-checkbox-checkmark{animation:.5s linear 0s mat-checkbox-indeterminate-checked-checkmark}.mat-checkbox-anim-indeterminate-checked .mat-checkbox-mixedmark{animation:.5s linear 0s mat-checkbox-indeterminate-checked-mixedmark}.mat-checkbox-anim-indeterminate-unchecked .mat-checkbox-background{animation:180ms linear 0s mat-checkbox-fade-out-background}.mat-checkbox-anim-indeterminate-unchecked .mat-checkbox-mixedmark{animation:.3s linear 0s mat-checkbox-indeterminate-unchecked-mixedmark}.mat-checkbox-input{bottom:0;left:50%}.mat-checkbox .mat-checkbox-ripple{position:absolute;left:calc(50% - 20px);top:calc(50% - 20px);height:40px;width:40px;z-index:1;pointer-events:none}</style><style>.mat-sort-header-container{display:flex;cursor:pointer;align-items:center}.mat-sort-header-disabled .mat-sort-header-container{cursor:default}.mat-sort-header-position-before{flex-direction:row-reverse}.mat-sort-header-button{border:none;background:0 0;display:flex;align-items:center;padding:0;cursor:inherit;outline:0;font:inherit;color:currentColor}.mat-sort-header-button::-moz-focus-inner{border:0}.mat-sort-header-arrow{height:12px;width:12px;min-width:12px;position:relative;display:flex;opacity:0}.mat-sort-header-arrow,[dir=rtl] .mat-sort-header-position-before .mat-sort-header-arrow{margin:0 0 0 6px}.mat-sort-header-position-before .mat-sort-header-arrow,[dir=rtl] .mat-sort-header-arrow{margin:0 6px 0 0}.mat-sort-header-stem{background:currentColor;height:10px;width:2px;margin:auto;display:flex;align-items:center}@media (-ms-high-contrast:active){.mat-sort-header-stem{width:0;border-left:solid 2px}}.mat-sort-header-indicator{width:100%;height:2px;display:flex;align-items:center;position:absolute;top:0;left:0}.mat-sort-header-pointer-middle{margin:auto;height:2px;width:2px;background:currentColor;transform:rotate(45deg)}@media (-ms-high-contrast:active){.mat-sort-header-pointer-middle{width:0;height:0;border-top:solid 2px;border-left:solid 2px}}.mat-sort-header-pointer-left,.mat-sort-header-pointer-right{background:currentColor;width:6px;height:2px;position:absolute;top:0}@media (-ms-high-contrast:active){.mat-sort-header-pointer-left,.mat-sort-header-pointer-right{width:0;height:0;border-left:solid 6px;border-top:solid 2px}}.mat-sort-header-pointer-left{transform-origin:right;left:0}.mat-sort-header-pointer-right{transform-origin:left;right:0}</style></head>


<body class="level22" style="padding-top: 70px; background-color: #FFF;" data-new-gr-c-s-check-loaded="8.907.0" data-gr-ext-installed="">
    <app-root _nghost-sc0="" ng-version="7.2.16" _nghost-serverapp-c0=""><div _ngcontent-serverapp-c0="" class="container_ddi" id="wrapper1" style="margin:0px;"><nav _ngcontent-serverapp-c0="" role="navigation" style="border: none" class="navbar navbar-inverse navbar-fixed-top row"><div _ngcontent-serverapp-c0="" class="navbar-header"><div _ngcontent-serverapp-c0="" class="nav-item" style="padding: 10px 0 10px 15px; float: left;"><a _ngcontent-serverapp-c0="" class="nav-link text-center nav-element" style="padding: 20px 15px" title="OmicsDI Homepage" href="https://www.omicsdi.org/"><img _ngcontent-serverapp-c0="" alt="logo" style="height: 25px;" src="page_with_metadata_files/imageedit_2_3220380406.png"><!----><span _ngcontent-serverapp-c0="" style="margin-left: 8px; font-size: 10pt; vertical-align: bottom" class="ng-star-inserted">OmicsDI</span></a></div><button _ngcontent-serverapp-c0="" class="navbar-toggle" type="button"><span _ngcontent-serverapp-c0="" class="sr-only">Toggle navigation</span><span _ngcontent-serverapp-c0="" class="icon-bar"></span><span _ngcontent-serverapp-c0="" class="icon-bar"></span><span _ngcontent-serverapp-c0="" class="icon-bar"></span></button></div><div _ngcontent-serverapp-c0="" class="nav navbar-left top-nav col-lg-5 col-md-10 col-sm-9 col-xs-8" style="padding: 0"><ul _ngcontent-serverapp-c0="" class="nav navbar-nav col-lg-7 col-md-12 col-sm-12 navbar-collapse collapse"><li _ngcontent-serverapp-c0="" class="nav-item"><a _ngcontent-serverapp-c0="" class="nav-link text-center nav-element" routerlink="/search" href="https://www.omicsdi.org/search">Browse</a></li><li _ngcontent-serverapp-c0="" class="nav-item"><a _ngcontent-serverapp-c0="" class="nav-link text-center nav-element">Submit Data</a></li><li _ngcontent-serverapp-c0="" class="nav-item"><a _ngcontent-serverapp-c0="" class="nav-link text-center nav-element" href="https://www.omicsdi.org/database">Databases</a></li><li _ngcontent-serverapp-c0="" class="nav-item"><a _ngcontent-serverapp-c0="" class="nav-link text-center nav-element">API</a></li><li _ngcontent-serverapp-c0="" class="nav-item" dropdown=""><a _ngcontent-serverapp-c0="" aria-controls="help-dropdown" class="nav-link text-center dropdown-toggle nav-element" dropdowntoggle="" aria-haspopup="true"><i _ngcontent-serverapp-c0="" class="fa fa-bell"></i> Help <b _ngcontent-serverapp-c0="" class="caret"></b></a><!----></li><!----><li _ngcontent-serverapp-c0="" dropdown="" class="ng-star-inserted"><a _ngcontent-serverapp-c0="" aria-controls="login-dropdown" class="nav-link text-center dropdown-toggle nav-element" data-icon="M" data-toggle="dropdown" dropdowntoggle="" style="color:white; border: none" aria-haspopup="true"><span _ngcontent-serverapp-c0="" class="glyphicon glyphicon-user"></span> Login </a><!----></li><!----><!----><!----></ul></div><ul _ngcontent-serverapp-c0="" class="nav navbar-right top-nav hidden-md hidden-sm hidden-xs col-lg-5 omics-search-box"><!----><li _ngcontent-serverapp-c0="" app-search-box="" class="col-md-6 col-sm-12 col-xs-12 functional ng-star-inserted" style="width:100%;"><div class="search-box"><form class="local-search ng-untouched ng-pristine ng-valid" name="queryForm" novalidate=""><div class="form-group "><div class="input-group "><app-autocomplete-n _nghost-serverapp-c6=""><div _ngcontent-serverapp-c6="" class="autocomplete"><div class="ngui-auto-complete-wrapper" style="position: relative;"><input _ngcontent-serverapp-c6="" auto-complete="" min-chars="3" no-match-found-text=" " placeholder="Organism, repository, gene, tissue, accession" type="text" class="ng-untouched ng-pristine ng-valid" autocomplete="off"></div></div></app-autocomplete-n><div class="input-group-btn"><button aria-haspopup="true" class="btn btn-primary ddi-btn" type="button"><i aria-hidden="true" class="fa fa-caret-down"></i> Advanced </button><button class="btn btn-primary ddi-btn" type="submit" value="submit"><i class="fa fa-search"></i> Search </button></div></div><!----><!----></div></form></div><mat-menu id="advancedSearchMenu" class="ng-tns-c7-0"><!----></mat-menu></li></ul></nav><ng-progress _ngcontent-serverapp-c0="" role="progressbar" spinnerposition="left" dir="ltr+" thick="false" fixed="true"><!----><!----><div class="ng-progress-bar ng-star-inserted" style="transition: opacity 200ms linear 0s;"><div class="ng-bar-placeholder"><div class="ng-bar" style="transform: translate3d(0%, 0px, 0px); background-color: red; transition: none 0s ease 0s;"><!----><div class="ng-meteor ng-star-inserted" style="box-shadow: red 0px 0px 10px, red 0px 0px 5px;"></div></div></div><!----></div></ng-progress><div _ngcontent-serverapp-c0="" style=""><router-outlet _ngcontent-serverapp-c0=""></router-outlet><app-dataset _nghost-serverapp-c10="" class="ng-star-inserted"><div _ngcontent-serverapp-c10="" class="container_ddi" id="datasetCtrl"><!----><div _ngcontent-serverapp-c10="" class="row ng-star-inserted"><div _ngcontent-serverapp-c10="" class="col-lg-1 visible-lg" id="app-social" style="display: block; position: relative"><span _ngcontent-serverapp-c10="" style="position: fixed; left: 2.5%"><app-socialnetworks _ngcontent-serverapp-c10="" _nghost-serverapp-c11=""><div _ngcontent-serverapp-c11="" id="socialnetworks" ng-show="get_dataset_fail.length&lt;1" style="display: flex;align-items: center;"><div _ngcontent-serverapp-c11="" class="simple-skinnyleft" style="margin: auto"><a _ngcontent-serverapp-c11="" class="tweetcountwrapper"><i _ngcontent-serverapp-c11="" class="fa fa-twitter"></i></a><a _ngcontent-serverapp-c11="" class="facebookcountwrapper"><i _ngcontent-serverapp-c11="" class="fa fa-facebook"></i></a><a _ngcontent-serverapp-c11="" class="emailcountwrapper"><i _ngcontent-serverapp-c11="" class="fa fa-envelope"></i></a><a _ngcontent-serverapp-c11="" class="tumblrcountwrapper"><i _ngcontent-serverapp-c11="" class="fa fa-tumblr"></i></a><a _ngcontent-serverapp-c11="" class="linkedincountwrapper"><i _ngcontent-serverapp-c11="" class="fa fa-linkedin"></i></a><a _ngcontent-serverapp-c11="" class="googlepluscountwrapper"><i _ngcontent-serverapp-c11="" class="fa fa-google-plus"></i></a></div></div></app-socialnetworks></span></div><div _ngcontent-serverapp-c10="" class="col-lg-8 col-md-9 col-sm-12 col-xs-12" id="centerpanel"><div _ngcontent-serverapp-c10="" class="mt-card"><div _ngcontent-serverapp-c10="" class="mt-card-header" style="padding: 10px 0; border-bottom: 1px solid #dddddd; display: flex; align-items: center"><div _ngcontent-serverapp-c10="" class="col-xs-10" style="display: flex; align-items: center"><div _ngcontent-serverapp-c10="" style="width: 50px"><app-omics-image _ngcontent-serverapp-c10="" _nghost-serverapp-c12=""><img _ngcontent-serverapp-c12="" style="border-radius: 50%" src="page_with_metadata_files/Transcriptomics.svg" alt="Transcriptomics" title="Transcriptomics"></app-omics-image></div><h3 _ngcontent-serverapp-c10="" class="mt-card-title" style="padding: 0; margin: 0 10px; font-weight: 500">Dataset Information</h3></div><div _ngcontent-serverapp-c10="" class="col-xs-2"><div _ngcontent-serverapp-c10="" apppiechart="" class="pull-right" style="position: relative;"><svg width="65" viewBox="0 0 720 720" class="rosette-chart" height="65"><defs><linearGradient id="E-GEOD-37196_gradient_0" x1="50%" y1="100%" x2="50%" y2="0%"><stop offset="0%" style="stop-color:rgb(255,255,255);stop-opacity:1"></stop><stop offset="100%" style="stop-color:#2E8B57;stop-opacity:1"></stop></linearGradient><linearGradient id="E-GEOD-37196_gradient_1" x1="21%" y1="100%" x2="79%" y2="0%"><stop offset="0%" style="stop-color:rgb(255,255,255);stop-opacity:1"></stop><stop offset="100%" style="stop-color:#2E8B57;stop-opacity:1"></stop></linearGradient><linearGradient id="E-GEOD-37196_gradient_2" x1="0%" y1="71%" x2="100%" y2="28.999999999999996%"><stop offset="0%" style="stop-color:rgb(255,255,255);stop-opacity:1"></stop><stop offset="100%" style="stop-color:#2E8B57;stop-opacity:1"></stop></linearGradient><linearGradient id="E-GEOD-37196_gradient_3" x1="0%" y1="0%" x2="100%" y2="100%"><stop offset="0%" style="stop-color:rgb(255,255,255);stop-opacity:1"></stop><stop offset="100%" style="stop-color:#2E8B57;stop-opacity:1"></stop></linearGradient><linearGradient id="E-GEOD-37196_gradient_4" x1="0%" y1="36.5%" x2="100%" y2="63.5%"><stop offset="0%" style="stop-color:rgb(255,255,255);stop-opacity:1"></stop><stop offset="100%" style="stop-color:#2E8B57;stop-opacity:1"></stop></linearGradient><linearGradient id="E-GEOD-37196_gradient_5" x1="21%" y1="0%" x2="79%" y2="100%"><stop offset="0%" style="stop-color:rgb(255,255,255);stop-opacity:1"></stop><stop offset="100%" style="stop-color:#2E8B57;stop-opacity:1"></stop></linearGradient><linearGradient id="E-GEOD-37196_gradient_6" x1="50%" y1="0%" x2="50%" y2="100%"><stop offset="0%" style="stop-color:rgb(255,255,255);stop-opacity:1"></stop><stop offset="100%" style="stop-color:#2E8B57;stop-opacity:1"></stop></linearGradient><linearGradient id="E-GEOD-37196_gradient_7" x1="86.5%" y1="0%" x2="13.5%" y2="100%"><stop offset="0%" style="stop-color:rgb(255,255,255);stop-opacity:1"></stop><stop offset="100%" style="stop-color:#2E8B57;stop-opacity:1"></stop></linearGradient><linearGradient id="E-GEOD-37196_gradient_8" x1="100%" y1="21%" x2="0%" y2="79%"><stop offset="0%" style="stop-color:rgb(255,255,255);stop-opacity:1"></stop><stop offset="100%" style="stop-color:#2E8B57;stop-opacity:1"></stop></linearGradient><linearGradient id="E-GEOD-37196_gradient_9" x1="100%" y1="50%" x2="0%" y2="50%"><stop offset="0%" style="stop-color:rgb(255,255,255);stop-opacity:1"></stop><stop offset="100%" style="stop-color:#2E8B57;stop-opacity:1"></stop></linearGradient><linearGradient id="E-GEOD-37196_gradient_10" x1="100%" y1="86.5%" x2="0%" y2="13.5%"><stop offset="0%" style="stop-color:rgb(255,255,255);stop-opacity:1"></stop><stop offset="100%" style="stop-color:#2E8B57;stop-opacity:1"></stop></linearGradient><linearGradient id="E-GEOD-37196_gradient_11" x1="129%" y1="100%" x2="-28.999999999999996%" y2="0%"><stop offset="0%" style="stop-color:rgb(255,255,255);stop-opacity:1"></stop><stop offset="100%" style="stop-color:#2E8B57;stop-opacity:1"></stop></linearGradient></defs><g style="cursor: pointer" transform="matrix(2.769449835201722,0,0,2.769449835201722,-623.698285897772,-621.9327602781555)"><path d="m 378.9895761619361,264.3949755505212 c -5.7581064046051,-9.760471988086 -13.9767744892666,-17.8967349868568 -23.7880988708504,-23.5771958597846 -19.6283610230305,11.35944983617 -32.8333198402356,32.5714433799553 -32.8333198402356,56.8810218555477 8e-13,24.309578475592 13.1997639769653,45.5005305810539 32.8281250000002,56.859980417222 -11.3189343625164,-19.6517526426536 -12.1487673822503,-44.6243573250858 0.00602185554,-65.67706984024 5.9880211686214,-10.3715569008459 14.3702453380125,-18.4586997617603 24.0528968555452,-24.0266605769822 -0.088750556225,-0.1532609037314 -0.1756740404876,-0.307601595516 -0.2656250000002,-0.4600759957605 z" fill="url('/dataset/arrayexpress-repository/E-GEOD-37196#E-GEOD-37196_gradient_0')" style="stroke:none"></path><path d="m 420.8837824508476,288.3712819635018 c -0.1064304300389,-11.3318698969113 -3.1558742772426,-22.4874143880489 -8.8125,-32.3125 -22.6783841986818,0.023391619623 -44.7202107622013,11.79103748485 -56.8749999999976,32.8437500000043 -12.1547892377953,21.0527125151545 -11.3189343625162,46.0044973573463 3e-13,65.6562500000003 0.023391619619,-22.6783841986843 11.7910374848463,-44.7202107622037 32.8437499999955,-56.8750000000028 10.371556900848,-5.9880211686176 21.6743474022139,-8.8005802455077 32.8437500000001,-8.7812500000001 -2.297844251e-4,-0.1771031141508 0.00166261591,-0.3542278162052 -2e-13,-0.5312500000001 z" fill="url('/dataset/arrayexpress-repository/E-GEOD-37196#E-GEOD-37196_gradient_1')" style="stroke:none"></path><path d="m 445.1770761619407,330.0824755505187 c 5.5737634923062,-9.8669024181249 8.5106398987825,-21.0526092640995 8.5244011291495,-32.3896958597847 -19.6517526426534,-11.3189343625115 -44.6243573250855,-12.1487673822458 -65.6770698402401,0.00602185555 -21.0527125151536,12.1547892377968 -32.8047333803807,34.1815962185377 -32.8281249999998,56.8599804172223 11.359449836168,-19.6283610230347 32.5714433799533,-32.8333198402394 56.8810218555426,-32.8333198402444 11.9760423372391,2e-12 23.1708255835205,3.2156476404536 32.8341468555453,8.8170894230179 0.088352557926,-0.1534906881563 0.1785537757177,-0.3059389796057 0.2656249999998,-0.4600759957606 z" fill="url('/dataset/arrayexpress-repository/E-GEOD-37196#E-GEOD-37196_gradient_2')" style="stroke:none"></path><path d="m 445.3600888638353,378.352075674592 c 9.760471988086,-5.758106404605 17.8967349868569,-13.9767744892664 23.5771958597846,-23.7880988708504 -11.3594498361699,-19.6283610230302 -32.5714433799552,-32.8333198402352 -56.8810218555477,-32.8333198402357 -24.309578475592,9e-13 -45.5005305810539,13.1997639769654 -56.8599804172219,32.8281250000004 19.6517526426538,-11.3189343625165 44.6243573250856,-12.1487673822505 65.6770698402401,0.00602185554 10.371556900846,5.9880211686212 18.4586997617602,14.3702453380126 24.0266605769822,24.0528968555453 0.1532609037312,-0.088750556225 0.307601595516,-0.1756740404876 0.4600759957604,-0.2656250000003 z" fill="url('/dataset/arrayexpress-repository/E-GEOD-37196#E-GEOD-37196_gradient_3')" style="stroke:none"></path><path d="m 421.3837824508547,420.2462819635038 c 11.3318698969114,-0.1064304300389 22.4874143880489,-3.1558742772425 32.3124999999998,-8.8125000000003 -0.023391619623,-22.6783841986814 -11.7910374848499,-44.720210762201 -32.8437500000043,-56.8749999999976 -21.0527125151545,-12.1547892377952 -46.004497357346,-11.3189343625162 -65.65625,4e-13 22.6783841986841,0.023391619619 44.7202107622035,11.7910374848462 56.8750000000027,32.8437499999955 5.9880211686178,10.371556900848 8.8005802455077,21.674347402214 8.78125,32.8437500000002 0.1771031141507,-2.297844253e-4 0.3542278162053,0.00166261591 0.5312500000001,-2e-13 z" fill="url('/dataset/arrayexpress-repository/E-GEOD-37196#E-GEOD-37196_gradient_4')" style="stroke:none"></path><path d="m 379.6725888638378,444.5395756745965 c 9.8669024181249,5.5737634923063 21.0526092640994,8.5106398987828 32.3896958597846,8.5244011291495 11.3189343625117,-19.6517526426532 12.1487673822459,-44.6243573250854 -0.00602185555,-65.6770698402401 -12.1547892377967,-21.0527125151536 -34.1815962185376,-32.8047333803806 -56.8599804172221,-32.8281249999996 19.6283610230345,11.3594498361679 32.8333198402394,32.5714433799532 32.8333198402444,56.8810218555426 -2.1e-12,11.9760423372391 -3.2156476404539,23.1708255835205 -8.8170894230179,32.8341468555455 0.1534906881566,0.088352557925 0.3059389796059,0.1785537757177 0.4600759957606,0.2656249999997 z" fill="url('/dataset/arrayexpress-repository/E-GEOD-37196#E-GEOD-37196_gradient_5')" style="stroke:none"></path><path d="m 331.4029887397643,444.7225883764912 c 5.758106404605,9.7604719880859 13.9767744892662,17.896734986857 23.7880988708504,23.5771958597845 19.6283610230303,-11.3594498361699 32.8333198402354,-32.5714433799552 32.8333198402359,-56.8810218555478 -1.1e-12,-24.309578475592 -13.1997639769654,-45.5005305810537 -32.8281250000003,-56.8599804172217 11.3189343625163,19.6517526426537 12.1487673822505,44.6243573250857 -0.00602185554,65.67706984024 -5.9880211686216,10.3715569008459 -14.370245338013,18.4586997617602 -24.0528968555454,24.0266605769824 0.088750556226,0.1532609037307 0.1756740404877,0.307601595516 0.2656250000002,0.4600759957603 z" fill="url('/dataset/arrayexpress-repository/E-GEOD-37196#E-GEOD-37196_gradient_6')" style="stroke:none"></path><path d="m 289.5087824508529,420.7462819635105 c 0.106430430039,11.3318698969113 3.1558742772424,22.4874143880491 8.8125000000001,32.3124999999998 22.6783841986818,-0.023391619623 44.7202107622011,-11.7910374848498 56.8749999999978,-32.8437500000043 12.1547892377952,-21.0527125151546 11.3189343625161,-46.0044973573459 -5e-13,-65.6562499999997 -0.023391619619,22.6783841986841 -11.7910374848461,44.7202107622035 -32.8437499999954,56.8750000000026 -10.3715569008481,5.9880211686177 -21.6743474022142,8.8005802455077 -32.8437500000004,8.7812500000003 2.297844263e-4,0.1771031141507 -0.00166261591,0.3542278162052 3e-13,0.5312499999998 z" fill="url('/dataset/arrayexpress-repository/E-GEOD-37196#E-GEOD-37196_gradient_7')" style="stroke:none"></path><path d="m 265.2154887397601,379.0350883764934 c -5.5737634923061,9.8669024181251 -8.5106398987828,21.0526092640995 -8.5244011291495,32.3896958597847 19.6517526426535,11.3189343625118 44.6243573250854,12.1487673822458 65.6770698402401,-0.00602185555 21.0527125151537,-12.1547892377968 32.8047333803805,-34.1815962185374 32.8281249999995,-56.859980417222 -11.359449836168,19.6283610230345 -32.5714433799531,32.8333198402395 -56.8810218555425,32.8333198402444 -11.9760423372391,-2.2e-12 -23.1708255835207,-3.2156476404538 -32.8341468555456,-8.8170894230179 -0.088352557924,0.1534906881571 -0.1785537757177,0.3059389796057 -0.2656249999997,0.4600759957606 z" fill="url('/dataset/arrayexpress-repository/E-GEOD-37196#E-GEOD-37196_gradient_8')" style="stroke:none"></path><path d="m 265.0324760378657,330.7654882524203 c -9.760471988086,5.7581064046052 -17.8967349868572,13.9767744892663 -23.5771958597847,23.7880988708502 11.35944983617,19.6283610230307 32.5714433799553,32.8333198402353 56.8810218555478,32.8333198402359 24.309578475592,-9e-13 45.5005305810537,-13.1997639769654 56.8599804172215,-32.8281250000001 -19.6517526426535,11.3189343625163 -44.6243573250855,12.1487673822505 -65.6770698402397,-0.00602185554 -10.3715569008459,-5.9880211686215 -18.4586997617603,-14.3702453380129 -24.0266605769824,-24.0528968555454 -0.1532609037301,0.088750556226 -0.3076015955161,0.1756740404876 -0.4600759957603,0.2656250000003 z" fill="url('/dataset/arrayexpress-repository/E-GEOD-37196#E-GEOD-37196_gradient_9')" style="stroke:none"></path><path d="m 289.0087824508462,288.8712819635089 c -11.3318698969114,0.106430430039 -22.4874143880491,3.1558742772423 -32.3124999999999,8.8125 0.023391619623,22.6783841986819 11.7910374848502,44.720210762201 32.8437500000042,56.8749999999978 21.0527125151547,12.1547892377951 46.0044973573459,11.318934362516 65.6562499999998,-5e-13 -22.6783841986841,-0.023391619619 -44.7202107622035,-11.791037484846 -56.8750000000025,-32.8437499999955 -5.9880211686176,-10.3715569008482 -8.8005802455075,-21.6743474022142 -8.7812500000001,-32.8437500000003 -0.1771031141501,2.297844264e-4 -0.3542278162053,-0.00166261591 -0.53125,3e-13 z" fill="url('/dataset/arrayexpress-repository/E-GEOD-37196#E-GEOD-37196_gradient_10')" style="stroke:none"></path><path d="m 330.7199760378628,264.5779882524163 c -9.8669024181249,-5.5737634923063 -21.0526092640994,-8.5106398987829 -32.3896958597846,-8.5244011291496 -11.3189343625118,19.6517526426536 -12.1487673822455,44.6243573250852 0.00602185555,65.6770698402401 12.154789237797,21.0527125151536 34.1815962185375,32.8047333803803 56.8599804172221,32.8281249999995 -19.6283610230345,-11.359449836168 -32.8333198402396,-32.571443379953 -32.8333198402443,-56.8810218555425 2.1e-12,-11.9760423372393 3.2156476404538,-23.1708255835206 8.8170894230178,-32.8341468555454 -0.1534906881565,-0.088352557924 -0.3059389796056,-0.1785537757178 -0.4600759957605,-0.2656249999999 z" fill="url('/dataset/arrayexpress-repository/E-GEOD-37196#E-GEOD-37196_gradient_11')" style="stroke:none"></path><circle cx="355" cy="355" fill="white" r="52"></circle><text x="355" y="355" dy=".3em" style="font-size: 32px" text-anchor="middle">2</text></g></svg><div class="mt-card tooltip-rosette" style="position: absolute; display: none; width: 220px; background: white; right: 75px; z-index: 9999; top: -100px; padding: 3px;" position="absolute"><div style="width: 100%; display: flex; align-items: center;"><svg viewBox="0 0 36 36" width="28px" height="28px" class="circular-chart"><path class="circle-bg" d="M18 2.0845
          a 15.9155 15.9155 0 0 1 0 31.831
          a 15.9155 15.9155 0 0 1 0 -31.831"></path><path class="circle" stroke-dasharray="0.215916101172116, 100" style="stroke: rgb(46, 139, 87);" d="M18 2.0845
          a 15.9155 15.9155 0 0 1 0 31.831
          a 15.9155 15.9155 0 0 1 0 -31.831"></path><text x="18" y="22" class="percentage">&lt;1</text></svg><p style="margin: 0px 10px; padding: 0px;">7 Views</p></div><div style="width: 100%; display: flex; align-items: center;"><svg viewBox="0 0 36 36" width="28px" height="28px" class="circular-chart"><path class="circle-bg" d="M18 2.0845
          a 15.9155 15.9155 0 0 1 0 31.831
          a 15.9155 15.9155 0 0 1 0 -31.831"></path><text x="18" y="22" class="percentage">0</text></svg><p style="margin: 0px 10px; padding: 0px;">0 Connections</p></div><div style="width: 100%; display: flex; align-items: center;"><svg viewBox="0 0 36 36" width="28px" height="28px" class="circular-chart"><path class="circle-bg" d="M18 2.0845
          a 15.9155 15.9155 0 0 1 0 31.831
          a 15.9155 15.9155 0 0 1 0 -31.831"></path><text x="18" y="22" class="percentage">0</text></svg><p style="margin: 0px 10px; padding: 0px;">0 Citations</p></div><div style="width: 100%; display: flex; align-items: center;"><svg viewBox="0 0 36 36" width="28px" height="28px" class="circular-chart"><path class="circle-bg" d="M18 2.0845
          a 15.9155 15.9155 0 0 1 0 31.831
          a 15.9155 15.9155 0 0 1 0 -31.831"></path><text x="18" y="22" class="percentage">0</text></svg><p style="margin: 0px 10px; padding: 0px;">0 Reanalyses</p></div><div style="width: 100%; display: flex; align-items: center;"><svg viewBox="0 0 36 36" width="28px" height="28px" class="circular-chart"><path class="circle-bg" d="M18 2.0845
          a 15.9155 15.9155 0 0 1 0 31.831
          a 15.9155 15.9155 0 0 1 0 -31.831"></path><text x="18" y="22" class="percentage">0</text></svg><p style="margin: 0px 10px; padding: 0px;">0 Downloads</p></div><hr style="width: 96%; margin: 10px auto;"><div style="width: 100%; padding: 0px 0px 10px 10px;">Omics score: <b>2</b><a style="float: right; border-bottom: medium;" href="http://blog.omicsdi.org/post/rosette-chart" target="_blank" class="rosette-help"><i style="font-size: 1.2em; margin-top: 3px; margin-right: 15px;" class="fa fa-question-circle"></i></a></div><div style="width: 0px; height: 0px; border-top: 8px solid transparent; border-bottom: 8px solid transparent; position: absolute; top: 110px; right: -8px; border-left: 8px solid rgb(192, 192, 192);"></div><div style="width: 0px; height: 0px; border-top: 8px solid transparent; border-bottom: 8px solid transparent; position: absolute; top: 110px; right: -7px; border-left: 8px solid white;"></div></div></div></div></div><div _ngcontent-serverapp-c10="" class="mt-card-body"><div _ngcontent-serverapp-c10="" class="" id="dataset_upper_right" style="overflow:hidden;"><div _ngcontent-serverapp-c10="" style="margin-bottom: 25px"><h3 _ngcontent-serverapp-c10="" id="dataset-title"><app-annotated-text _ngcontent-serverapp-c10="" _nghost-serverapp-c13=""><!----><span _ngcontent-serverapp-c13="" class="ng-star-inserted">Interference of PPAR gamma signaling in thoracic aorta</span><!----></app-annotated-text></h3><p _ngcontent-serverapp-c10="" class="hotword" id="ontology_highlight_box" style=""><i _ngcontent-serverapp-c10="" aria-hidden="true" class="fa fa-pencil-square-o">Ontology highlight</i></p></div><hr _ngcontent-serverapp-c10=""><div _ngcontent-serverapp-c10=""><div _ngcontent-serverapp-c10=""><p _ngcontent-serverapp-c10="" class="elements"><b _ngcontent-serverapp-c10="">ABSTRACT</b>: <app-annotated-text _ngcontent-serverapp-c10="" id="abstract" _nghost-serverapp-c13=""><!----><span _ngcontent-serverapp-c13="" class="ng-star-inserted">This
 SuperSeries is composed of the following subset Series: GSE37194: Gene 
expression profiling during interference with PPAR gamma signaling in 
thoracic aorta GSE37195: Gene expression profiling using exon arrays 
during interference with PPAR gamma signaling in thoracic aorta Refer to
 individual Series</span><!----></app-annotated-text></p></div></div><!----><!----><!----><p _ngcontent-serverapp-c10="" class="elements ng-star-inserted" id="organisms"><b _ngcontent-serverapp-c10="">ORGANISM(S): </b><!----><span _ngcontent-serverapp-c10="" class="ng-star-inserted"> Mus Musculus </span></p><!----><!----><!----><p _ngcontent-serverapp-c10="" class="elements ng-star-inserted" id="submitter"><b _ngcontent-serverapp-c10="">SUBMITTER: </b><!----><span _ngcontent-serverapp-c10="" class="ng-star-inserted"><!----><span _ngcontent-serverapp-c10="" class="ng-star-inserted"><a _ngcontent-serverapp-c10="" href="mailto:henry-keen@uiowa.edu"><img _ngcontent-serverapp-c10="" src="page_with_metadata_files/rsz_mail.png">Henry Keen </a></span><!---->&nbsp; </span></p><!----><p _ngcontent-serverapp-c10="" class="elements"><b _ngcontent-serverapp-c10="">PROVIDER: </b><span _ngcontent-serverapp-c10=""><a _ngcontent-serverapp-c10="" id="id" href="https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-37196">E-GEOD-37196</a> | <a _ngcontent-serverapp-c10="" href="https://www.ebi.ac.uk/arrayexpress/"><span _ngcontent-serverapp-c10="" class="ng-binding" id="source">ArrayExpress</span></a><span _ngcontent-serverapp-c10="" id="publicationDate"> | 2012-10-03 </span></span></p><!----><!----><p _ngcontent-serverapp-c10="" class="elements ng-star-inserted"><b _ngcontent-serverapp-c10="">REPOSITORIES: </b><!----><span _ngcontent-serverapp-c10="" class="repositories ng-star-inserted"> ArrayExpress</span></p><p _ngcontent-serverapp-c10=""><a _ngcontent-serverapp-c10="" class="hidden-sm hidden-md hidden-lg" href="https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-37196">ACCESS DATA</a></p></div><div _ngcontent-serverapp-c10="" class="row" id="dataset_bottom_repository" style="margin-top:10px"></div><div _ngcontent-serverapp-c10="" class="row"><div _ngcontent-serverapp-c10="" class="col-lg-2 hidden-xs"><!----><!----><a _ngcontent-serverapp-c10="" class="btn btn-primary ng-star-inserted" id="full_dataset_link" style="color: white; border: 1px solid transparent; margin-left: 9px" href="https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-37196">ACCESS DATA</a></div><div _ngcontent-serverapp-c10="" class="col-lg-10 col-xs-12"><div _ngcontent-serverapp-c10="" class="btn-group pull-right widget-btn-group" role="group"><button _ngcontent-serverapp-c10="" class="btn btn-primary" type="button"><i _ngcontent-serverapp-c10="" aria-hidden="true" class="fa fa-quote-left"></i> Cite&nbsp;&nbsp;&nbsp;&nbsp; </button><!----><!----><a _ngcontent-serverapp-c10="" class="btn btn-success" style="color: white; border: 1px solid transparent" target="_blank" title="Download Dataset as JSON" href="https://www.omicsdi.org/ws/dataset/arrayexpress-repository/E-GEOD-37196.json"><i _ngcontent-serverapp-c10="" aria-hidden="true" class="fa fa-file-text-o"></i><span _ngcontent-serverapp-c10=""> Json</span></a><a _ngcontent-serverapp-c10="" class="btn btn-warning" style="color: white; border: 1px solid transparent" target="_blank" title="Download Dataset as XML" href="https://www.omicsdi.org/ws/dataset/arrayexpress-repository/E-GEOD-37196.xml"><i _ngcontent-serverapp-c10="" aria-hidden="true" class="fa fa-file-code-o"></i><span _ngcontent-serverapp-c10=""> Xml</span></a></div></div></div></div></div><!----><div _ngcontent-serverapp-c10="" class="mt-card ng-star-inserted"><div _ngcontent-serverapp-c10="" class="mt-card-header" style="padding: 5px; border-bottom: 1px solid #dddddd"><div _ngcontent-serverapp-c10="" class="col-xs-6" style="display: flex; align-items: center; margin-top: 7px"><div _ngcontent-serverapp-c10="" style="width: 50px"><i _ngcontent-serverapp-c10="" class="fa fa-3x fa-folder-open" style="color: #0099cc; margin-left: 3px"></i></div><h3 _ngcontent-serverapp-c10="" class="mt-card-title" style="padding: 0; margin: 0 10px; font-weight: 500">Dataset's files</h3></div><div _ngcontent-serverapp-c10="" class="col-xs-6" style="display: flex; place-items: center; justify-content: flex-end"><div _ngcontent-serverapp-c10="" class="col-xs-8" style="float: right;"><mat-form-field _ngcontent-serverapp-c10="" appearance="outline" class="mat-form-field ng-tns-c14-1 mat-primary mat-form-field-type-mat-input mat-form-field-appearance-outline mat-form-field-can-float" id="file-filter"><div class="mat-form-field-wrapper"><div class="mat-form-field-flex"><!----><!----><div class="mat-form-field-outline ng-tns-c14-1 ng-star-inserted"><div class="mat-form-field-outline-start"></div><div class="mat-form-field-outline-gap"></div><div class="mat-form-field-outline-end"></div></div><div class="mat-form-field-outline mat-form-field-outline-thick ng-tns-c14-1 ng-star-inserted"><div class="mat-form-field-outline-start"></div><div class="mat-form-field-outline-gap"></div><div class="mat-form-field-outline-end"></div></div><!----><div class="mat-form-field-infix"><input _ngcontent-serverapp-c10="" class="mat-input-element mat-form-field-autofill-control cdk-text-field-autofill-monitored" matinput="" placeholder="Filter" id="mat-input-0" aria-invalid="false" aria-required="false"><span class="mat-form-field-label-wrapper"><!----></span></div><!----></div><!----><div class="mat-form-field-subscript-wrapper"><!----><!----><div class="mat-form-field-hint-wrapper ng-tns-c14-1 ng-trigger ng-trigger-transitionMessages ng-star-inserted" style="opacity:1;transform:translateY(0%);0:opacity;1:transform;"><!----><div class="mat-form-field-hint-spacer"></div></div></div></div></mat-form-field></div><span _ngcontent-serverapp-c10="" style="float: right">Source: <mat-form-field _ngcontent-serverapp-c10="" class="mat-form-field ng-tns-c14-2 mat-primary mat-form-field-type-mat-select mat-form-field-appearance-legacy mat-form-field-can-float mat-form-field-hide-placeholder ng-untouched ng-pristine ng-valid mat-form-field-should-float" style="width: 80px;"><div class="mat-form-field-wrapper"><div class="mat-form-field-flex"><!----><!----><div class="mat-form-field-infix"><mat-select _ngcontent-serverapp-c10="" class="mat-select ng-tns-c15-3 ng-untouched ng-pristine ng-valid ng-star-inserted" role="listbox" id="mat-select-0" tabindex="0" aria-required="false" aria-disabled="false" aria-invalid="false" aria-multiselectable="false"><div class="mat-select-trigger" aria-hidden="true" cdk-overlay-origin=""><div class="mat-select-value"><!----><!----><span class="mat-select-value-text ng-tns-c15-3 ng-star-inserted"><!----><span class="ng-tns-c15-3 ng-star-inserted">primary</span><!----></span></div><div class="mat-select-arrow-wrapper"><div class="mat-select-arrow"></div></div></div><!----></mat-select><span class="mat-form-field-label-wrapper"><!----></span></div><!----></div><!----><div class="mat-form-field-underline ng-tns-c14-2 ng-star-inserted"><span class="mat-form-field-ripple"></span></div><div class="mat-form-field-subscript-wrapper"><!----><!----><div class="mat-form-field-hint-wrapper ng-tns-c14-2 ng-trigger ng-trigger-transitionMessages ng-star-inserted" style="opacity:1;transform:translateY(0%);0:opacity;1:transform;"><!----><div class="mat-form-field-hint-spacer"></div></div></div></div></mat-form-field></span></div></div><div _ngcontent-serverapp-c10="" class="mt-card-body"><table _ngcontent-serverapp-c10="" class="mat-table" mat-table="" matsort="" role="grid"><thead role="rowgroup"><!----><tr _ngcontent-serverapp-c10="" class="mat-header-row ng-star-inserted" mat-header-row="" role="row"><!----><th _ngcontent-serverapp-c10="" class="mat-header-cell cdk-column-select mat-column-select ng-star-inserted" mat-header-cell="" role="columnheader"><mat-checkbox _ngcontent-serverapp-c10="" class="mat-checkbox mat-accent" id="mat-checkbox-1"><label class="mat-checkbox-layout" for="mat-checkbox-1-input"><div class="mat-checkbox-inner-container mat-checkbox-inner-container-no-side-margin"><input class="mat-checkbox-input cdk-visually-hidden" type="checkbox" id="mat-checkbox-1-input" tabindex="0" aria-label="deselect all" aria-checked="false"><div class="mat-checkbox-ripple mat-ripple" matripple=""><div class="mat-ripple-element mat-checkbox-persistent-ripple"></div></div><div class="mat-checkbox-frame"></div><div class="mat-checkbox-background"><svg xml:space="preserve" class="mat-checkbox-checkmark" focusable="false" version="1.1" viewBox="0 0 24 24"><path class="mat-checkbox-checkmark-path" d="M4.1,12.7 9,17.6 20.3,6.3" fill="none" stroke="white"></path></svg><div class="mat-checkbox-mixedmark"></div></div></div><span class="mat-checkbox-label"><span style="display:none">&nbsp;</span></span></label></mat-checkbox></th><th _ngcontent-serverapp-c10="" class="mat-header-cell ng-tns-c19-5 cdk-column-name mat-column-name ng-star-inserted" mat-header-cell="" mat-sort-header="" role="columnheader" style="padding-left: 10px"><div class="mat-sort-header-container"><button class="mat-sort-header-button" type="button" aria-label="Change sorting for name"> Name</button><!----><div class="mat-sort-header-arrow ng-trigger ng-trigger-arrowPosition ng-tns-c19-5 ng-star-inserted" style="transform:translateY(25%);opacity:0;0:transform;1:opacity;"><div class="mat-sort-header-stem"></div><div class="mat-sort-header-indicator ng-trigger ng-trigger-indicator" style="transform:translateY(0px);0:transform;"><div class="mat-sort-header-pointer-left ng-trigger ng-trigger-leftPointer" style="transform:rotate(-45deg);0:transform;"></div><div class="mat-sort-header-pointer-right ng-trigger ng-trigger-rightPointer" style="transform:rotate(45deg);0:transform;"></div><div class="mat-sort-header-pointer-middle"></div></div></div></div></th><th _ngcontent-serverapp-c10="" class="mat-header-cell ng-tns-c19-6 cdk-column-category mat-column-category ng-star-inserted" mat-header-cell="" mat-sort-header="" role="columnheader"><div class="mat-sort-header-container"><button class="mat-sort-header-button" type="button" aria-label="Change sorting for category"> Type</button><!----><div class="mat-sort-header-arrow ng-trigger ng-trigger-arrowPosition ng-tns-c19-6 ng-star-inserted" style="transform:translateY(25%);opacity:0;0:transform;1:opacity;"><div class="mat-sort-header-stem"></div><div class="mat-sort-header-indicator ng-trigger ng-trigger-indicator" style="transform:translateY(0px);0:transform;"><div class="mat-sort-header-pointer-left ng-trigger ng-trigger-leftPointer" style="transform:rotate(-45deg);0:transform;"></div><div class="mat-sort-header-pointer-right ng-trigger ng-trigger-rightPointer" style="transform:rotate(45deg);0:transform;"></div><div class="mat-sort-header-pointer-middle"></div></div></div></div></th><th _ngcontent-serverapp-c10="" class="mat-header-cell cdk-column-action mat-column-action ng-star-inserted" mat-header-cell="" role="columnheader"> Action</th><th _ngcontent-serverapp-c10="" class="mat-header-cell cdk-column-drs mat-column-drs ng-star-inserted" mat-header-cell="" role="columnheader"> DRS</th></tr></thead><tbody role="rowgroup"><!----><tr _ngcontent-serverapp-c10="" class="mat-row ng-star-inserted" mat-row="" role="row"><!----><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-select mat-column-select ng-star-inserted" mat-cell="" role="gridcell"><mat-checkbox _ngcontent-serverapp-c10="" class="mat-checkbox mat-accent" id="mat-checkbox-2"><label class="mat-checkbox-layout" for="mat-checkbox-2-input"><div class="mat-checkbox-inner-container mat-checkbox-inner-container-no-side-margin"><input class="mat-checkbox-input cdk-visually-hidden" type="checkbox" id="mat-checkbox-2-input" tabindex="0" aria-label="select row 1" aria-checked="false"><div class="mat-checkbox-ripple mat-ripple" matripple=""><div class="mat-ripple-element mat-checkbox-persistent-ripple"></div></div><div class="mat-checkbox-frame"></div><div class="mat-checkbox-background"><svg xml:space="preserve" class="mat-checkbox-checkmark" focusable="false" version="1.1" viewBox="0 0 24 24"><path class="mat-checkbox-checkmark-path" d="M4.1,12.7 9,17.6 20.3,6.3" fill="none" stroke="white"></path></svg><div class="mat-checkbox-mixedmark"></div></div></div><span class="mat-checkbox-label"><span style="display:none">&nbsp;</span></span></label></mat-checkbox></td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-name mat-column-name ng-star-inserted" mat-cell="" role="gridcell" style="padding-left: 10px"><a _ngcontent-serverapp-c10="" target="_blank" href="ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/GEOD/E-GEOD-37196/E-GEOD-37196.README.txt">E-GEOD-37196.README.txt</a></td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-category mat-column-category ng-star-inserted" mat-cell="" role="gridcell"> Txt </td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-action mat-column-action ng-star-inserted" mat-cell="" role="gridcell"><a _ngcontent-serverapp-c10="" style="border-bottom: none" target="_blank" href="ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/GEOD/E-GEOD-37196/E-GEOD-37196.README.txt"><i _ngcontent-serverapp-c10="" class="fa fa-cloud-download" style="font-size: 1.5em"></i></a></td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-drs mat-column-drs ng-star-inserted" mat-cell="" role="gridcell"><!----></td></tr><tr _ngcontent-serverapp-c10="" class="mat-row ng-star-inserted" mat-row="" role="row"><!----><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-select mat-column-select ng-star-inserted" mat-cell="" role="gridcell"><mat-checkbox _ngcontent-serverapp-c10="" class="mat-checkbox mat-accent" id="mat-checkbox-3"><label class="mat-checkbox-layout" for="mat-checkbox-3-input"><div class="mat-checkbox-inner-container mat-checkbox-inner-container-no-side-margin"><input class="mat-checkbox-input cdk-visually-hidden" type="checkbox" id="mat-checkbox-3-input" tabindex="0" aria-label="select row 2" aria-checked="false"><div class="mat-checkbox-ripple mat-ripple" matripple=""><div class="mat-ripple-element mat-checkbox-persistent-ripple"></div></div><div class="mat-checkbox-frame"></div><div class="mat-checkbox-background"><svg xml:space="preserve" class="mat-checkbox-checkmark" focusable="false" version="1.1" viewBox="0 0 24 24"><path class="mat-checkbox-checkmark-path" d="M4.1,12.7 9,17.6 20.3,6.3" fill="none" stroke="white"></path></svg><div class="mat-checkbox-mixedmark"></div></div></div><span class="mat-checkbox-label"><span style="display:none">&nbsp;</span></span></label></mat-checkbox></td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-name mat-column-name ng-star-inserted" mat-cell="" role="gridcell" style="padding-left: 10px"><a _ngcontent-serverapp-c10="" target="_blank" href="ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/GEOD/E-GEOD-37196/E-GEOD-37196.idf.txt">E-GEOD-37196.idf.txt</a></td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-category mat-column-category ng-star-inserted" mat-cell="" role="gridcell"> Idf </td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-action mat-column-action ng-star-inserted" mat-cell="" role="gridcell"><a _ngcontent-serverapp-c10="" style="border-bottom: none" target="_blank" href="ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/GEOD/E-GEOD-37196/E-GEOD-37196.idf.txt"><i _ngcontent-serverapp-c10="" class="fa fa-cloud-download" style="font-size: 1.5em"></i></a></td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-drs mat-column-drs ng-star-inserted" mat-cell="" role="gridcell"><!----></td></tr><tr _ngcontent-serverapp-c10="" class="mat-row ng-star-inserted" mat-row="" role="row"><!----><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-select mat-column-select ng-star-inserted" mat-cell="" role="gridcell"><mat-checkbox _ngcontent-serverapp-c10="" class="mat-checkbox mat-accent" id="mat-checkbox-4"><label class="mat-checkbox-layout" for="mat-checkbox-4-input"><div class="mat-checkbox-inner-container mat-checkbox-inner-container-no-side-margin"><input class="mat-checkbox-input cdk-visually-hidden" type="checkbox" id="mat-checkbox-4-input" tabindex="0" aria-label="select row 3" aria-checked="false"><div class="mat-checkbox-ripple mat-ripple" matripple=""><div class="mat-ripple-element mat-checkbox-persistent-ripple"></div></div><div class="mat-checkbox-frame"></div><div class="mat-checkbox-background"><svg xml:space="preserve" class="mat-checkbox-checkmark" focusable="false" version="1.1" viewBox="0 0 24 24"><path class="mat-checkbox-checkmark-path" d="M4.1,12.7 9,17.6 20.3,6.3" fill="none" stroke="white"></path></svg><div class="mat-checkbox-mixedmark"></div></div></div><span class="mat-checkbox-label"><span style="display:none">&nbsp;</span></span></label></mat-checkbox></td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-name mat-column-name ng-star-inserted" mat-cell="" role="gridcell" style="padding-left: 10px"><a _ngcontent-serverapp-c10="" target="_blank" href="ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/GEOD/E-GEOD-37196/E-GEOD-37196.processed.1.zip">E-GEOD-37196.processed.1.zip</a></td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-category mat-column-category ng-star-inserted" mat-cell="" role="gridcell"> Processed </td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-action mat-column-action ng-star-inserted" mat-cell="" role="gridcell"><a _ngcontent-serverapp-c10="" style="border-bottom: none" target="_blank" href="ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/GEOD/E-GEOD-37196/E-GEOD-37196.processed.1.zip"><i _ngcontent-serverapp-c10="" class="fa fa-cloud-download" style="font-size: 1.5em"></i></a></td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-drs mat-column-drs ng-star-inserted" mat-cell="" role="gridcell"><!----></td></tr><tr _ngcontent-serverapp-c10="" class="mat-row ng-star-inserted" mat-row="" role="row"><!----><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-select mat-column-select ng-star-inserted" mat-cell="" role="gridcell"><mat-checkbox _ngcontent-serverapp-c10="" class="mat-checkbox mat-accent" id="mat-checkbox-5"><label class="mat-checkbox-layout" for="mat-checkbox-5-input"><div class="mat-checkbox-inner-container mat-checkbox-inner-container-no-side-margin"><input class="mat-checkbox-input cdk-visually-hidden" type="checkbox" id="mat-checkbox-5-input" tabindex="0" aria-label="select row 4" aria-checked="false"><div class="mat-checkbox-ripple mat-ripple" matripple=""><div class="mat-ripple-element mat-checkbox-persistent-ripple"></div></div><div class="mat-checkbox-frame"></div><div class="mat-checkbox-background"><svg xml:space="preserve" class="mat-checkbox-checkmark" focusable="false" version="1.1" viewBox="0 0 24 24"><path class="mat-checkbox-checkmark-path" d="M4.1,12.7 9,17.6 20.3,6.3" fill="none" stroke="white"></path></svg><div class="mat-checkbox-mixedmark"></div></div></div><span class="mat-checkbox-label"><span style="display:none">&nbsp;</span></span></label></mat-checkbox></td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-name mat-column-name ng-star-inserted" mat-cell="" role="gridcell" style="padding-left: 10px"><a _ngcontent-serverapp-c10="" target="_blank" href="ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/GEOD/E-GEOD-37196/E-GEOD-37196.raw.1.zip">E-GEOD-37196.raw.1.zip</a></td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-category mat-column-category ng-star-inserted" mat-cell="" role="gridcell"> Raw </td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-action mat-column-action ng-star-inserted" mat-cell="" role="gridcell"><a _ngcontent-serverapp-c10="" style="border-bottom: none" target="_blank" href="ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/GEOD/E-GEOD-37196/E-GEOD-37196.raw.1.zip"><i _ngcontent-serverapp-c10="" class="fa fa-cloud-download" style="font-size: 1.5em"></i></a></td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-drs mat-column-drs ng-star-inserted" mat-cell="" role="gridcell"><!----></td></tr><tr _ngcontent-serverapp-c10="" class="mat-row ng-star-inserted" mat-row="" role="row"><!----><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-select mat-column-select ng-star-inserted" mat-cell="" role="gridcell"><mat-checkbox _ngcontent-serverapp-c10="" class="mat-checkbox mat-accent" id="mat-checkbox-6"><label class="mat-checkbox-layout" for="mat-checkbox-6-input"><div class="mat-checkbox-inner-container mat-checkbox-inner-container-no-side-margin"><input class="mat-checkbox-input cdk-visually-hidden" type="checkbox" id="mat-checkbox-6-input" tabindex="0" aria-label="select row 5" aria-checked="false"><div class="mat-checkbox-ripple mat-ripple" matripple=""><div class="mat-ripple-element mat-checkbox-persistent-ripple"></div></div><div class="mat-checkbox-frame"></div><div class="mat-checkbox-background"><svg xml:space="preserve" class="mat-checkbox-checkmark" focusable="false" version="1.1" viewBox="0 0 24 24"><path class="mat-checkbox-checkmark-path" d="M4.1,12.7 9,17.6 20.3,6.3" fill="none" stroke="white"></path></svg><div class="mat-checkbox-mixedmark"></div></div></div><span class="mat-checkbox-label"><span style="display:none">&nbsp;</span></span></label></mat-checkbox></td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-name mat-column-name ng-star-inserted" mat-cell="" role="gridcell" style="padding-left: 10px"><a _ngcontent-serverapp-c10="" target="_blank" href="ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/GEOD/E-GEOD-37196/E-GEOD-37196.raw.2.zip">E-GEOD-37196.raw.2.zip</a></td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-category mat-column-category ng-star-inserted" mat-cell="" role="gridcell"> Raw </td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-action mat-column-action ng-star-inserted" mat-cell="" role="gridcell"><a _ngcontent-serverapp-c10="" style="border-bottom: none" target="_blank" href="ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/GEOD/E-GEOD-37196/E-GEOD-37196.raw.2.zip"><i _ngcontent-serverapp-c10="" class="fa fa-cloud-download" style="font-size: 1.5em"></i></a></td><td _ngcontent-serverapp-c10="" class="mat-cell cdk-column-drs mat-column-drs ng-star-inserted" mat-cell="" role="gridcell"><!----></td></tr></tbody><tfoot role="rowgroup" style="display: none; bottom: 0px; position: sticky; z-index: 10;" class="mat-table-sticky"><!----></tfoot></table><table _ngcontent-serverapp-c10=""><tbody _ngcontent-serverapp-c10=""><tr _ngcontent-serverapp-c10=""><td _ngcontent-serverapp-c10=""><button _ngcontent-serverapp-c10="" aria-haspopup="true" mat-stroked-button="" style="float: left; margin-top: 15px" class="mat-stroked-button" disabled="disabled"><span class="mat-button-wrapper"><i _ngcontent-serverapp-c10="" aria-hidden="true" class="fa fa-cloud-upload"></i> Upload file(s) to Galaxy server </span><div class="mat-button-ripple mat-ripple" matripple=""></div><div class="mat-button-focus-overlay"></div></button></td><td _ngcontent-serverapp-c10="" style="padding-left: 10px; text-align: center;"><!----></td><td _ngcontent-serverapp-c10=""><mat-menu _ngcontent-serverapp-c10="" class="ng-tns-c7-4"><!----></mat-menu><mat-paginator _ngcontent-serverapp-c10="" class="mat-paginator" showfirstlastbuttons="" style="display: inherit; float: right"><div class="mat-paginator-outer-container"><div class="mat-paginator-container"><!----><div class="mat-paginator-page-size ng-star-inserted"><div class="mat-paginator-page-size-label">Items per page:</div><!----><mat-form-field class="mat-paginator-page-size-select mat-form-field ng-tns-c14-7 mat-primary mat-form-field-type-mat-select mat-form-field-appearance-legacy mat-form-field-can-float mat-form-field-hide-placeholder ng-star-inserted mat-form-field-should-float"><div class="mat-form-field-wrapper"><div class="mat-form-field-flex"><!----><!----><div class="mat-form-field-infix"><mat-select class="mat-select ng-tns-c15-8 ng-star-inserted" role="listbox" id="mat-select-1" tabindex="0" aria-label="Items per page:" aria-required="false" aria-disabled="false" aria-invalid="false" aria-multiselectable="false"><div class="mat-select-trigger" aria-hidden="true" cdk-overlay-origin=""><div class="mat-select-value"><!----><!----><span class="mat-select-value-text ng-tns-c15-8 ng-star-inserted"><!----><span class="ng-tns-c15-8 ng-star-inserted">5</span><!----></span></div><div class="mat-select-arrow-wrapper"><div class="mat-select-arrow"></div></div></div><!----></mat-select><span class="mat-form-field-label-wrapper"><!----></span></div><!----></div><!----><div class="mat-form-field-underline ng-tns-c14-7 ng-star-inserted"><span class="mat-form-field-ripple"></span></div><div class="mat-form-field-subscript-wrapper"><!----><!----><div class="mat-form-field-hint-wrapper ng-tns-c14-7 ng-trigger ng-trigger-transitionMessages ng-star-inserted" style="opacity:1;transform:translateY(0%);0:opacity;1:transform;"><!----><div class="mat-form-field-hint-spacer"></div></div></div></div></mat-form-field><!----></div><div class="mat-paginator-range-actions"><div class="mat-paginator-range-label">1 - 5 of 6</div><!----><button class="mat-paginator-navigation-first mat-icon-button ng-star-inserted" mat-icon-button="" type="button" aria-describedby="cdk-describedby-message-2" cdk-describedby-host="" aria-label="First page" disabled="disabled"><span class="mat-button-wrapper"><svg class="mat-paginator-icon" focusable="false" viewBox="0 0 24 24"><path d="M18.41 16.59L13.82 12l4.59-4.59L17 6l-6 6 6 6zM6 6h2v12H6z"></path></svg></span><div class="mat-button-ripple mat-ripple mat-button-ripple-round" matripple=""></div><div class="mat-button-focus-overlay"></div></button><!----><button class="mat-paginator-navigation-previous mat-icon-button" mat-icon-button="" type="button" aria-describedby="cdk-describedby-message-0" cdk-describedby-host="" aria-label="Previous page" disabled="disabled"><span class="mat-button-wrapper"><svg class="mat-paginator-icon" focusable="false" viewBox="0 0 24 24"><path d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z"></path></svg></span><div class="mat-button-ripple mat-ripple mat-button-ripple-round" matripple=""></div><div class="mat-button-focus-overlay"></div></button><button class="mat-paginator-navigation-next mat-icon-button" mat-icon-button="" type="button" aria-describedby="cdk-describedby-message-1" cdk-describedby-host="" aria-label="Next page"><span class="mat-button-wrapper"><svg class="mat-paginator-icon" focusable="false" viewBox="0 0 24 24"><path d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"></path></svg></span><div class="mat-button-ripple mat-ripple mat-button-ripple-round" matripple=""></div><div class="mat-button-focus-overlay"></div></button><!----><button class="mat-paginator-navigation-last mat-icon-button ng-star-inserted" mat-icon-button="" type="button" aria-describedby="cdk-describedby-message-3" cdk-describedby-host="" aria-label="Last page"><span class="mat-button-wrapper"><svg class="mat-paginator-icon" focusable="false" viewBox="0 0 24 24"><path d="M5.59 7.41L10.18 12l-4.59 4.59L7 18l6-6-6-6zM16 6h2v12h-2z"></path></svg></span><div class="mat-button-ripple mat-ripple mat-button-ripple-round" matripple=""></div><div class="mat-button-focus-overlay"></div></button><!----></div></div></div></mat-paginator></td></tr></tbody></table></div></div><app-publication _ngcontent-serverapp-c10="" _nghost-serverapp-c22=""><!----></app-publication><!----><app-similar-molecule _ngcontent-serverapp-c10="" _nghost-serverapp-c23="" class="ng-star-inserted"><!----></app-similar-molecule></div><div _ngcontent-serverapp-c10="" class="col-lg-3 col-md-3 col-sm-12 col-xs-12"><app-similar _ngcontent-serverapp-c10="" _nghost-serverapp-c24=""><!----><div _ngcontent-serverapp-c24="" class="mt-card ng-star-inserted" style="position: relative;"><div _ngcontent-serverapp-c24="" class="mt-card-header" style="padding: 20px 0; border-bottom: 1px solid #dddddd"><div _ngcontent-serverapp-c24="" class="col-xs-10" style="display: flex; align-items: center"><h3 _ngcontent-serverapp-c24="" class="mt-card-title" style="padding: 0; margin: 0 10px; font-weight: 500">Similar Datasets</h3></div></div><div _ngcontent-serverapp-c24="" class="mt-card-body" id="similar-datasets" style="padding: 0"><!----><div _ngcontent-serverapp-c24="" class="result animate-repeat ng-scope similar-item ng-star-inserted" style=""><div _ngcontent-serverapp-c24="" class="project-widget  list-group-item ddi-card-panel"><div _ngcontent-serverapp-c24="" class="project-widget-accession"><app-omics-image _ngcontent-serverapp-c24="" _nghost-serverapp-c12=""><img _ngcontent-serverapp-c12="" style="border-radius: 50%; width: 20px; height: 20px;" src="page_with_metadata_files/Multiomics.svg" alt="Transcriptomics,Multiomics" title="Transcriptomics,Multiomics"></app-omics-image><a _ngcontent-serverapp-c24="" class="align-justify ng-binding similar_title" href="https://www.omicsdi.org/dataset/arrayexpress-repository/E-GEOD-37195"> Gene expression profiling using exon arrays during interference with PPAR gamma signaling in thoracic aorta</a></div><div _ngcontent-serverapp-c24="" class="project-widget-paragraph hidden ng-binding"><b _ngcontent-serverapp-c24="">Project description:</b>Pharmacological
 activation of the transcription factor PPAR gamma lowers blood pressure
 and improves glucose tolerance in humans.  In contrast, naturally 
occurring mutations (e.g., P467L, V290M) in the ligand binding domain of
 PPAR gamma in humans leads to severe insulin resistance and early-onset
 hypertension. Experimental evidence, including whole genome expression 
profiling, suggests that these mutant versions of PPAR gamma act in a 
dominant negative manner. Because PPAR gamma is expressed in a variety 
of cell types and tissues, we generated a transgenic mouse model 
(SP467L) specifically targeting dominant negative PPAR gamma to the 
vascular smooth muscle cells in order to determine the action of PPAR 
gamma in the blood vessel independent of its systemic metabolic actions.
   In the data set provided herein, we examined the gene expression 
profile in thoracic aorta from SP467L mice and their control littermates
 using the Affymetrix mouse exon 1.0 ST array. We generated transgenic 
mice specifically targeting expression of mutant dominant negative human
 PPAR gamma (P467L) to vascular smooth muscle using a smooth 
muscle-specific promoter (smooth muscle myosin heavy chain or SMMHC).  
Thoracic aortas were isolated from 7 male transgenic  mice and 5 
non-transgenic littermate controls. Total RNA was prepared using 
conventional methods and quality was assessed using the Bioanalyzer 2100
 (Agilent Technologies). For the microarray hybridizations, each sample 
corresponded to aorta derived from one mouse. All procedures were 
conducted at the University of Iowa DNA Core facility using standard 
Affymetrix protocols. In brief, approximately 50 ng of total RNA was 
used as input to a two-step amplification procedure (NuGen, 
http://www.nugeninc.com/) to generate biotin-labeled RNA fragments for 
hybridization to the Affymetrix mouse exon 1.0 ST array. <!----></div><!----><div _ngcontent-serverapp-c24="" class="project-widget-paragraph ng-binding"><span _ngcontent-serverapp-c24="" class="ng-binding"><span _ngcontent-serverapp-c24="" class="similar_publicationDate">2012-10-03</span> | </span><span _ngcontent-serverapp-c24="" class="similar_id">E-GEOD-37195</span> | <span _ngcontent-serverapp-c24="" class="similar_source">ArrayExpress</span></div></div></div><div _ngcontent-serverapp-c24="" class="result animate-repeat ng-scope similar-item ng-star-inserted" style=""><div _ngcontent-serverapp-c24="" class="project-widget  list-group-item ddi-card-panel"><div _ngcontent-serverapp-c24="" class="project-widget-accession"><app-omics-image _ngcontent-serverapp-c24="" _nghost-serverapp-c12=""><img _ngcontent-serverapp-c12="" style="border-radius: 50%; width: 20px; height: 20px;" src="page_with_metadata_files/Transcriptomics.svg" alt="Transcriptomics" title="Transcriptomics"></app-omics-image><a _ngcontent-serverapp-c24="" class="align-justify ng-binding similar_title" href="https://www.omicsdi.org/dataset/arrayexpress-repository/E-GEOD-37194"> Gene expression profiling  during interference with PPAR gamma signaling in thoracic aorta</a></div><div _ngcontent-serverapp-c24="" class="project-widget-paragraph hidden ng-binding"><b _ngcontent-serverapp-c24="">Project description:</b>Pharmacological
 activation of the transcription factor PPAR gamma lowers blood pressure
 and improves glucose tolerance in humans.  In contrast, naturally 
occurring mutations (e.g., P467L, V290M) in the ligand binding domain of
 PPAR gamma in humans leads to severe insulin resistance and early-onset
 hypertension. Experimental evidence, including whole genome expression 
profiling, suggests that these mutant versions of PPAR gamma act in a 
dominant negative manner. Because PPAR gamma is expressed in a variety 
of cell types and tissues, we generated a transgenic mouse model 
(SP467L) specifically targeting dominant negative PPAR gamma to the 
vascular smooth muscle cells in order to determine the action of PPAR 
gamma in the blood vessel independent of its systemic metabolic actions.
   In the data set provided herein, we examined the gene expression 
profile in thoracic aorta from SP467L mice and their control littermates
 using the Affymetrix Mouse Genome 430 2.0 array. We generated 
transgenic mice specifically targeting expression of mutant dominant 
negative human PPAR gamma (P467L) to vascular smooth muscle using a 
smooth muscle-specific promoter (smooth muscle myosin heavy chain or 
SMMHC).  Thoracic aortas were isolated from male transgenic mice and 
corresponding non-transgenic littermate controls. Total RNA was prepared
 using conventional methods and quality was assessed using the 
Bioanalyzer 2100 (Agilent Technologies). For the microarray 
hybridizations, 2 samples from control mice and 3 samples from 
transgenic mice were used. Each sample was an independent biological 
replicate generated by pooling total RNA from 6-8 separate mouse aortas.
 All procedures were conducted at the University of Iowa DNA Core 
facility using standard Affymetrix protocols. In brief, approximately 
3-5 ug of total RNA was used as input to a one-step amplification 
procedure to generate biotin-labeled RNA fragments for hybridization to 
the Affymetrix Mouse Genome 430 2.0 array. <!----></div><!----><div _ngcontent-serverapp-c24="" class="project-widget-paragraph ng-binding"><span _ngcontent-serverapp-c24="" class="ng-binding"><span _ngcontent-serverapp-c24="" class="similar_publicationDate">2012-10-03</span> | </span><span _ngcontent-serverapp-c24="" class="similar_id">E-GEOD-37194</span> | <span _ngcontent-serverapp-c24="" class="similar_source">ArrayExpress</span></div></div></div><div _ngcontent-serverapp-c24="" class="result animate-repeat ng-scope similar-item ng-star-inserted" style=""><div _ngcontent-serverapp-c24="" class="project-widget  list-group-item ddi-card-panel"><div _ngcontent-serverapp-c24="" class="project-widget-accession"><app-omics-image _ngcontent-serverapp-c24="" _nghost-serverapp-c12=""><img _ngcontent-serverapp-c12="" style="border-radius: 50%; width: 20px; height: 20px;" src="page_with_metadata_files/Unknown.svg" alt="Unknown" title="Unknown"></app-omics-image><a _ngcontent-serverapp-c24="" class="align-justify ng-binding similar_title" href="https://www.omicsdi.org/dataset/biostudies/E-GEOD-37196"> Interference of PPAR gamma signaling in thoracic aorta</a></div><div _ngcontent-serverapp-c24="" class="project-widget-paragraph hidden ng-binding"><b _ngcontent-serverapp-c24="">Project description:</b>This
 SuperSeries is composed of the following subset Series: GSE37194: Gene 
expression profiling during interference with PPAR gamma signaling in 
thoracic aorta GSE37195: Gene expression profiling using exon arrays 
during interference with PPAR gamma signaling in thoracic aorta Refer to
 individual Series <!----></div><!----><div _ngcontent-serverapp-c24="" class="project-widget-paragraph ng-binding"><span _ngcontent-serverapp-c24="" class="ng-binding"><span _ngcontent-serverapp-c24="" class="similar_publicationDate"></span> | </span><span _ngcontent-serverapp-c24="" class="similar_id">E-GEOD-37196</span> | <span _ngcontent-serverapp-c24="" class="similar_source">BioStudies</span></div></div></div><div _ngcontent-serverapp-c24="" class="result animate-repeat ng-scope similar-item ng-star-inserted" style=""><div _ngcontent-serverapp-c24="" class="project-widget  list-group-item ddi-card-panel"><div _ngcontent-serverapp-c24="" class="project-widget-accession"><app-omics-image _ngcontent-serverapp-c24="" _nghost-serverapp-c12=""><img _ngcontent-serverapp-c12="" style="border-radius: 50%; width: 20px; height: 20px;" src="page_with_metadata_files/Unknown.svg" alt="Unknown" title="Unknown"></app-omics-image><a _ngcontent-serverapp-c24="" class="align-justify ng-binding similar_title" href="https://www.omicsdi.org/dataset/biostudies/E-GEOD-37195"> Gene expression profiling using exon arrays during interference with PPAR gamma signaling in thoracic aorta</a></div><div _ngcontent-serverapp-c24="" class="project-widget-paragraph hidden ng-binding"><b _ngcontent-serverapp-c24="">Project description:</b>Pharmacological
 activation of the transcription factor PPAR gamma lowers blood pressure
 and improves glucose tolerance in humans.  In contrast, naturally 
occurring mutations (e.g., P467L, V290M) in the ligand binding domain of
 PPAR gamma in humans leads to severe insulin resistance and early-onset
 hypertension. Experimental evidence, including whole genome expression 
profiling, suggests that these mutant versions of PPAR gamma act in a 
dominant negative manner. Because PPAR gamma is expressed in a variety 
of cell types and tissues, we generated a transgenic mouse model 
(SP467L) specifically targeting dominant negative PPAR gamma to the 
vascular smooth muscle cells in order to determine the action of PPAR 
gamma in the blood vessel independent of its systemic metabolic actions.
   In the data set provided herein, we examined the gene expression 
profile in thoracic aorta from SP467L mice and their control littermates
 using the Affymetrix mouse exon 1.0 ST array. We generated transgenic 
mice specifically targeting expression of mutant dominant negative human
 PPAR gamma (P467L) to vascular smooth muscle using a smooth 
muscle-specific promoter (smooth muscle myosin heavy chain or SMMHC).  
Thoracic aortas were isolated from 7 male transgenic  mice and 5 
non-transgenic littermate controls. Total RNA was prepared using 
conventional methods and quality was assessed using the Bioanalyzer 2100
 (Agilent Technologies). For the microarray hybridizations, each sample 
corresponded to aorta derived from one mouse. All procedures were 
conducted at the University of Iowa DNA Core facility using standard 
Affymetrix protocols. In brief, approximately 50 ng of total RNA was 
used as input to a two-step amplification procedure (NuGen, 
http://www.nugeninc.com/) to generate biotin-labeled RNA fragments for 
hybridization to the Affymetrix mouse exon 1.0 ST array. <!----></div><!----><div _ngcontent-serverapp-c24="" class="project-widget-paragraph ng-binding"><span _ngcontent-serverapp-c24="" class="ng-binding"><span _ngcontent-serverapp-c24="" class="similar_publicationDate"></span> | </span><span _ngcontent-serverapp-c24="" class="similar_id">E-GEOD-37195</span> | <span _ngcontent-serverapp-c24="" class="similar_source">BioStudies</span></div></div></div><div _ngcontent-serverapp-c24="" class="result animate-repeat ng-scope similar-item ng-star-inserted" style=""><div _ngcontent-serverapp-c24="" class="project-widget  list-group-item ddi-card-panel"><div _ngcontent-serverapp-c24="" class="project-widget-accession"><app-omics-image _ngcontent-serverapp-c24="" _nghost-serverapp-c12=""><img _ngcontent-serverapp-c12="" style="border-radius: 50%; width: 20px; height: 20px;" src="page_with_metadata_files/Unknown.svg" alt="Unknown" title="Unknown"></app-omics-image><a _ngcontent-serverapp-c24="" class="align-justify ng-binding similar_title" href="https://www.omicsdi.org/dataset/biostudies/E-GEOD-37194"> Gene expression profiling  during interference with PPAR gamma signaling in thoracic aorta</a></div><div _ngcontent-serverapp-c24="" class="project-widget-paragraph hidden ng-binding"><b _ngcontent-serverapp-c24="">Project description:</b>Pharmacological
 activation of the transcription factor PPAR gamma lowers blood pressure
 and improves glucose tolerance in humans.  In contrast, naturally 
occurring mutations (e.g., P467L, V290M) in the ligand binding domain of
 PPAR gamma in humans leads to severe insulin resistance and early-onset
 hypertension. Experimental evidence, including whole genome expression 
profiling, suggests that these mutant versions of PPAR gamma act in a 
dominant negative manner. Because PPAR gamma is expressed in a variety 
of cell types and tissues, we generated a transgenic mouse model 
(SP467L) specifically targeting dominant negative PPAR gamma to the 
vascular smooth muscle cells in order to determine the action of PPAR 
gamma in the blood vessel independent of its systemic metabolic actions.
   In the data set provided herein, we examined the gene expression 
profile in thoracic aorta from SP467L mice and their control littermates
 using the Affymetrix Mouse Genome 430 2.0 array. We generated 
transgenic mice specifically targeting expression of mutant dominant 
negative human PPAR gamma (P467L) to vascular smooth muscle using a 
smooth muscle-specific promoter (smooth muscle myosin heavy chain or 
SMMHC).  Thoracic aortas were isolated from male transgenic mice and 
corresponding non-transgenic littermate controls. Total RNA was prepared
 using conventional methods and quality was assessed using the 
Bioanalyzer 2100 (Agilent Technologies). For the microarray 
hybridizations, 2 samples from control mice and 3 samples from 
transgenic mice were used. Each sample was an independent biological 
replicate generated by pooling total RNA from 6-8 separate mouse aortas.
 All procedures were conducted at the University of Iowa DNA Core 
facility using standard Affymetrix protocols. In brief, approximately 
3-5 ug of total RNA was used as input to a one-step amplification 
procedure to generate biotin-labeled RNA fragments for hybridization to 
the Affymetrix Mouse Genome 430 2.0 array. <!----></div><!----><div _ngcontent-serverapp-c24="" class="project-widget-paragraph ng-binding"><span _ngcontent-serverapp-c24="" class="ng-binding"><span _ngcontent-serverapp-c24="" class="similar_publicationDate"></span> | </span><span _ngcontent-serverapp-c24="" class="similar_id">E-GEOD-37194</span> | <span _ngcontent-serverapp-c24="" class="similar_source">BioStudies</span></div></div></div><div _ngcontent-serverapp-c24="" class="result animate-repeat ng-scope similar-item ng-star-inserted" style=""><div _ngcontent-serverapp-c24="" class="project-widget  list-group-item ddi-card-panel"><div _ngcontent-serverapp-c24="" class="project-widget-accession"><app-omics-image _ngcontent-serverapp-c24="" _nghost-serverapp-c12=""><img _ngcontent-serverapp-c12="" style="border-radius: 50%; width: 20px; height: 20px;" src="page_with_metadata_files/Unknown.svg" alt="Unknown" title="Unknown"></app-omics-image><a _ngcontent-serverapp-c24="" class="align-justify ng-binding similar_title" href="https://www.omicsdi.org/dataset/geo/GSE37194"> Gene expression profiling during interference with PPAR gamma signaling in thoracic aorta</a></div><div _ngcontent-serverapp-c24="" class="project-widget-paragraph hidden ng-binding"><b _ngcontent-serverapp-c24="">Project description:</b>Pharmacological
 activation of the transcription factor PPAR gamma lowers blood pressure
 and improves glucose tolerance in humans. In contrast, naturally 
occurring mutations (e.g., P467L, V290M) in the ligand binding domain of
 PPAR gamma in humans leads to severe insulin resistance and early-onset
 hypertension. Experimental evidence, including whole genome expression 
profiling, suggests that these mutant versions of PPAR gamma act in a 
dominant negative manner. Because PPAR gamma is expressed in a variety 
of cell types and tissues, we generated a transgenic mouse model 
(SP467L) specifically targeting dominant negative PPAR gamma to the 
vascular smooth muscle cells in order to determine the action of PPAR 
gamma in the blood vessel independent of its systemic metabolic actions.
 In the data set provided herein, we examined the gene expression 
profile in thoracic aorta from SP467L mice and their control littermates
 using the Affymetrix Mouse Genome 430 2.0 array. <!----></div><!----><div _ngcontent-serverapp-c24="" class="project-widget-paragraph ng-binding"><span _ngcontent-serverapp-c24="" class="ng-binding"><span _ngcontent-serverapp-c24="" class="similar_publicationDate">2012-10-03</span> | </span><span _ngcontent-serverapp-c24="" class="similar_id">GSE37194</span> | <span _ngcontent-serverapp-c24="" class="similar_source">GEO</span></div></div></div><div _ngcontent-serverapp-c24="" class="result animate-repeat ng-scope similar-item ng-star-inserted" style=""><div _ngcontent-serverapp-c24="" class="project-widget  list-group-item ddi-card-panel"><div _ngcontent-serverapp-c24="" class="project-widget-accession"><app-omics-image _ngcontent-serverapp-c24="" _nghost-serverapp-c12=""><img _ngcontent-serverapp-c12="" style="border-radius: 50%; width: 20px; height: 20px;" src="page_with_metadata_files/Unknown.svg" alt="Unknown" title="Unknown"></app-omics-image><a _ngcontent-serverapp-c24="" class="align-justify ng-binding similar_title" href="https://www.omicsdi.org/dataset/geo/GSE37195"> Gene expression profiling using exon arrays during interference with PPAR gamma signaling in thoracic aorta</a></div><div _ngcontent-serverapp-c24="" class="project-widget-paragraph hidden ng-binding"><b _ngcontent-serverapp-c24="">Project description:</b>Pharmacological
 activation of the transcription factor PPAR gamma lowers blood pressure
 and improves glucose tolerance in humans. In contrast, naturally 
occurring mutations (e.g., P467L, V290M) in the ligand binding domain of
 PPAR gamma in humans leads to severe insulin resistance and early-onset
 hypertension. Experimental evidence, including whole genome expression 
profiling, suggests that these mutant versions of PPAR gamma act in a 
dominant negative manner. Because PPAR gamma is expressed in a variety 
of cell types and tissues, we generated a transgenic mouse model 
(SP467L) specifically targeting dominant negative PPAR gamma to the 
vascular smooth muscle cells in order to determine the action of PPAR 
gamma in the blood vessel independent of its systemic metabolic actions.
 In the data set provided herein, we examined the gene expression 
profile in thoracic aorta from SP467L mice and their control littermates
 using the Affymetrix mouse exon 1.0 ST array. <!----></div><!----><div _ngcontent-serverapp-c24="" class="project-widget-paragraph ng-binding"><span _ngcontent-serverapp-c24="" class="ng-binding"><span _ngcontent-serverapp-c24="" class="similar_publicationDate">2012-10-03</span> | </span><span _ngcontent-serverapp-c24="" class="similar_id">GSE37195</span> | <span _ngcontent-serverapp-c24="" class="similar_source">GEO</span></div></div></div><div _ngcontent-serverapp-c24="" class="result animate-repeat ng-scope similar-item ng-star-inserted" style=""><div _ngcontent-serverapp-c24="" class="project-widget  list-group-item ddi-card-panel"><div _ngcontent-serverapp-c24="" class="project-widget-accession"><app-omics-image _ngcontent-serverapp-c24="" _nghost-serverapp-c12=""><img _ngcontent-serverapp-c12="" style="border-radius: 50%; width: 20px; height: 20px;" src="page_with_metadata_files/Multiomics.svg" alt="Unknown,Multiomics" title="Unknown,Multiomics"></app-omics-image><a _ngcontent-serverapp-c24="" class="align-justify ng-binding similar_title" href="https://www.omicsdi.org/dataset/atlas-experiments/E-GEOD-37195"> Gene expression profiling using exon arrays during interference with PPAR gamma signaling in thoracic aorta</a></div><div _ngcontent-serverapp-c24="" class="project-widget-paragraph hidden ng-binding"><b _ngcontent-serverapp-c24="">Project description:</b>Pharmacological
 activation of the transcription factor PPAR gamma lowers blood pressure
 and improves glucose tolerance in humans.  In contrast, naturally 
occurring mutations (e.g., P467L, V290M) in the ligand binding domain of
 PPAR gamma in humans leads to severe insulin resistance and early-onset
 hypertension. Experimental evidence, including whole genome expression 
profiling, suggests that these mutant versions of PPAR gamma act in a 
dominant negative manner. Because PPAR gamma is expressed in a variety 
of cell types and tissues, we generated a transgenic mouse model 
(SP467L) specifically targeting dominant negative PPAR gamma to the 
vascular smooth muscle cells in order to determine the action of PPAR 
gamma in the blood vessel independent of its systemic metabolic actions.
   In the data set provided herein, we examined the gene expression 
profile in thoracic aorta from SP467L mice and their control littermates
 using the Affymetrix mouse exon 1.0 ST array. We generated transgenic 
mice specifically targeting expression of mutant dominant negative human
 PPAR gamma (P467L) to vascular smooth muscle using a smooth 
muscle-specific promoter (smooth muscle myosin heavy chain or SMMHC).  
Thoracic aortas were isolated from 7 male transgenic  mice and 5 
non-transgenic littermate controls. Total RNA was prepared using 
conventional methods and quality was assessed using the Bioanalyzer 2100
 (Agilent Technologies). For the microarray hybridizations, each sample 
corresponded to aorta derived from one mouse. All procedures were 
conducted at the University of Iowa DNA Core facility using standard 
Affymetrix protocols. In brief, approximately 50 ng of total RNA was 
used as input to a two-step amplification procedure (NuGen, 
http://www.nugeninc.com/) to generate biotin-labeled RNA fragments for 
hybridization to the Affymetrix mouse exon 1.0 ST array. <!----></div><!----><div _ngcontent-serverapp-c24="" class="project-widget-paragraph ng-binding"><span _ngcontent-serverapp-c24="" class="ng-binding"><span _ngcontent-serverapp-c24="" class="similar_publicationDate">2015-09-28</span> | </span><span _ngcontent-serverapp-c24="" class="similar_id">E-GEOD-37195</span> | <span _ngcontent-serverapp-c24="" class="similar_source">ExpressionAtlas</span></div></div></div><div _ngcontent-serverapp-c24="" class="result animate-repeat ng-scope similar-item ng-star-inserted" style=""><div _ngcontent-serverapp-c24="" class="project-widget  list-group-item ddi-card-panel"><div _ngcontent-serverapp-c24="" class="project-widget-accession"><app-omics-image _ngcontent-serverapp-c24="" _nghost-serverapp-c12=""><img _ngcontent-serverapp-c12="" style="border-radius: 50%; width: 20px; height: 20px;" src="page_with_metadata_files/Multiomics.svg" alt="Unknown,Multiomics" title="Unknown,Multiomics"></app-omics-image><a _ngcontent-serverapp-c24="" class="align-justify ng-binding similar_title" href="https://www.omicsdi.org/dataset/omics_ena_project/PRJNA158553"> </a></div><div _ngcontent-serverapp-c24="" class="project-widget-paragraph hidden ng-binding"><b _ngcontent-serverapp-c24="">Project description:</b>Interference of PPAR gamma signaling in thoracic aorta <!----></div><!----><div _ngcontent-serverapp-c24="" class="project-widget-paragraph ng-binding"><span _ngcontent-serverapp-c24="" class="ng-binding"><span _ngcontent-serverapp-c24="" class="similar_publicationDate"></span> | </span><span _ngcontent-serverapp-c24="" class="similar_id">PRJNA158553</span> | <span _ngcontent-serverapp-c24="" class="similar_source">ENA</span></div></div></div><div _ngcontent-serverapp-c24="" class="result animate-repeat ng-scope similar-item ng-star-inserted" style=""><div _ngcontent-serverapp-c24="" class="project-widget  list-group-item ddi-card-panel"><div _ngcontent-serverapp-c24="" class="project-widget-accession"><app-omics-image _ngcontent-serverapp-c24="" _nghost-serverapp-c12=""><img _ngcontent-serverapp-c12="" style="border-radius: 50%; width: 20px; height: 20px;" src="page_with_metadata_files/Multiomics.svg" alt="Transcriptomics,Multiomics" title="Transcriptomics,Multiomics"></app-omics-image><a _ngcontent-serverapp-c24="" class="align-justify ng-binding similar_title" href="https://www.omicsdi.org/dataset/arrayexpress-repository/E-GEOD-36482"> Interference with PPAR gamma signaling in mesenteric artery</a></div><div _ngcontent-serverapp-c24="" class="project-widget-paragraph hidden ng-binding"><b _ngcontent-serverapp-c24="">Project description:</b>Pharmacological
 activation of the transcription factor PPAR gamma lowers blood pressure
 and improves glucose tolerance in humans.  In contrast, naturally 
occurring mutations (e.g., P467L, V290M) in the ligand binding domain of
 PPAR gamma in humans leads to severe insulin resistance and early-onset
 hypertension. Experimental evidence, including whole genome expression 
profiling, suggests that these mutant versions of PPAR gamma act in a 
dominant negative manner. Because PPAR gamma is expressed in a variety 
of cell types and tissues, we generated a transgenic mouse model 
(SP467L) specifically targeting dominant negative PPAR gamma to the 
vascular smooth muscle cells in order to determine the action of PPAR 
gamma in the blood vessel independent of its systemic metabolic actions.
   In the data set provided herein, we examined the gene expression 
profile in mesenteric vessels from SP467L mice and their control 
littermates using  the Affymetrix mouse exon 1.0 ST array. We generated 
transgenic mice specifically targeting expression of mutant dominant 
negative human PPAR gamma (P467L) to vascular smooth muscle using a 
smooth muscle-specific promoter (smooth muscle myosin heavy chain or 
SMMHC).  Mesenteric arteries were isolated from 3 male transgenic mice 
and 4 non-transgenic littermate controls. Total RNA was prepared using 
conventional methods and quality was assessed using the Bioanalyzer 2100
 (Agilent Technologies). For the microarray hybridizations, each sample 
corresponded to mesenteric artery derived from one mouse. All procedures
 were conducted at the University of Iowa DNA Core facility using 
standard Affymetrix protocols. In brief, approximately 50 ng of total 
RNA was used as input to a two-step amplification procedure (NuGen, 
http://www.nugeninc.com/) to generate biotin-labeled RNA fragments for 
hybridization to the Affymetrix mouse exon 1.0 ST array. <!----></div><!----><div _ngcontent-serverapp-c24="" class="project-widget-paragraph ng-binding"><span _ngcontent-serverapp-c24="" class="ng-binding"><span _ngcontent-serverapp-c24="" class="similar_publicationDate">2012-09-24</span> | </span><span _ngcontent-serverapp-c24="" class="similar_id">E-GEOD-36482</span> | <span _ngcontent-serverapp-c24="" class="similar_source">ArrayExpress</span></div></div></div><div _ngcontent-serverapp-c24="" class="input-btn center" style="width: 102px; margin: 10px auto;"><button _ngcontent-serverapp-c24="" class="btn btn-default  ng-binding" type="submit"><i _ngcontent-serverapp-c24="" class="fa fa-search"></i>Load More </button></div></div></div></app-similar></div></div><!----></div><!----><ngx-json-ld _ngcontent-serverapp-c10="" class="ng-star-inserted"><script type="application/ld+json">{
  "mainEntity": {
    "creator": [
      {
        "name": "Henry Keen",
        "@type": "Person"
      }
    ],
    "keywords": "transcription profiling by array",
    "citation": {
      "name": "Interference of PPAR gamma signaling in thoracic aorta",
      "url": "http://www.omicsdi.org/dataset/arrayexpress-repository/E-GEOD-37196",
      "author": {
        "name": "Henry Keen",
        "@type": "Person"
      },
      "publisher": {
        "name": "ARRAYEXPRESS-REPOSITORY",
        "@type": "Organization"
      },
      "@type": "CreativeWork"
    },
    "@type": "Dataset",
    "name": "Interference of PPAR gamma signaling in thoracic aorta",
    "variableMeasured": "Transcriptomics",
    "description": "This SuperSeries is composed of the following subset Series: GSE37194: Gene expression profiling during interference with PPAR gamma signaling in thoracic aorta GSE37195: Gene expression profiling using exon arrays during interference with PPAR gamma signaling in thoracic aorta Refer to individual Series",
    "distribution": {
      "contentUrl": "http://www.omicsdi.org/ws/dataset/arrayexpress-repository/E-GEOD-37196.xml",
      "@type": "DataDownload",
      "encodingFormat": "XML"
    },
    "sameAs": "https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-37196"
  },
  "breadcrumb": {
    "itemListElement": [
      {
        "item": "https://www.omicsdi.org",
        "@type": "ListItem",
        "name": "OmicsDI",
        "position": 1
      },
      {
        "item": "https://www.omicsdi.org/search?q=(repository:%22ArrayExpress%22)",
        "@type": "ListItem",
        "name": "ArrayExpress",
        "position": 2
      },
      {
        "item": "http://www.omicsdi.org/dataset/arrayexpress-repository/E-GEOD-37196",
        "@type": "ListItem",
        "name": "E-GEOD-37196",
        "position": 3
      }
    ],
    "@type": "BreadcrumbList"
  },
  "@type": "ItemPage",
  "@context": "http://schema.org"
}</script></ngx-json-ld></app-dataset></div><div _ngcontent-serverapp-c0="" data-color="none" data-use-basic-styles="true" id="elixir-banner"><div _ngcontent-serverapp-c0="" class="elixir-ribbon" id="elixir-ribbon"><div _ngcontent-serverapp-c0="" class="row"><div _ngcontent-serverapp-c0="" class="column"><a _ngcontent-serverapp-c0="" href="https://www.elixir-europe.org/platforms/interoperability"><div _ngcontent-serverapp-c0="" class="elixir-logo-kite"></div><h5 _ngcontent-serverapp-c0=""><span _ngcontent-serverapp-c0="" class="elixir-banner-name">OmicsDI</span> is part of the ELIXIR infrastructure </h5><div _ngcontent-serverapp-c0="" id="elixir-banner-info"><small _ngcontent-serverapp-c0=""><span _ngcontent-serverapp-c0="" class="elixir-banner-description">OmicsDI is an Elixir interoperability service. </span><span _ngcontent-serverapp-c0="" class="readmore">Learn more </span></small></div></a></div></div></div></div><footer _ngcontent-serverapp-c0=""><div _ngcontent-serverapp-c0="" class="row" id="global-footer" style="border-top:none;"><nav _ngcontent-serverapp-c0="" class="col-md-12" id="global-nav-expanded"><!----><!----><div _ngcontent-serverapp-c0="" class="col-md-2 col-sm-12 col-xs-12 ng-star-inserted"><a _ngcontent-serverapp-c0="" class="no-icon" href="https://elixir-europe.org/platforms/interoperability" target="_blank" title="Elixir Interoperability"><img _ngcontent-serverapp-c0="" alt="EMBL-EBI logo" class="img_at_footer ebi-img" src="page_with_metadata_files/elixir1.png" style="width:150px; margin-top:9px"></a><a _ngcontent-serverapp-c0="" class="no-icon" href="http://www.ebi.ac.uk/" target="_blank" title="European Bioinformatics Institute homepage"><img _ngcontent-serverapp-c0="" alt="EMBL-EBI logo" class="img_at_footer ebi-img" src="page_with_metadata_files/embl-ebi.jpg" style="width:150px; margin-top:9px"></a><a _ngcontent-serverapp-c0="" class="no-icon" href="http://www.heartbd2k.org/" target="_blank" title="BD2K"><img _ngcontent-serverapp-c0="" alt="" class="img_at_footer img" src="page_with_metadata_files/bd2k.png" style="width:150px; margin-top:9px"></a><a _ngcontent-serverapp-c0="" href="http://metabolomexchange.org/" target="_blank"><img _ngcontent-serverapp-c0="" class="img_at_footer" src="page_with_metadata_files/metabolome.png" style="width:150px; margin-top:9px"></a><a _ngcontent-serverapp-c0="" class="img_at_footer" href="http://www.proteomexchange.org/" target="_blank"><img _ngcontent-serverapp-c0="" src="page_with_metadata_files/proteome.png" style="width:150px; margin-top:9px"></a><a _ngcontent-serverapp-c0="" class="img_at_footer" href="https://www.ebi.ac.uk/ega/" target="_blank"><img _ngcontent-serverapp-c0="" src="page_with_metadata_files/ega_phenome_logo.jpg" style="width:150px; margin-top:9px"></a></div><div _ngcontent-serverapp-c0="" class="col-md-8 col-sm-8 col-xs-12"><h3 _ngcontent-serverapp-c0="" class="about"> OmicsDI Databases </h3><!----><!----><div _ngcontent-serverapp-c0="" class="col-md-3 col-sm-4 col-xs-4 ng-star-inserted"><ul _ngcontent-serverapp-c0=""><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="http://www.ebi.ac.uk/pride/archive/" target="_blank">PRIDE</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="http://www.peptideatlas.org/" target="_blank">PeptideAtlas</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="https://massive.ucsd.edu/ProteoSAFe/datasets.jsp" target="_blank">MassIVE</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="https://jpostdb.org/" target="_blank">JPOST Repository</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="http://physiomeproject.org/" target="_blank">Physiome Model Repository</a></li></ul></div><!----><div _ngcontent-serverapp-c0="" class="col-md-3 col-sm-4 col-xs-4 ng-star-inserted"><ul _ngcontent-serverapp-c0=""><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="https://www.ebi.ac.uk/ega/" target="_blank">EGA</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="https://www.ebi.ac.uk/eva/" target="_blank">EVA</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="https://www.ebi.ac.uk/ena/" target="_blank">ENA</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="http://lincsportal.ccs.miami.edu/dcic-portal/" target="_blank">LINCS</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="https://pax-db.org/" target="_blank">PAXDB</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="https://cellcollective.org/" target="_blank">Cell Collective</a></li></ul></div><!----><div _ngcontent-serverapp-c0="" class="col-md-3 col-sm-4 col-xs-4 ng-star-inserted"><ul _ngcontent-serverapp-c0=""><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="http://www.ebi.ac.uk/metabolights/" target="_blank">MetaboLights</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="http://www.metabolomicsworkbench.org/" target="_blank">Metabolomics Workbench</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="https://www.metabolome-express.org/" target="_blank">MetabolomeExpress</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="https://gnps.ucsd.edu/" target="_blank">GNPS</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="https://www.ebi.ac.uk/biomodels" target="_blank">BioModels</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="https://fairdomhub.org/" target="_blank">FAIRDOMHub</a></li></ul></div><!----><div _ngcontent-serverapp-c0="" class="col-md-3 col-sm-4 col-xs-4 ng-star-inserted"><ul _ngcontent-serverapp-c0=""><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="https://www.ebi.ac.uk/arrayexpress/" target="_blank">ArrayExpress</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="https://www.ncbi.nlm.nih.gov/gap" target="_blank">dbGaP</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="http://www.ebi.ac.uk/gxa/home" target="_blank">ExpressionAtlas</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="https://www.ncbi.nlm.nih.gov/geo/" target="_blank">GEO</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="https://www.biosino.org/node/" target="_blank">NODE</a></li></ul></div></div><div _ngcontent-serverapp-c0="" class="col-md-2 col-sm-4 col-xs-12"><h3 _ngcontent-serverapp-c0="" class="about">Information</h3><ul _ngcontent-serverapp-c0=""><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" href="https://www.omicsdi.org/database">Databases</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" href="http://blog.omicsdi.org/">Help</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" href="http://blog.omicsdi.org/post/introduction-api/">API</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" href="http://www.ebi.ac.uk/support/index.php?query=omics-di">Contact us</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="https://github.com/OmicsDI/" target="_blank">Code on GitHub</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" class="no-icon" href="http://www.ebi.ac.uk/about/terms-of-use" target="_blank">Terms of use</a></li><li _ngcontent-serverapp-c0=""><a _ngcontent-serverapp-c0="" href="https://www.ebi.ac.uk/biostudies/">Submit Data</a></li></ul></div></nav></div></footer></div><simple-notifications _ngcontent-serverapp-c0=""><div class="simple-notification-wrapper bottom right"><!----></div></simple-notifications></app-root>
    <!-- Google Analytics -->
    <script>
        window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
        ga('create', 'UA-70417662-2', 'auto');
        ga('send', 'pageview');
    </script>
    <script async="" src="page_with_metadata_files/google-analytics_analytics.js"></script>
    <!-- End Google Analytics -->
<script type="text/javascript" src="page_with_metadata_files/runtime.7b6ccbe810cddbb76a94.js"></script><script type="text/javascript" src="page_with_metadata_files/polyfills.af82295a1475f52a2d32.js"></script><script type="text/javascript" src="page_with_metadata_files/main.c2b8230bd8d22c750657.js"></script>
<script>
    if (window.location.href.includes("rosette")) {    
        // Remove Node.js scripts
        var scripts = document.getElementsByTagName('script');
        var i = scripts.length;
        while (i--) {        
            if (scripts[i].type == "text/javascript") {
                scripts[i].parentNode.removeChild(scripts[i]);
            }
        }
    }
</script>

<script id="serverApp-state" type="application/json">{&q;http_requests:__BASE_URL_0__database/all?&q;:{&q;body&q;:[{&q;databaseName&q;:&q;PeptideAtlas&q;,&q;title&q;:&q;PeptideAtlas Database&q;,&q;sourceUrl&q;:&q;http://www.peptideatlas.org/&q;,&q;imgAlt&q;:&q;RefSeq logo&q;,&q;repository&q;:&q;PeptideAtlas&q;,&q;source&q;:&q;peptide_atlas&q;,&q;lastUpdated&q;:&q;2016-06-21&q;,&q;orcidName&q;:&q;PEPTIDEATLAS&q;,&q;domain&q;:&q;PeptideAtlas&q;,&q;description&q;:&q;is a multi-organism, publicly accessible compendium of peptides identified in a large set of tandem mass spectrometry proteomics experiments.\n&q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:null},{&q;databaseName&q;:&q;MetabolomeExpress&q;,&q;title&q;:&q;MetabolomeExpress Database&q;,&q;sourceUrl&q;:&q;https://www.metabolome-express.org/&q;,&q;imgAlt&q;:&q;MetabolomeExpress logo&q;,&q;repository&q;:&q;MetabolomeExpress&q;,&q;source&q;:&q;metabolome_express&q;,&q;lastUpdated&q;:&q;2015-07-29&q;,&q;orcidName&q;:&q;METABOLOMEEXPRESS&q;,&q;domain&q;:&q;MetabolomeExpress&q;,&q;description&q;:&q;A public place to process, interpret and share GC/MS metabolomics datasets.\n\n&q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:null},{&q;databaseName&q;:&q;NODE&q;,&q;title&q;:&q;NODE Database&q;,&q;sourceUrl&q;:&q;https://www.biosino.org/node/&q;,&q;imgAlt&q;:&q;NODE logo&q;,&q;repository&q;:&q;NODE&q;,&q;source&q;:&q;node&q;,&q;lastUpdated&q;:&q;2021-01-11&q;,&q;orcidName&q;:&q;NODE&q;,&q;domain&q;:&q;NODE&q;,&q;description&q;:&q;The National Omics Data Encyclopedia (NODE) stores raw sequence data from next-generation sequencing technologies including 454, IonTorrent, Illumina, SOLiD, Helicos and Complete Genomics.&q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:[&q;NODE&q;]},{&q;databaseName&q;:&q;MetabolomicsWorkbench&q;,&q;title&q;:&q;Metabolomics Workbench Database&q;,&q;sourceUrl&q;:&q;http://www.metabolomicsworkbench.org/&q;,&q;imgAlt&q;:&q;Metabolomics Workbench logo&q;,&q;repository&q;:&q;MetabolomicsWorkbench&q;,&q;source&q;:&q;metabolomics_workbench&q;,&q;lastUpdated&q;:&q;2023-03-10&q;,&q;orcidName&q;:&q;METABOLOMICSWORKBENCH&q;,&q;domain&q;:&q;Metabolomics Workbench&q;,&q;description&q;:&q;is a scalable and extensible informatics infrastructure which will serve as a national metabolomics resource.\n&q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:null},{&q;databaseName&q;:&q;NCBI&q;,&q;title&q;:&q;NCBI&q;,&q;sourceUrl&q;:null,&q;imgAlt&q;:null,&q;repository&q;:&q;NCBI&q;,&q;source&q;:&q;NCBI&q;,&q;lastUpdated&q;:&q;2018-12-10&q;,&q;orcidName&q;:null,&q;domain&q;:&q;NCBI&q;,&q;description&q;:&q;&q;,&q;image&q;:null,&q;icon&q;:&q;img/ncbi.ico&q;,&q;urlTemplate&q;:&q;https://www.ncbi.nlm.nih.gov/bioproject/{accession}&q;,&q;accessionPrefix&q;:[&q;PRJNA&q;]},{&q;databaseName&q;:&q;GNPS&q;,&q;title&q;:&q;GNPS Database&q;,&q;sourceUrl&q;:&q;https://gnps.ucsd.edu/ProteoSAFe/static/gnps-splash.jsp&q;,&q;imgAlt&q;:&q;GNPS logo&q;,&q;repository&q;:&q;GNPS&q;,&q;source&q;:&q;gnps&q;,&q;lastUpdated&q;:&q;2023-10-10&q;,&q;orcidName&q;:&q;GNPS&q;,&q;domain&q;:&q;GNPS&q;,&q;description&q;:&q;The Global Natural Products Social Molecular Networking (GNPS) is a platform for providing an overview of the molecular features in mass spectrometry based metabolomics by comparing fragmentation patterns to identify chemical relationships.\n&q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:null},{&q;databaseName&q;:&q;PAXDB&q;,&q;title&q;:&q;PAXDB&q;,&q;sourceUrl&q;:&q;http://pax-db.org/&q;,&q;imgAlt&q;:&q;PAXDB&q;,&q;repository&q;:&q;PAXDB&q;,&q;source&q;:&q;paxdb&q;,&q;lastUpdated&q;:&q;2015-03-12&q;,&q;orcidName&q;:&q;PAXDB&q;,&q;domain&q;:&q;Paxdb&q;,&q;description&q;:&q;PaxDb contains estimated abundance values for a large number of proteins in several different species. Furthermore, you can find information about inter-species variation of protein abundances.\n                                            &q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:null},{&q;databaseName&q;:&q;ENA&q;,&q;title&q;:null,&q;sourceUrl&q;:&q;https://www.ebi.ac.uk/ena&q;,&q;imgAlt&q;:&q;ENA logo&q;,&q;repository&q;:&q;ENA&q;,&q;source&q;:&q;omics_ena_project&q;,&q;lastUpdated&q;:&q;2022-07-19&q;,&q;orcidName&q;:&q;ENA&q;,&q;domain&q;:&q;Omics ENA Project&q;,&q;description&q;:&q;European Nucleotide Archive. Note that the number of datasets below is smaller than the number of datasets returned when you click on that link. This is because the former refers to the datasets provided by ENA and the latter also includes datasets in other repositories that have re-analysed ENA data.&q;,&q;image&q;:null,&q;icon&q;:&q;img/ENA.png&q;,&q;urlTemplate&q;:&q;https://www.ebi.ac.uk/ena/data/view/{accession}&q;,&q;accessionPrefix&q;:[&q;SRP&q;,&q;ERP&q;]},{&q;databaseName&q;:&q;LINCS&q;,&q;title&q;:&q;LINCS&q;,&q;sourceUrl&q;:&q;  http://lincsportal.ccs.miami.edu/dcic-portal/&q;,&q;imgAlt&q;:&q;LINCS logo&q;,&q;repository&q;:&q;LINCS&q;,&q;source&q;:&q;lincs&q;,&q;lastUpdated&q;:&q;2022-09-29&q;,&q;orcidName&q;:&q;LINCS&q;,&q;domain&q;:&q;LINCS&q;,&q;description&q;:&q;The database contains all publicly available HMS LINCS datasets and information for each dataset about experimental reagents (small molecule perturbagens, cells, antibodies, and proteins) and experimental and data analysis protocols.&q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:null},{&q;databaseName&q;:&q;Pride&q;,&q;title&q;:&q;PRIDE Database&q;,&q;sourceUrl&q;:&q;http://www.ebi.ac.uk/pride/archive/&q;,&q;imgAlt&q;:&q;PRIDE logo&q;,&q;repository&q;:&q;Pride&q;,&q;source&q;:&q;pride&q;,&q;lastUpdated&q;:&q;2023-10-14&q;,&q;orcidName&q;:&q;PRIDE&q;,&q;domain&q;:&q;PRIDE&q;,&q;description&q;:&q;is a centralized, standards compliant, public data repository for proteomics data, including protein and peptide identifications, post-translational modifications and supporting spectral evidence.\n&q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:null},{&q;databaseName&q;:&q;GEO&q;,&q;title&q;:null,&q;sourceUrl&q;:&q;https://www.ncbi.nlm.nih.gov/geo/&q;,&q;imgAlt&q;:&q;GEO logo&q;,&q;repository&q;:&q;GEO&q;,&q;source&q;:&q;geo&q;,&q;lastUpdated&q;:&q;2022-08-01&q;,&q;orcidName&q;:&q;GEO&q;,&q;domain&q;:&q;GEO&q;,&q;description&q;:&q;NCBI Gene Expression Omnibus. Note that the number of datasets below is smaller than the number of datasets returned when you click on that link. This is because the former refers to the datasets provided by GEO and the latter also includes datasets in other repositories that have re-analysed GEO data or mirorred them.&q;,&q;image&q;:null,&q;icon&q;:&q;img/ncbi.ico&q;,&q;urlTemplate&q;:&q;https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc={accession}&q;,&q;accessionPrefix&q;:[&q;GSE&q;,&q;GDS&q;]},{&q;databaseName&q;:&q;JPOST Repository&q;,&q;title&q;:&q;JPOST Repository&q;,&q;sourceUrl&q;:&q;https://jpostdb.org/&q;,&q;imgAlt&q;:&q;JPOST Repository logo&q;,&q;repository&q;:&q;jPOST&q;,&q;source&q;:&q;jpost&q;,&q;lastUpdated&q;:&q;2023-10-14&q;,&q;orcidName&q;:&q;JPOST&q;,&q;domain&q;:&q;jPOST&q;,&q;description&q;:&q;Japan ProteOme STandard Repository is a new data repository of sharing MS raw/processed data.&q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:[&q;PXD&q;]},{&q;databaseName&q;:&q;dbGaP&q;,&q;title&q;:&q;dbGaP database&q;,&q;sourceUrl&q;:&q;https://www.ncbi.nlm.nih.gov/gap&q;,&q;imgAlt&q;:&q;dbGaP logo&q;,&q;repository&q;:&q;dbGaP&q;,&q;source&q;:&q;dbgap&q;,&q;lastUpdated&q;:&q;2020-08-26&q;,&q;orcidName&q;:&q;dbGaP&q;,&q;domain&q;:&q;dbGaP&q;,&q;description&q;:&q;The database of Genotypes and Phenotypes was developed to archive and distribute the data and results from studies that have investigated the interaction of genotype and phenotype in humans.&q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:[&q;phs&q;]},{&q;databaseName&q;:&q;BioModels&q;,&q;title&q;:&q;BioModels&q;,&q;sourceUrl&q;:&q;https://www.ebi.ac.uk/biomodels/&q;,&q;imgAlt&q;:&q;ExpressioAtlas logo&q;,&q;repository&q;:&q;BioModels&q;,&q;source&q;:&q;biomodels&q;,&q;lastUpdated&q;:&q;2023-10-09&q;,&q;orcidName&q;:&q;BIOMODELS&q;,&q;domain&q;:&q;BioModels&q;,&q;description&q;:&q;BioModels Database is a repository of computational models of biological processes. Models described from literature are manually curated and enriched with cross-references.\n&q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:null},{&q;databaseName&q;:&q;Cell Collective&q;,&q;title&q;:&q;Cell Collective&q;,&q;sourceUrl&q;:&q;https://cellcollective.org/&q;,&q;imgAlt&q;:&q;Cell Collective logo&q;,&q;repository&q;:&q;Cell Collective&q;,&q;source&q;:&q;cellcollective&q;,&q;lastUpdated&q;:&q;2021-10-25&q;,&q;orcidName&q;:&q;Cell Collective&q;,&q;domain&q;:&q;Cell Collective&q;,&q;description&q;:&q;Interactive Modelling of Biological Networks.\n\n\n\n &q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:null},{&q;databaseName&q;:&q;MetaboLights&q;,&q;title&q;:&q;MetaboLights Database&q;,&q;sourceUrl&q;:&q;http://www.ebi.ac.uk/metabolights/&q;,&q;imgAlt&q;:&q;MetaboLignts logo&q;,&q;repository&q;:&q;MetaboLights&q;,&q;source&q;:&q;metabolights_dataset&q;,&q;lastUpdated&q;:&q;2023-10-16&q;,&q;orcidName&q;:&q;METABOLIGHTS&q;,&q;domain&q;:&q;MetaboLights Dataset&q;,&q;description&q;:&q;is a database for Metabolomics experiments and derived information.&q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:null},{&q;databaseName&q;:&q;ArrayExpress&q;,&q;title&q;:&q;ArrayExpress Database&q;,&q;sourceUrl&q;:&q;https://www.ebi.ac.uk/arrayexpress/&q;,&q;imgAlt&q;:&q;ArrayExpress logo&q;,&q;repository&q;:&q;ArrayExpress&q;,&q;source&q;:&q;arrayexpress-repository&q;,&q;lastUpdated&q;:&q;2023-01-01&q;,&q;orcidName&q;:&q;ARRAYEXPRESS&q;,&q;domain&q;:&q;ArrayExpress&q;,&q;description&q;:&q;ArrayExpress Archive of Functional Genomics Data stores data from high-throughput functional genomics experiments, and provides these data for reuse to the research community.\n&q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:[&q;E-&q;]},{&q;databaseName&q;:&q;EVA&q;,&q;title&q;:&q;EVA database&q;,&q;sourceUrl&q;:&q;https://www.ebi.ac.uk/eva/&q;,&q;imgAlt&q;:&q;EVA logo&q;,&q;repository&q;:&q;EVA&q;,&q;source&q;:&q;eva&q;,&q;lastUpdated&q;:&q;2022-07-25&q;,&q;orcidName&q;:&q;EVA&q;,&q;domain&q;:&q;EVA&q;,&q;description&q;:&q;The European Variation Archive is an open-access database of all types of genetic variation data from all species.&q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:null},{&q;databaseName&q;:&q;ExpressionAtlas&q;,&q;title&q;:&q;Expression Atlas Database&q;,&q;sourceUrl&q;:&q;http://www.ebi.ac.uk/gxa/home&q;,&q;imgAlt&q;:&q;ExpressioAtlas logo&q;,&q;repository&q;:&q;ExpressionAtlas&q;,&q;source&q;:&q;atlas-experiments&q;,&q;lastUpdated&q;:&q;2023-06-22&q;,&q;orcidName&q;:&q;EXPRESSIONATLAS&q;,&q;domain&q;:&q;Expression Atlas Experiments&q;,&q;description&q;:&q;The Expression Atlas provides information on gene expression patterns under different biological conditions. Gene expression data is re-analysed in-house to detect genes showing interesting baseline and differential expression patterns.\n&q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:null},{&q;databaseName&q;:&q;EGA&q;,&q;title&q;:&q;EGA Database&q;,&q;sourceUrl&q;:&q;https://ega-archive.org&q;,&q;imgAlt&q;:&q;EGA logo&q;,&q;repository&q;:&q;EGA&q;,&q;source&q;:&q;ega&q;,&q;lastUpdated&q;:&q;2023-08-14&q;,&q;orcidName&q;:&q;EGA&q;,&q;domain&q;:&q;EGA&q;,&q;description&q;:&q;The European Genome-phenome Archive allows you to explore datasets from genomic studies, provided by a range of data providers. Access to datasets must be approved by the specified Data Access Committee (DAC).&q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:[&q;EGA&q;]},{&q;databaseName&q;:&q;MassIVE&q;,&q;title&q;:&q;MassIVE Database&q;,&q;sourceUrl&q;:&q;https://massive.ucsd.edu/ProteoSAFe/datasets.jsp&q;,&q;imgAlt&q;:&q;MassIVE logo&q;,&q;repository&q;:&q;MassIVE&q;,&q;source&q;:&q;massive&q;,&q;lastUpdated&q;:&q;2023-10-16&q;,&q;orcidName&q;:&q;MASSIVE&q;,&q;domain&q;:&q;MassIVE&q;,&q;description&q;:&q;is a community resource developed by the NIH-funded Center for Computational Mass Spectrometry to promote the global, free exchange of mass spectrometry data.\n&q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:[&q;MSV&q;]},{&q;databaseName&q;:&q;Physiome Model Repository&q;,&q;title&q;:&q;Physiome Model Repository&q;,&q;sourceUrl&q;:&q;http://physiomeproject.org/&q;,&q;imgAlt&q;:&q;Physiome Model Repository logo&q;,&q;repository&q;:&q;PMR&q;,&q;source&q;:&q;physiome&q;,&q;lastUpdated&q;:&q;2020-03-11&q;,&q;orcidName&q;:&q;Physiome Model Repository&q;,&q;domain&q;:&q;Physiome Model Repository&q;,&q;description&q;:&q;The main goal of the Physiome Model Repository is to provide a resource for the community to store, retrieve, search, reference, and reuse CellML models.\n***Integration of this database into OmicsDI is in an incomplete, early testing stage.*** &q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:null},{&q;databaseName&q;:&q;FAIRDOMHub&q;,&q;title&q;:&q;FAIRDOMHub&q;,&q;sourceUrl&q;:&q;https://fairdomhub.org/&q;,&q;imgAlt&q;:&q;FAIRDOMHub logo&q;,&q;repository&q;:&q;FAIRDOMHub&q;,&q;source&q;:&q;fairdomhub&q;,&q;lastUpdated&q;:&q;2022-04-12&q;,&q;orcidName&q;:&q;FAIRDOMHub&q;,&q;domain&q;:&q;FAIRDOMHub&q;,&q;description&q;:&q;FAIRDOMHub is built upon the SEEK software suite, which is an open source web platform for sharing scientific research assets, processes and outcomes. \n***Integration of this database into OmicsDI is in an incomplete, early testing stage.*** &q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:null},{&q;databaseName&q;:&q;BioStudies&q;,&q;title&q;:&q;BioStudies&q;,&q;sourceUrl&q;:&q;https://www.ebi.ac.uk/biostudies/&q;,&q;imgAlt&q;:&q;BioStudies logo&q;,&q;repository&q;:&q;biostudies&q;,&q;source&q;:&q;biostudies&q;,&q;lastUpdated&q;:&q;2019-01-01&q;,&q;orcidName&q;:&q;BioStudies&q;,&q;domain&q;:&q;BioStudies&q;,&q;description&q;:&q;The BioStudies database holds descriptions of biological studies, links to data from these studies in other databases at EMBL-EBI or outside, as well as data that do not fit in the structured archives at EMBL-EBI.&q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:[&q;S-EPM&q;]},{&q;databaseName&q;:&q;iProX&q;,&q;title&q;:&q;iProX&q;,&q;sourceUrl&q;:&q;https://www.iprox.cn/&q;,&q;imgAlt&q;:&q;iProX logo&q;,&q;repository&q;:&q;iProX&q;,&q;source&q;:&q;iprox&q;,&q;lastUpdated&q;:&q;2023-10-17&q;,&q;orcidName&q;:&q;iProX&q;,&q;domain&q;:&q;iProX&q;,&q;description&q;:&q;iProX is an integrated proteome resources center in China, which is built to accelerate the worldwide data sharing in proteomics. iProX is composed of a data submission system and a proteome database.\n&q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:null},{&q;databaseName&q;:&q;PanoramaPublic&q;,&q;title&q;:&q;Panorama Public&q;,&q;sourceUrl&q;:&q;https://panoramaweb.org/&q;,&q;imgAlt&q;:&q;Panorama Public logo&q;,&q;repository&q;:&q;PanoramaPublic&q;,&q;source&q;:&q;panorama&q;,&q;lastUpdated&q;:&q;2023-01-26&q;,&q;orcidName&q;:&q;PanoramaPublic&q;,&q;domain&q;:&q;panorama&q;,&q;description&q;:&q;Panorama Public, a repository of Skyline documents and supplementary data associated with published manuscripts.  Visitors may freely browse publicly available data, as well as download Skyline documents.\n&q;,&q;image&q;:null,&q;icon&q;:null,&q;urlTemplate&q;:null,&q;accessionPrefix&q;:null}]},&q;http_requests:__BASE_URL_0__dataset/get?accession=E-GEOD-37196&a;database=arrayexpress-repository&q;:{&q;body&q;:{&q;id&q;:&q;E-GEOD-37196&q;,&q;source&q;:&q;arrayexpress-repository&q;,&q;name&q;:&q;Interference of PPAR gamma signaling in thoracic aorta&q;,&q;description&q;:&q;This SuperSeries is composed of the following subset Series: GSE37194: Gene expression profiling during interference with PPAR gamma signaling in thoracic aorta GSE37195: Gene expression profiling using exon arrays during interference with PPAR gamma signaling in thoracic aorta Refer to individual Series&q;,&q;keywords&q;:null,&q;organisms&q;:[{&q;acc&q;:&q;&q;,&q;name&q;:&q;Mus Musculus&q;}],&q;publicationDate&q;:&q;2012-10-03&q;,&q;publicationIds&q;:null,&q;protocols&q;:[{&q;name&q;:&q;data_protocol&q;,&q;description&q;:&q;Assay Data Transformation - ID_REF =  VALUE = RMA normalized gene level expression values\n Feature Extraction - Affymetrix Power Tools (APT, release apt-1.12.0) was used to process the microarray data files (i.e., *.CEL files) and to normalize the data according to the RMA (robust multiarray average) algorithm to generate gene-level (i.e., transcript) signal values.\n Image Adquisition - The scanning of the arrays was performed at the University of Iowa DNA Core facility using protocols recommended by the manufacturer (Affymetrix).&q;},{&q;name&q;:&q;sample_protocol&q;,&q;description&q;:&q;Hybridization - The hybridization procedures were performed at the University of Iowa DNA Core facility using protocols recommended by the manufacturer (Affymetrix).\n Labeling - A one-step amplification and labeling procedure (Affymetrix) starting with approximately 3-5 ug of total RNA was used to generate biotin-labeled RNA fragments.\n Nucleic Acid Extraction - Total RNA was extracted using Trizol reagent (Invitrogen, Carlsbad, CA) and was purified using a column-based procedure following a standard protocol.&q;}],&q;instruments&q;:null,&q;citationsCount&q;:0,&q;connectionsCount&q;:0,&q;reanalysisCount&q;:0,&q;viewsCount&q;:7,&q;downloadCount&q;:0,&q;citationsCountScaled&q;:0,&q;connectionsCountScaled&q;:0,&q;reanalysisCountScaled&q;:0,&q;viewsCountScaled&q;:0.00215916101172116,&q;downloadCountScaled&q;:0,&q;currentStatus&q;:&q;Enriched&q;,&q;experimentType&q;:null,&q;labMembers&q;:null,&q;full_dataset_link&q;:&q;https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-37196&q;,&q;tissues&q;:null,&q;diseases&q;:null,&q;omics_type&q;:[&q;Transcriptomics&q;],&q;similars&q;:null,&q;organization&q;:null,&q;dates&q;:{&q;publication&q;:[&q;2012-10-03&q;],&q;updated&q;:[&q;2012-10-18&q;]},&q;submitter&q;:[&q;Henry Keen&q;],&q;submitterMail&q;:[&q;henry-keen@uiowa.edu&q;],&q;labHead&q;:null,&q;labHeadMail&q;:null,&q;secondary_accession&q;:null,&q;repositories&q;:[&q;ArrayExpress&q;],&q;scores&q;:{&q;citationCount&q;:0,&q;reanalysisCount&q;:0,&q;viewCount&q;:7,&q;searchCount&q;:0},&q;claimable&q;:true}},&q;http_requests:__BASE_URL_0__dataset/getDRS?accession=E-GEOD-37196&a;database=arrayexpress&q;:{&q;body&q;:[]},&q;http_requests:__BASE_URL_0__seo/schema/arrayexpress-repository/E-GEOD-37196&q;:{&q;body&q;:{&q;mainEntity&q;:{&q;creator&q;:[{&q;name&q;:&q;Henry Keen&q;,&q;@type&q;:&q;Person&q;}],&q;keywords&q;:&q;transcription profiling by array&q;,&q;citation&q;:{&q;name&q;:&q;Interference of PPAR gamma signaling in thoracic aorta&q;,&q;url&q;:&q;http://www.omicsdi.org/dataset/arrayexpress-repository/E-GEOD-37196&q;,&q;author&q;:{&q;name&q;:&q;Henry Keen&q;,&q;@type&q;:&q;Person&q;},&q;publisher&q;:{&q;name&q;:&q;ARRAYEXPRESS-REPOSITORY&q;,&q;@type&q;:&q;Organization&q;},&q;@type&q;:&q;CreativeWork&q;},&q;@type&q;:&q;Dataset&q;,&q;name&q;:&q;Interference of PPAR gamma signaling in thoracic aorta&q;,&q;variableMeasured&q;:&q;Transcriptomics&q;,&q;description&q;:&q;This SuperSeries is composed of the following subset Series: GSE37194: Gene expression profiling during interference with PPAR gamma signaling in thoracic aorta GSE37195: Gene expression profiling using exon arrays during interference with PPAR gamma signaling in thoracic aorta Refer to individual Series&q;,&q;distribution&q;:{&q;contentUrl&q;:&q;http://www.omicsdi.org/ws/dataset/arrayexpress-repository/E-GEOD-37196.xml&q;,&q;@type&q;:&q;DataDownload&q;,&q;encodingFormat&q;:&q;XML&q;},&q;sameAs&q;:&q;https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-37196&q;},&q;breadcrumb&q;:{&q;itemListElement&q;:[{&q;item&q;:&q;https://www.omicsdi.org&q;,&q;@type&q;:&q;ListItem&q;,&q;name&q;:&q;OmicsDI&q;,&q;position&q;:1},{&q;item&q;:&q;https://www.omicsdi.org/search?q=(repository:%22ArrayExpress%22)&q;,&q;@type&q;:&q;ListItem&q;,&q;name&q;:&q;ArrayExpress&q;,&q;position&q;:2},{&q;item&q;:&q;http://www.omicsdi.org/dataset/arrayexpress-repository/E-GEOD-37196&q;,&q;@type&q;:&q;ListItem&q;,&q;name&q;:&q;E-GEOD-37196&q;,&q;position&q;:3}],&q;@type&q;:&q;BreadcrumbList&q;},&q;@type&q;:&q;ItemPage&q;,&q;@context&q;:&q;http://schema.org&q;}},&q;http_requests:__BASE_URL_0__dataset/getDRS?accession=E-GEOD-37196&a;database=arrayexpress-repository&q;:{&q;body&q;:[]},&q;http_requests:__BASE_URL_0__dataset/arrayexpress-repository/E-GEOD-37196.json&q;:{&q;body&q;:{&q;database&q;:&q;ArrayExpress&q;,&q;file_versions&q;:[{&q;files&q;:{&q;Txt&q;:[&q;ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/GEOD/E-GEOD-37196/E-GEOD-37196.README.txt&q;,&q;ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/GEOD/E-GEOD-37196/E-GEOD-37196.sdrf.txt&q;],&q;Raw&q;:[&q;ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/GEOD/E-GEOD-37196/E-GEOD-37196.raw.2.zip&q;,&q;ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/GEOD/E-GEOD-37196/E-GEOD-37196.raw.1.zip&q;],&q;Idf&q;:[&q;ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/GEOD/E-GEOD-37196/E-GEOD-37196.idf.txt&q;],&q;Processed&q;:[&q;ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/GEOD/E-GEOD-37196/E-GEOD-37196.processed.1.zip&q;]},&q;type&q;:&q;primary&q;},{&q;files&q;:{&q;Txt&q;:[&q;ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE37nnn/GSE37196/suppl/filelist.txt&q;],&q;Raw&q;:[&q;ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE37nnn/GSE37196/suppl/GSE37196_RAW.tar&q;]},&q;type&q;:&q;mirror&q;}],&q;scores&q;:{&q;citationCount&q;:0,&q;reanalysisCount&q;:0,&q;viewCount&q;:7,&q;searchCount&q;:0},&q;additional&q;:{&q;omics_type&q;:[&q;Transcriptomics&q;],&q;submitter&q;:[&q;Henry Keen&q;],&q;species&q;:[&q;Mus Musculus&q;],&q;full_dataset_link&q;:[&q;https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-37196&q;],&q;submitter_keywords&q;:[&q;transcription profiling by array&q;],&q;submitter_email&q;:[&q;henry-keen@uiowa.edu&q;],&q;repository&q;:[&q;ArrayExpress&q;],&q;sample_protocol&q;:[&q;Hybridization - The hybridization procedures were performed at the University of Iowa DNA Core facility using protocols recommended by the manufacturer (Affymetrix).\n Labeling - A one-step amplification and labeling procedure (Affymetrix) starting with approximately 3-5 ug of total RNA was used to generate biotin-labeled RNA fragments.\n Nucleic Acid Extraction - Total RNA was extracted using Trizol reagent (Invitrogen, Carlsbad, CA) and was purified using a column-based procedure following a standard protocol.&q;],&q;data_protocol&q;:[&q;Assay Data Transformation - ID_REF =  VALUE = RMA normalized gene level expression values\n Feature Extraction - Affymetrix Power Tools (APT, release apt-1.12.0) was used to process the microarray data files (i.e., *.CEL files) and to normalize the data according to the RMA (robust multiarray average) algorithm to generate gene-level (i.e., transcript) signal values.\n Image Adquisition - The scanning of the arrays was performed at the University of Iowa DNA Core facility using protocols recommended by the manufacturer (Affymetrix).&q;],&q;view_count&q;:[&q;7&q;],&q;description_synonyms&q;:[&q;Profiling, biological signaling, Transcriptome, aorta thoracalis, Monitorings, Gene, compositionality, composed of, Gene Expression Profilings, Gene Expression, Differential Display, transcription profiling, Transcript Expression Analysis, Thiazolidinedione Receptor, arcus aortae., mRNA Differential Displays, Transcript Expression Analyses, Peroxisome Proliferator-Activated Receptor gamma, Gene Expression Pattern Analysis, Gene Expression Monitorings, Analysis, Profilings, Peroxisome Proliferator Activated Receptor gamma, mRNA Differential Display, gene expression profiling, Gene Expression Monitoring, Transcriptome Profiling, thoracic aorta, Analyses, mRNA, exonic region, Transcriptome Profilings, content, Thiazolidinedione, PPARgamma2, thoracic part of aorta, PPARgamma3, composition, Transcriptomics, Differential Displays, Transcript Expression, Transcriptome Analysis, Transcriptome Analyses, pars thoracica aortae, Monitoring, signalling, arcus aortae, Gene Expression Analysis, mPPARgamma1, aorta thoracica, Gene Expression Analyses, mPPARgamma2, signalling process, PPARgamma, signaling process, structure, Expression Analysis, Receptor, Expression Analyses, aortic arch, single organism signaling&q;],&q;data_synonyms&q;:[&q;projections, Apt, d230, DNS, (Deoxyribonucleotide)n, Ref(2)P, lamellae, CELNOS, nucleocapsid, IMAGE, dTAFII250, chronic eosinophilic leukemia, EfW1, process of organ, Deoxyribonucleic acids, protrusion, lamella, dmTAF[[II]]230, ref2p, Deoxyribonucleic Acid, dmTAF1, Taf230, CEL, Ref(2)p, THOC4, p62, Lysophospholipase I, Lysophospholipase 1, l(2)k11531, APT monohydrobromide, TAF250, ALY|REF, Normalities, average, Taf200, thymus nucleic acid, dTAF[[II]]250, TFIID TAF250, cel, cell, Lysophospholipase I wt Allele, ALY/REF, LPL-I, LYPLA1 wt Allele, Double Stranded, Taf1p, Deoxyribonucleic acid, ridges, study assay, DmelCG5393, BEF, dTAF250, l(2)06369, data processing, not otherwise specified, scientific observation, papilla, chronic eosinophilic leukemia/hypereosinophilic syndrome, REF, Double-Stranded DNA, (Deoxyribonucleotide)m, deoxyribonucleic acids, DNAn, TAF, laminae, ref(2)Pn, measuring, data, TAF[[II]]250, dTAF[[II]]230, anatomical protrusion, DmelCG10360, 3041, ALY, anatomical process, DNAn+1, l(2)03041, CEL/hypereosinophilic syndrome, lamina, ref, TAF200, LYSOPLA, flanges, ref(2)p, l(3)84Ab, Bssl, BG:DS00004.13, APT dihydrobromide, Normalcy, Double-Stranded, TAFII-250, TAF250/230, feature_extraction, 3.1.2.-, 1810036E18Rik, dTAF230, (Deoxyribonucleotide)n+m, any method, Ref2P, ref(2)Po2, LysoPLA I, TAFII250, p230, Algorithm, shelf, APT dihydrochloride, TAF[[II]]250/230, TFIID, University, ds-DNA, desoxyribose nucleic acid, flange, organ process, eosinophilic leukemia, Taf[[II]]250, l(2)09049, TAF[[II]]230, core., TDF/APT, CG5393, LPL1, Normality, shelves, APT-1, APT1, INSDC_feature:gene, TAF[II]250, TDF, CG17603, projection, TAF[[II]], ridge, process, processes, data analysis, Health, DmelCG17603, Taf250, spine, SR3-5, hAPT1, Bal, BAL, ds DNA, Desoxyribonukleinsaeure, processus, microarray, l(2)59Ea, DNA, tdf, CG10360, TAF230, TAF1&q;],&q;sample_synonyms&q;:[&q;4-d]imidazol-4-yl)pentanoic acid, Ribonucleic, Deacura, DNS, GRP1/cytohesin 1, vitamin B7, (Deoxyribonucleotide)n, Biotine, nucleocapsid, Medebiotin, Vitamin H, Deoxyribonucleic acids, Intervention or Procedure, CG11633, cytohesin/GRP1, method, Deoxyribonucleic Acid, 5-[(3aS, Biotin, GRP1, Grp1, 4beta, Gene Products, 6aR)-Hexahydro-2-oxo-1H-thieno[3, 6aalpha))-, Gelfert, Non Polyadenylated, RNA Gene Products, Biotin Hermes, thymus nucleic acid, reactivo, Coenzyme R, interventionDescription, PTPSTEP, (3aS, cis-Hexahydro-2-oxo-1H-thieno(3, biotina, l(2)SH2 0323, Double Stranded, Interventional, Deoxyribonucleic acid, labeling, biotine, BIOTIN, SURGICAL AND MEDICAL PROCEDURES, Non-Polyadenylated RNA, 4-d]imidazole-4-valeric acid, Neural-specific protein-tyrosine phosphatase, Hermes, column, method used in an experiment., (+)-cis-Hexahydro-2-oxo-1H-thieno[3, l(2)k08110, Double-Stranded DNA, (Deoxyribonucleotide)m, deoxyribonucleic acids, biotinum, DNAn, reagents, GPH, Intervention Strategies, Medobiotin, RNA, total RNA extract, Rombellin, ribose nucleic acid, DNAn+1, ribonucleic acids, Roche, stepk, Biotin Gelfert, RNS, Double-Stranded, Procedure, 5-(2-oxohexahydro-1H-thieno[3, 4]imidazole-4-valeric acid, 4S, 4-d)imidazole-4-pentanoic acid, (3aS-(3aalpha, (Deoxyribonucleotide)n+m, 6aR)-2-oxohexahydro-1H-thieno[3, cis-Tetrahydro-2-oxothieno(3, 4]imidazoline-4-valeric acid, biotin, yeast nucleic acid, Ribonukleinsaeure, 4-d]imidazol-4-yl]pentanoic acid, 3.1.3.48, core, Biotin Ratiopharm, pentosenucleic acids, University, Ribonucleic acids, ds-DNA, 4-d)imidazoline-4-valeric acid, desoxyribose nucleic acid, reactif, l(2)SH0323, Biodermatin, D-Biotin, CYH1, Intervention, Biotin-Ratiopharm, 1H-Thieno(3, ribonucleic acid, Acid, Biotine Roche, Gabunat, Step, CG11628, Non Polyadenylated RNA, hexahydro-2-oxo-, Non-Polyadenylated, reagent, Striatum-enriched protein-tyrosine phosphatase, Ribonucleic Acid, 4)imidazole-4-valeric acid, study protocol, DmelCG11628, plan specification, STEP, ds DNA, Desoxyribonukleinsaeure, cis-(+)-Tetrahydro-2-oxothieno[3, DNA, D-(+)-biotin, Biokur&q;],&q;name_synonyms&q;:[&q;Thiazolidinedione Receptor, arcus aortae., biological signaling, Peroxisome Proliferator-Activated Receptor gamma, mPPARgamma1, aorta thoracica, mPPARgamma2, signalling process, thoracic aorta, aorta thoracalis, PPARgamma, signaling process, Thiazolidinedione, PPARgamma2, thoracic part of aorta, PPARgamma3, Receptor, Peroxisome Proliferator Activated Receptor gamma, pars thoracica aortae, aortic arch, signalling, single organism signaling&q;],&q;citation_count_scaled&q;:[&q;0.0&q;],&q;download_count_scaled&q;:[&q;0.0&q;],&q;normalized_connections&q;:[&q;0.0&q;],&q;reanalysis_count_scaled&q;:[&q;0.0&q;],&q;view_count_scaled&q;:[&q;0.00215916101172116&q;],&q;citation_count&q;:[&q;0&q;],&q;additional_accession&q;:[]},&q;is_claimable&q;:true,&q;name&q;:&q;Interference of PPAR gamma signaling in thoracic aorta&q;,&q;description&q;:&q;This SuperSeries is composed of the following subset Series: GSE37194: Gene expression profiling during interference with PPAR gamma signaling in thoracic aorta GSE37195: Gene expression profiling using exon arrays during interference with PPAR gamma signaling in thoracic aorta Refer to individual Series&q;,&q;dates&q;:{&q;publication&q;:&q;2012-10-03&q;,&q;updated&q;:&q;2012-10-18&q;},&q;accession&q;:&q;E-GEOD-37196&q;,&q;cross_references&q;:{&q;TAXONOMY&q;:[&q;10090&q;]}}},&q;http_requests:__BASE_URL_0__dataset/getSimilar?accession=E-GEOD-37196&a;domain=omics&a;database=arrayexpress-repository&q;:{&q;body&q;:{&q;count&q;:14,&q;datasets&q;:[{&q;id&q;:&q;E-GEOD-37195&q;,&q;source&q;:&q;arrayexpress-repository&q;,&q;title&q;:&q;Gene expression profiling using exon arrays during interference with PPAR gamma signaling in thoracic aorta&q;,&q;description&q;:&q;Pharmacological activation of the transcription factor PPAR gamma lowers blood pressure and improves glucose tolerance in humans.  In contrast, naturally occurring mutations (e.g., P467L, V290M) in the ligand binding domain of PPAR gamma in humans leads to severe insulin resistance and early-onset hypertension. Experimental evidence, including whole genome expression profiling, suggests that these mutant versions of PPAR gamma act in a dominant negative manner. Because PPAR gamma is expressed in a variety of cell types and tissues, we generated a transgenic mouse model (SP467L) specifically targeting dominant negative PPAR gamma to the vascular smooth muscle cells in order to determine the action of PPAR gamma in the blood vessel independent of its systemic metabolic actions.   In the data set provided herein, we examined the gene expression profile in thoracic aorta from SP467L mice and their control littermates using the Affymetrix mouse exon 1.0 ST array. We generated transgenic mice specifically targeting expression of mutant dominant negative human PPAR gamma (P467L) to vascular smooth muscle using a smooth muscle-specific promoter (smooth muscle myosin heavy chain or SMMHC).  Thoracic aortas were isolated from 7 male transgenic  mice and 5 non-transgenic littermate controls. Total RNA was prepared using conventional methods and quality was assessed using the Bioanalyzer 2100 (Agilent Technologies). For the microarray hybridizations, each sample corresponded to aorta derived from one mouse. All procedures were conducted at the University of Iowa DNA Core facility using standard Affymetrix protocols. In brief, approximately 50 ng of total RNA was used as input to a two-step amplification procedure (NuGen, http://www.nugeninc.com/) to generate biotin-labeled RNA fragments for hybridization to the Affymetrix mouse exon 1.0 ST array.&q;,&q;keywords&q;:null,&q;organisms&q;:null,&q;publicationDate&q;:&q;20121003&q;,&q;score&q;:&q;32.157513&q;,&q;omicsType&q;:[&q;Transcriptomics&q;,&q;Multiomics&q;],&q;citationsCount&q;:0,&q;connectionsCount&q;:0,&q;reanalysisCount&q;:0,&q;viewsCount&q;:0,&q;downloadCount&q;:0,&q;citationsCountScaled&q;:0,&q;connectionsCountScaled&q;:0,&q;reanalysisCountScaled&q;:0,&q;viewsCountScaled&q;:0,&q;downloadCountScaled&q;:0,&q;claimable&q;:false},{&q;id&q;:&q;E-GEOD-37194&q;,&q;source&q;:&q;arrayexpress-repository&q;,&q;title&q;:&q;Gene expression profiling  during interference with PPAR gamma signaling in thoracic aorta&q;,&q;description&q;:&q;Pharmacological activation of the transcription factor PPAR gamma lowers blood pressure and improves glucose tolerance in humans.  In contrast, naturally occurring mutations (e.g., P467L, V290M) in the ligand binding domain of PPAR gamma in humans leads to severe insulin resistance and early-onset hypertension. Experimental evidence, including whole genome expression profiling, suggests that these mutant versions of PPAR gamma act in a dominant negative manner. Because PPAR gamma is expressed in a variety of cell types and tissues, we generated a transgenic mouse model (SP467L) specifically targeting dominant negative PPAR gamma to the vascular smooth muscle cells in order to determine the action of PPAR gamma in the blood vessel independent of its systemic metabolic actions.   In the data set provided herein, we examined the gene expression profile in thoracic aorta from SP467L mice and their control littermates using the Affymetrix Mouse Genome 430 2.0 array. We generated transgenic mice specifically targeting expression of mutant dominant negative human PPAR gamma (P467L) to vascular smooth muscle using a smooth muscle-specific promoter (smooth muscle myosin heavy chain or SMMHC).  Thoracic aortas were isolated from male transgenic mice and corresponding non-transgenic littermate controls. Total RNA was prepared using conventional methods and quality was assessed using the Bioanalyzer 2100 (Agilent Technologies). For the microarray hybridizations, 2 samples from control mice and 3 samples from transgenic mice were used. Each sample was an independent biological replicate generated by pooling total RNA from 6-8 separate mouse aortas. All procedures were conducted at the University of Iowa DNA Core facility using standard Affymetrix protocols. In brief, approximately 3-5 ug of total RNA was used as input to a one-step amplification procedure to generate biotin-labeled RNA fragments for hybridization to the Affymetrix Mouse Genome 430 2.0 array.&q;,&q;keywords&q;:null,&q;organisms&q;:null,&q;publicationDate&q;:&q;20121003&q;,&q;score&q;:&q;32.006542&q;,&q;omicsType&q;:[&q;Transcriptomics&q;],&q;citationsCount&q;:0,&q;connectionsCount&q;:0,&q;reanalysisCount&q;:0,&q;viewsCount&q;:0,&q;downloadCount&q;:0,&q;citationsCountScaled&q;:0,&q;connectionsCountScaled&q;:0,&q;reanalysisCountScaled&q;:0,&q;viewsCountScaled&q;:0,&q;downloadCountScaled&q;:0,&q;claimable&q;:false},{&q;id&q;:&q;E-GEOD-37196&q;,&q;source&q;:&q;biostudies&q;,&q;title&q;:&q;Interference of PPAR gamma signaling in thoracic aorta&q;,&q;description&q;:&q;This SuperSeries is composed of the following subset Series: GSE37194: Gene expression profiling during interference with PPAR gamma signaling in thoracic aorta GSE37195: Gene expression profiling using exon arrays during interference with PPAR gamma signaling in thoracic aorta Refer to individual Series&q;,&q;keywords&q;:null,&q;organisms&q;:null,&q;publicationDate&q;:null,&q;score&q;:&q;31.006674&q;,&q;omicsType&q;:[&q;Unknown&q;],&q;citationsCount&q;:0,&q;connectionsCount&q;:0,&q;reanalysisCount&q;:0,&q;viewsCount&q;:0,&q;downloadCount&q;:0,&q;citationsCountScaled&q;:0,&q;connectionsCountScaled&q;:0,&q;reanalysisCountScaled&q;:0,&q;viewsCountScaled&q;:0,&q;downloadCountScaled&q;:0,&q;claimable&q;:false},{&q;id&q;:&q;E-GEOD-37195&q;,&q;source&q;:&q;biostudies&q;,&q;title&q;:&q;Gene expression profiling using exon arrays during interference with PPAR gamma signaling in thoracic aorta&q;,&q;description&q;:&q;Pharmacological activation of the transcription factor PPAR gamma lowers blood pressure and improves glucose tolerance in humans.  In contrast, naturally occurring mutations (e.g., P467L, V290M) in the ligand binding domain of PPAR gamma in humans leads to severe insulin resistance and early-onset hypertension. Experimental evidence, including whole genome expression profiling, suggests that these mutant versions of PPAR gamma act in a dominant negative manner. Because PPAR gamma is expressed in a variety of cell types and tissues, we generated a transgenic mouse model (SP467L) specifically targeting dominant negative PPAR gamma to the vascular smooth muscle cells in order to determine the action of PPAR gamma in the blood vessel independent of its systemic metabolic actions.   In the data set provided herein, we examined the gene expression profile in thoracic aorta from SP467L mice and their control littermates using the Affymetrix mouse exon 1.0 ST array. We generated transgenic mice specifically targeting expression of mutant dominant negative human PPAR gamma (P467L) to vascular smooth muscle using a smooth muscle-specific promoter (smooth muscle myosin heavy chain or SMMHC).  Thoracic aortas were isolated from 7 male transgenic  mice and 5 non-transgenic littermate controls. Total RNA was prepared using conventional methods and quality was assessed using the Bioanalyzer 2100 (Agilent Technologies). For the microarray hybridizations, each sample corresponded to aorta derived from one mouse. All procedures were conducted at the University of Iowa DNA Core facility using standard Affymetrix protocols. In brief, approximately 50 ng of total RNA was used as input to a two-step amplification procedure (NuGen, http://www.nugeninc.com/) to generate biotin-labeled RNA fragments for hybridization to the Affymetrix mouse exon 1.0 ST array.&q;,&q;keywords&q;:null,&q;organisms&q;:null,&q;publicationDate&q;:null,&q;score&q;:&q;23.14249&q;,&q;omicsType&q;:[&q;Unknown&q;],&q;citationsCount&q;:0,&q;connectionsCount&q;:0,&q;reanalysisCount&q;:0,&q;viewsCount&q;:0,&q;downloadCount&q;:0,&q;citationsCountScaled&q;:0,&q;connectionsCountScaled&q;:0,&q;reanalysisCountScaled&q;:0,&q;viewsCountScaled&q;:0,&q;downloadCountScaled&q;:0,&q;claimable&q;:false},{&q;id&q;:&q;E-GEOD-37194&q;,&q;source&q;:&q;biostudies&q;,&q;title&q;:&q;Gene expression profiling  during interference with PPAR gamma signaling in thoracic aorta&q;,&q;description&q;:&q;Pharmacological activation of the transcription factor PPAR gamma lowers blood pressure and improves glucose tolerance in humans.  In contrast, naturally occurring mutations (e.g., P467L, V290M) in the ligand binding domain of PPAR gamma in humans leads to severe insulin resistance and early-onset hypertension. Experimental evidence, including whole genome expression profiling, suggests that these mutant versions of PPAR gamma act in a dominant negative manner. Because PPAR gamma is expressed in a variety of cell types and tissues, we generated a transgenic mouse model (SP467L) specifically targeting dominant negative PPAR gamma to the vascular smooth muscle cells in order to determine the action of PPAR gamma in the blood vessel independent of its systemic metabolic actions.   In the data set provided herein, we examined the gene expression profile in thoracic aorta from SP467L mice and their control littermates using the Affymetrix Mouse Genome 430 2.0 array. We generated transgenic mice specifically targeting expression of mutant dominant negative human PPAR gamma (P467L) to vascular smooth muscle using a smooth muscle-specific promoter (smooth muscle myosin heavy chain or SMMHC).  Thoracic aortas were isolated from male transgenic mice and corresponding non-transgenic littermate controls. Total RNA was prepared using conventional methods and quality was assessed using the Bioanalyzer 2100 (Agilent Technologies). For the microarray hybridizations, 2 samples from control mice and 3 samples from transgenic mice were used. Each sample was an independent biological replicate generated by pooling total RNA from 6-8 separate mouse aortas. All procedures were conducted at the University of Iowa DNA Core facility using standard Affymetrix protocols. In brief, approximately 3-5 ug of total RNA was used as input to a one-step amplification procedure to generate biotin-labeled RNA fragments for hybridization to the Affymetrix Mouse Genome 430 2.0 array.&q;,&q;keywords&q;:null,&q;organisms&q;:null,&q;publicationDate&q;:null,&q;score&q;:&q;22.442125&q;,&q;omicsType&q;:[&q;Unknown&q;],&q;citationsCount&q;:0,&q;connectionsCount&q;:0,&q;reanalysisCount&q;:0,&q;viewsCount&q;:0,&q;downloadCount&q;:0,&q;citationsCountScaled&q;:0,&q;connectionsCountScaled&q;:0,&q;reanalysisCountScaled&q;:0,&q;viewsCountScaled&q;:0,&q;downloadCountScaled&q;:0,&q;claimable&q;:false},{&q;id&q;:&q;GSE37194&q;,&q;source&q;:&q;geo&q;,&q;title&q;:&q;Gene expression profiling during interference with PPAR gamma signaling in thoracic aorta&q;,&q;description&q;:&q;Pharmacological activation of the transcription factor PPAR gamma lowers blood pressure and improves glucose tolerance in humans. In contrast, naturally occurring mutations (e.g., P467L, V290M) in the ligand binding domain of PPAR gamma in humans leads to severe insulin resistance and early-onset hypertension. Experimental evidence, including whole genome expression profiling, suggests that these mutant versions of PPAR gamma act in a dominant negative manner. Because PPAR gamma is expressed in a variety of cell types and tissues, we generated a transgenic mouse model (SP467L) specifically targeting dominant negative PPAR gamma to the vascular smooth muscle cells in order to determine the action of PPAR gamma in the blood vessel independent of its systemic metabolic actions. In the data set provided herein, we examined the gene expression profile in thoracic aorta from SP467L mice and their control littermates using the Affymetrix Mouse Genome 430 2.0 array.&q;,&q;keywords&q;:null,&q;organisms&q;:null,&q;publicationDate&q;:&q;20121003&q;,&q;score&q;:&q;22.010906&q;,&q;omicsType&q;:[&q;Unknown&q;],&q;citationsCount&q;:0,&q;connectionsCount&q;:0,&q;reanalysisCount&q;:0,&q;viewsCount&q;:0,&q;downloadCount&q;:0,&q;citationsCountScaled&q;:0,&q;connectionsCountScaled&q;:0,&q;reanalysisCountScaled&q;:0,&q;viewsCountScaled&q;:0,&q;downloadCountScaled&q;:0,&q;claimable&q;:false},{&q;id&q;:&q;GSE37195&q;,&q;source&q;:&q;geo&q;,&q;title&q;:&q;Gene expression profiling using exon arrays during interference with PPAR gamma signaling in thoracic aorta&q;,&q;description&q;:&q;Pharmacological activation of the transcription factor PPAR gamma lowers blood pressure and improves glucose tolerance in humans. In contrast, naturally occurring mutations (e.g., P467L, V290M) in the ligand binding domain of PPAR gamma in humans leads to severe insulin resistance and early-onset hypertension. Experimental evidence, including whole genome expression profiling, suggests that these mutant versions of PPAR gamma act in a dominant negative manner. Because PPAR gamma is expressed in a variety of cell types and tissues, we generated a transgenic mouse model (SP467L) specifically targeting dominant negative PPAR gamma to the vascular smooth muscle cells in order to determine the action of PPAR gamma in the blood vessel independent of its systemic metabolic actions. In the data set provided herein, we examined the gene expression profile in thoracic aorta from SP467L mice and their control littermates using the Affymetrix mouse exon 1.0 ST array.&q;,&q;keywords&q;:null,&q;organisms&q;:null,&q;publicationDate&q;:&q;20121003&q;,&q;score&q;:&q;21.334171&q;,&q;omicsType&q;:[&q;Unknown&q;],&q;citationsCount&q;:0,&q;connectionsCount&q;:0,&q;reanalysisCount&q;:0,&q;viewsCount&q;:0,&q;downloadCount&q;:0,&q;citationsCountScaled&q;:0,&q;connectionsCountScaled&q;:0,&q;reanalysisCountScaled&q;:0,&q;viewsCountScaled&q;:0,&q;downloadCountScaled&q;:0,&q;claimable&q;:false},{&q;id&q;:&q;E-GEOD-37195&q;,&q;source&q;:&q;atlas-experiments&q;,&q;title&q;:&q;Gene expression profiling using exon arrays during interference with PPAR gamma signaling in thoracic aorta&q;,&q;description&q;:&q;Pharmacological activation of the transcription factor PPAR gamma lowers blood pressure and improves glucose tolerance in humans.  In contrast, naturally occurring mutations (e.g., P467L, V290M) in the ligand binding domain of PPAR gamma in humans leads to severe insulin resistance and early-onset hypertension. Experimental evidence, including whole genome expression profiling, suggests that these mutant versions of PPAR gamma act in a dominant negative manner. Because PPAR gamma is expressed in a variety of cell types and tissues, we generated a transgenic mouse model (SP467L) specifically targeting dominant negative PPAR gamma to the vascular smooth muscle cells in order to determine the action of PPAR gamma in the blood vessel independent of its systemic metabolic actions.   In the data set provided herein, we examined the gene expression profile in thoracic aorta from SP467L mice and their control littermates using the Affymetrix mouse exon 1.0 ST array. We generated transgenic mice specifically targeting expression of mutant dominant negative human PPAR gamma (P467L) to vascular smooth muscle using a smooth muscle-specific promoter (smooth muscle myosin heavy chain or SMMHC).  Thoracic aortas were isolated from 7 male transgenic  mice and 5 non-transgenic littermate controls. Total RNA was prepared using conventional methods and quality was assessed using the Bioanalyzer 2100 (Agilent Technologies). For the microarray hybridizations, each sample corresponded to aorta derived from one mouse. All procedures were conducted at the University of Iowa DNA Core facility using standard Affymetrix protocols. In brief, approximately 50 ng of total RNA was used as input to a two-step amplification procedure (NuGen, http://www.nugeninc.com/) to generate biotin-labeled RNA fragments for hybridization to the Affymetrix mouse exon 1.0 ST array.&q;,&q;keywords&q;:null,&q;organisms&q;:null,&q;publicationDate&q;:&q;20150928&q;,&q;score&q;:&q;21.0417&q;,&q;omicsType&q;:[&q;Unknown&q;,&q;Multiomics&q;],&q;citationsCount&q;:0,&q;connectionsCount&q;:0,&q;reanalysisCount&q;:0,&q;viewsCount&q;:0,&q;downloadCount&q;:0,&q;citationsCountScaled&q;:0,&q;connectionsCountScaled&q;:0,&q;reanalysisCountScaled&q;:0,&q;viewsCountScaled&q;:0,&q;downloadCountScaled&q;:0,&q;claimable&q;:false},{&q;id&q;:&q;PRJNA158553&q;,&q;source&q;:&q;omics_ena_project&q;,&q;title&q;:null,&q;description&q;:&q;Interference of PPAR gamma signaling in thoracic aorta&q;,&q;keywords&q;:null,&q;organisms&q;:null,&q;publicationDate&q;:null,&q;score&q;:&q;20.42776&q;,&q;omicsType&q;:[&q;Unknown&q;,&q;Multiomics&q;],&q;citationsCount&q;:0,&q;connectionsCount&q;:0,&q;reanalysisCount&q;:0,&q;viewsCount&q;:0,&q;downloadCount&q;:0,&q;citationsCountScaled&q;:0,&q;connectionsCountScaled&q;:0,&q;reanalysisCountScaled&q;:0,&q;viewsCountScaled&q;:0,&q;downloadCountScaled&q;:0,&q;claimable&q;:false},{&q;id&q;:&q;E-GEOD-36482&q;,&q;source&q;:&q;arrayexpress-repository&q;,&q;title&q;:&q;Interference with PPAR gamma signaling in mesenteric artery&q;,&q;description&q;:&q;Pharmacological activation of the transcription factor PPAR gamma lowers blood pressure and improves glucose tolerance in humans.  In contrast, naturally occurring mutations (e.g., P467L, V290M) in the ligand binding domain of PPAR gamma in humans leads to severe insulin resistance and early-onset hypertension. Experimental evidence, including whole genome expression profiling, suggests that these mutant versions of PPAR gamma act in a dominant negative manner. Because PPAR gamma is expressed in a variety of cell types and tissues, we generated a transgenic mouse model (SP467L) specifically targeting dominant negative PPAR gamma to the vascular smooth muscle cells in order to determine the action of PPAR gamma in the blood vessel independent of its systemic metabolic actions.   In the data set provided herein, we examined the gene expression profile in mesenteric vessels from SP467L mice and their control littermates using  the Affymetrix mouse exon 1.0 ST array. We generated transgenic mice specifically targeting expression of mutant dominant negative human PPAR gamma (P467L) to vascular smooth muscle using a smooth muscle-specific promoter (smooth muscle myosin heavy chain or SMMHC).  Mesenteric arteries were isolated from 3 male transgenic mice and 4 non-transgenic littermate controls. Total RNA was prepared using conventional methods and quality was assessed using the Bioanalyzer 2100 (Agilent Technologies). For the microarray hybridizations, each sample corresponded to mesenteric artery derived from one mouse. All procedures were conducted at the University of Iowa DNA Core facility using standard Affymetrix protocols. In brief, approximately 50 ng of total RNA was used as input to a two-step amplification procedure (NuGen, http://www.nugeninc.com/) to generate biotin-labeled RNA fragments for hybridization to the Affymetrix mouse exon 1.0 ST array.&q;,&q;keywords&q;:null,&q;organisms&q;:null,&q;publicationDate&q;:&q;20120924&q;,&q;score&q;:&q;19.236282&q;,&q;omicsType&q;:[&q;Transcriptomics&q;,&q;Multiomics&q;],&q;citationsCount&q;:0,&q;connectionsCount&q;:0,&q;reanalysisCount&q;:0,&q;viewsCount&q;:0,&q;downloadCount&q;:0,&q;citationsCountScaled&q;:0,&q;connectionsCountScaled&q;:0,&q;reanalysisCountScaled&q;:0,&q;viewsCountScaled&q;:0,&q;downloadCountScaled&q;:0,&q;claimable&q;:false},{&q;id&q;:&q;GSE8949&q;,&q;source&q;:&q;geo&q;,&q;title&q;:&q;Gene expression changes in mouse aorta during activation of or interference with PPAR gamma signaling.&q;,&q;description&q;:&q;Ligand-mediated activation of the nuclear hormone receptor PPAR gamma lowers blood pressure and improves glucose tolerance in humans. Two naturally occurring mutations (P467L, V290M) in the ligand binding domain of PPAR gamma have been described in humans that lead to severe insulin resistance and hypertension. Experimental evidence suggests that these mutant versions of PPAR gamma act in a dominant negative fashion. To better understand the molecular mechanisms underlying PPAR gamma action in the vasculature, we determined the gene expression patterns in mouse aorta in response to activation or interference with the PPAR gamma signaling pathway. Keywords: time course, dose response&q;,&q;keywords&q;:null,&q;organisms&q;:null,&q;publicationDate&q;:&q;20071212&q;,&q;score&q;:&q;18.236683&q;,&q;omicsType&q;:[&q;Unknown&q;],&q;citationsCount&q;:0,&q;connectionsCount&q;:0,&q;reanalysisCount&q;:0,&q;viewsCount&q;:0,&q;downloadCount&q;:0,&q;citationsCountScaled&q;:0,&q;connectionsCountScaled&q;:0,&q;reanalysisCountScaled&q;:0,&q;viewsCountScaled&q;:0,&q;downloadCountScaled&q;:0,&q;claimable&q;:false},{&q;id&q;:&q;E-GEOD-8949&q;,&q;source&q;:&q;biostudies&q;,&q;title&q;:&q;Transcription profiling of mouse aorta during activation of or interference with PPAR gamma signaling.&q;,&q;description&q;:&q;Ligand-mediated activation of the nuclear hormone receptor PPAR gamma lowers blood pressure and improves glucose tolerance in humans.  Two naturally occurring mutations (P467L, V290M) in the ligand binding domain of PPAR gamma have been described in humans that lead to severe insulin resistance and hypertension.  Experimental evidence suggests that these mutant versions of PPAR gamma act in a dominant negative fashion.  To better understand the molecular mechanisms underlying PPAR gamma action in the vasculature, we determined the gene expression patterns in mouse aorta in response to activation or interference with the PPAR gamma signaling pathway. Experiment Overall Design: To assess the response to PPAR gamma interference, we used adult mice containing a dominant negative form of PPAR gamma. These mice have a targeted P465L mutation, which is equivalent to the P467L mutant, described in human patients. Wild-type littermates were used as the genetic control. The PPAR gamma signaling pathway was activated by administration of rosiglitazone for either 2 or 14 days to adult mice (C57BL/6J strain) at a dose of 3 or 10 mg/kg/day via the food. Control mice were fed standard mouse chow. For the microarray hybridizations, 2-3 biological replicates from each experimental group were used.  Biological replicates were RNA pooled from 8 different mouse aortas. All the microarray procedures were conducted at the University of Iowa DNA Core facility using standard Affymetrix protocols.  In brief, approximately 3 ug of total RNA was used as input to a one-step amplification procedure to generate biotin-labeled RNA fragments for hybridization to the Affymetrix GeneChip Mouse Genome 430 2.0 array.&q;,&q;keywords&q;:null,&q;organisms&q;:null,&q;publicationDate&q;:null,&q;score&q;:&q;18.117537&q;,&q;omicsType&q;:[&q;Unknown&q;],&q;citationsCount&q;:0,&q;connectionsCount&q;:0,&q;reanalysisCount&q;:0,&q;viewsCount&q;:0,&q;downloadCount&q;:0,&q;citationsCountScaled&q;:0,&q;connectionsCountScaled&q;:0,&q;reanalysisCountScaled&q;:0,&q;viewsCountScaled&q;:0,&q;downloadCountScaled&q;:0,&q;claimable&q;:false},{&q;id&q;:&q;PRJNA158653&q;,&q;source&q;:&q;omics_ena_project&q;,&q;title&q;:&q;Mus musculus&q;,&q;description&q;:&q;Gene expression profiling during interference with PPAR gamma signaling in thoracic aorta&q;,&q;keywords&q;:null,&q;organisms&q;:null,&q;publicationDate&q;:null,&q;score&q;:&q;17.273653&q;,&q;omicsType&q;:[&q;Unknown&q;,&q;Multiomics&q;],&q;citationsCount&q;:0,&q;connectionsCount&q;:0,&q;reanalysisCount&q;:0,&q;viewsCount&q;:0,&q;downloadCount&q;:0,&q;citationsCountScaled&q;:0,&q;connectionsCountScaled&q;:0,&q;reanalysisCountScaled&q;:0,&q;viewsCountScaled&q;:0,&q;downloadCountScaled&q;:0,&q;claimable&q;:false},{&q;id&q;:&q;E-GEOD-8949&q;,&q;source&q;:&q;arrayexpress-repository&q;,&q;title&q;:&q;Transcription profiling of mouse aorta during activation of or interference with PPAR gamma signaling.&q;,&q;description&q;:&q;Ligand-mediated activation of the nuclear hormone receptor PPAR gamma lowers blood pressure and improves glucose tolerance in humans.  Two naturally occurring mutations (P467L, V290M) in the ligand binding domain of PPAR gamma have been described in humans that lead to severe insulin resistance and hypertension.  Experimental evidence suggests that these mutant versions of PPAR gamma act in a dominant negative fashion.  To better understand the molecular mechanisms underlying PPAR gamma action in the vasculature, we determined the gene expression patterns in mouse aorta in response to activation or interference with the PPAR gamma signaling pathway. Experiment Overall Design: To assess the response to PPAR gamma interference, we used adult mice containing a dominant negative form of PPAR gamma. These mice have a targeted P465L mutation, which is equivalent to the P467L mutant, described in human patients. Wild-type littermates were used as the genetic control. The PPAR gamma signaling pathway was activated by administration of rosiglitazone for either 2 or 14 days to adult mice (C57BL/6J strain) at a dose of 3 or 10 mg/kg/day via the food. Control mice were fed standard mouse chow. For the microarray hybridizations, 2-3 biological replicates from each experimental group were used.  Biological replicates were RNA pooled from 8 different mouse aortas. All the microarray procedures were conducted at the University of Iowa DNA Core facility using standard Affymetrix protocols.  In brief, approximately 3 ug of total RNA was used as input to a one-step amplification procedure to generate biotin-labeled RNA fragments for hybridization to the Affymetrix GeneChip Mouse Genome 430 2.0 array.&q;,&q;keywords&q;:null,&q;organisms&q;:null,&q;publicationDate&q;:&q;20080616&q;,&q;score&q;:&q;17.064844&q;,&q;omicsType&q;:[&q;Transcriptomics&q;],&q;citationsCount&q;:0,&q;connectionsCount&q;:0,&q;reanalysisCount&q;:0,&q;viewsCount&q;:0,&q;downloadCount&q;:0,&q;citationsCountScaled&q;:0,&q;connectionsCountScaled&q;:0,&q;reanalysisCountScaled&q;:0,&q;viewsCountScaled&q;:0,&q;downloadCountScaled&q;:0,&q;claimable&q;:false}],&q;facets&q;:[]}}}</script><div class="cdk-live-announcer-element cdk-visually-hidden" aria-atomic="true" aria-live="polite"></div><div id="cdk-describedby-message-container" aria-hidden="true" style="display: none;"><div id="cdk-describedby-message-0">Previous page</div><div id="cdk-describedby-message-1">Next page</div><div id="cdk-describedby-message-2">First page</div><div id="cdk-describedby-message-3">Last page</div></div></body><grammarly-desktop-integration data-grammarly-shadow-root="true"></grammarly-desktop-integration></html>