{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# This Helper fetches de resolver dataset from the EBI identifiers.org platform, to transform it and put it into\n",
    "# the new MongoDB based data backend.\n",
    "# Author: Manuel Bernal Llinares <mbdebian@gmail.com>\n",
    "import time\n",
    "import random\n",
    "import pymongo\n",
    "import requests\n",
    "import threading\n",
    "import matplotlib.pyplot as plt\n",
    "import data.models.hq.registry as hqregistry\n",
    "\n",
    "%matplotlib inline"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Set up the environment\n",
    "identifiersorg_url_resolver_data_source = 'https://identifiers.org/rest/collections/expand'\n",
    "# TODO Maybe get this from environment variables later on when everything is working\n",
    "mongodb_username = 'devusername'\n",
    "mongodb_password = 'devpassword'\n",
    "mongodb_database = 'devdbregistry'\n",
    "mongodb_endpoint = 'mongodb://localhost:27017'\n",
    "mongodb_authendpoint = \"mongodb://{}:{}@localhost:27017\".format(mongodb_username, mongodb_password)\n",
    "# Collections\n",
    "collection_name_institution = 'institution'\n",
    "collection_name_location = 'location'\n",
    "collection_name_namespaceSynonym = 'namespace_synonym'\n",
    "collection_name_resource = 'resource'\n",
    "collection_name_namespace = 'namespace'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Helpers\n",
    "def make_rest_request_content_type_json(url):\n",
    "    # TODO - Magic number here!!!\n",
    "    n_attempts = 42\n",
    "    response = None\n",
    "    while n_attempts:\n",
    "        n_attempts -= 1\n",
    "        try:\n",
    "            response = requests.get(url, headers={\"Content-Type\": \"application/json\"})\n",
    "        except Exception as e:\n",
    "            # Any possible exception counts towards the attempt counter\n",
    "            # Random wait - TODO - Another magic number!!!\n",
    "            time.sleep(random.randint(30))\n",
    "            continue\n",
    "        if response.ok:\n",
    "            return response.json()\n",
    "        # Random wait - TODO - Another magic number!!!\n",
    "        time.sleep(random.randint(10))\n",
    "    response.raise_for_status()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Initialise pseudo-random number generator\n",
    "random.seed(time.time())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "---> DB Client instantiated\n",
      "\tDatabases: ['admin', 'config', 'local']\n"
     ]
    }
   ],
   "source": [
    "# Preparing Database\n",
    "dbclient = pymongo.MongoClient(mongodb_authendpoint)\n",
    "print(\"---> DB Client instantiated\")\n",
    "print(\"\\tDatabases: {}\".format(dbclient.list_database_names()))\n",
    "dbregistry = dbclient[mongodb_database]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Retrieve the Resolution Source dataset\n",
    "resolution_dataset_source = make_rest_request_content_type_json(identifiersorg_url_resolver_data_source)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "---> Processing Prefix 'MIR:00000002' - 'chebi'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59223, MIR:00000002, chebi)\n",
      "---> Processing Prefix 'MIR:00000003' - 'ensembl'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59224, MIR:00000003, ensembl)\n",
      "---> Processing Prefix 'MIR:00000004' - 'ec-code'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59225, MIR:00000004, ec-code)\n",
      "---> Processing Prefix 'MIR:00000005' - 'uniprot'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59226, MIR:00000005, uniprot)\n",
      "---> Processing Prefix 'MIR:00000006' - 'taxonomy'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59227, MIR:00000006, taxonomy)\n",
      "---> Processing Prefix 'MIR:00000007' - 'biomodels.db'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59228, MIR:00000007, biomodels.db)\n",
      "---> Processing Prefix 'MIR:00000008' - 'miriam.collection'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59229, MIR:00000008, miriam.collection)\n",
      "---> Processing Prefix 'MIR:00000009' - 'icd'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5922a, MIR:00000009, icd)\n",
      "---> Processing Prefix 'MIR:00000010' - 'intact'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5922b, MIR:00000010, intact)\n",
      "---> Processing Prefix 'MIR:00000011' - 'interpro'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5922c, MIR:00000011, interpro)\n",
      "---> Processing Prefix 'MIR:00000012' - 'kegg.pathway'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5922d, MIR:00000012, kegg.pathway)\n",
      "---> Processing Prefix 'MIR:00000013' - 'kegg.compound'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5922e, MIR:00000013, kegg.compound)\n",
      "---> Processing Prefix 'MIR:00000014' - 'kegg.reaction'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5922f, MIR:00000014, kegg.reaction)\n",
      "---> Processing Prefix 'MIR:00000015' - 'pubmed'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59230, MIR:00000015, pubmed)\n",
      "---> Processing Prefix 'MIR:00000016' - 'mim'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59231, MIR:00000016, mim)\n",
      "---> Processing Prefix 'MIR:00000017' - 'pirsf'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59232, MIR:00000017, pirsf)\n",
      "---> Processing Prefix 'MIR:00000018' - 'reactome'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59233, MIR:00000018, reactome)\n",
      "---> Processing Prefix 'MIR:00000019' - 'doi'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59234, MIR:00000019, doi)\n",
      "---> Processing Prefix 'MIR:00000020' - 'pdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59235, MIR:00000020, pdb)\n",
      "---> Processing Prefix 'MIR:00000022' - 'go'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59236, MIR:00000022, go)\n",
      "---> Processing Prefix 'MIR:00000023' - 'sgd'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59237, MIR:00000023, sgd)\n",
      "---> Processing Prefix 'MIR:00000024' - 'sbo'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59238, MIR:00000024, sbo)\n",
      "---> Processing Prefix 'MIR:00000025' - 'kegg.drug'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59239, MIR:00000025, kegg.drug)\n",
      "---> Processing Prefix 'MIR:00000026' - 'kegg.glycan'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5923a, MIR:00000026, kegg.glycan)\n",
      "---> Processing Prefix 'MIR:00000027' - 'wb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5923b, MIR:00000027, wb)\n",
      "---> Processing Prefix 'MIR:00000028' - 'pfam'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5923c, MIR:00000028, pfam)\n",
      "---> Processing Prefix 'MIR:00000029' - 'insdc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5923d, MIR:00000029, insdc)\n",
      "---> Processing Prefix 'MIR:00000030' - 'fb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5923e, MIR:00000030, fb)\n",
      "---> Processing Prefix 'MIR:00000031' - 'wormpep'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5923f, MIR:00000031, wormpep)\n",
      "---> Processing Prefix 'MIR:00000032' - 'prosite'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59240, MIR:00000032, prosite)\n",
      "---> Processing Prefix 'MIR:00000033' - 'pubchem.substance'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59241, MIR:00000033, pubchem.substance)\n",
      "---> Processing Prefix 'MIR:00000034' - 'pubchem.compound'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59242, MIR:00000034, pubchem.compound)\n",
      "---> Processing Prefix 'MIR:00000035' - 'arxiv'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59243, MIR:00000035, arxiv)\n",
      "---> Processing Prefix 'MIR:00000036' - 'arrayexpress'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59244, MIR:00000036, arrayexpress)\n",
      "---> Processing Prefix 'MIR:00000037' - 'mgi'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59245, MIR:00000037, mgi)\n",
      "---> Processing Prefix 'MIR:00000038' - 'sabiork.reaction'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59246, MIR:00000038, sabiork.reaction)\n",
      "---> Processing Prefix 'MIR:00000039' - 'refseq'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59247, MIR:00000039, refseq)\n",
      "---> Processing Prefix 'MIR:00000040' - 'tcdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59248, MIR:00000040, tcdb)\n",
      "---> Processing Prefix 'MIR:00000041' - 'uniparc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59249, MIR:00000041, uniparc)\n",
      "---> Processing Prefix 'MIR:00000042' - 'mint'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5924a, MIR:00000042, mint)\n",
      "---> Processing Prefix 'MIR:00000044' - 'dip'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5924b, MIR:00000044, dip)\n",
      "---> Processing Prefix 'MIR:00000045' - 'signaling-gateway'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5924c, MIR:00000045, signaling-gateway)\n",
      "---> Processing Prefix 'MIR:00000046' - 'resid'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5924d, MIR:00000046, resid)\n",
      "---> Processing Prefix 'MIR:00000047' - 'rgd'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5924e, MIR:00000047, rgd)\n",
      "---> Processing Prefix 'MIR:00000048' - 'tair.protein'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5924f, MIR:00000048, tair.protein)\n",
      "---> Processing Prefix 'MIR:00000049' - 'tair.gene'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59250, MIR:00000049, tair.gene)\n",
      "---> Processing Prefix 'MIR:00000050' - 'tair.locus'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59251, MIR:00000050, tair.locus)\n",
      "---> Processing Prefix 'MIR:00000051' - 'hmdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59252, MIR:00000051, hmdb)\n",
      "---> Processing Prefix 'MIR:00000052' - 'lipidmaps'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59253, MIR:00000052, lipidmaps)\n",
      "---> Processing Prefix 'MIR:00000053' - 'peptideatlas'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59254, MIR:00000053, peptideatlas)\n",
      "---> Processing Prefix 'MIR:00000054' - 'geo'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59255, MIR:00000054, geo)\n",
      "---> Processing Prefix 'MIR:00000055' - 'eco'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59256, MIR:00000055, eco)\n",
      "---> Processing Prefix 'MIR:00000056' - 'mod'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59257, MIR:00000056, mod)\n",
      "---> Processing Prefix 'MIR:00000057' - 'sgd.pathways'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59258, MIR:00000057, sgd.pathways)\n",
      "---> Processing Prefix 'MIR:00000058' - 'biogrid'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59259, MIR:00000058, biogrid)\n",
      "---> Processing Prefix 'MIR:00000059' - 'merops'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5925a, MIR:00000059, merops)\n",
      "---> Processing Prefix 'MIR:00000060' - 'panther.family'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5925b, MIR:00000060, panther.family)\n",
      "---> Processing Prefix 'MIR:00000061' - 'prints'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5925c, MIR:00000061, prints)\n",
      "---> Processing Prefix 'MIR:00000062' - 'ligandexpo'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5925d, MIR:00000062, ligandexpo)\n",
      "---> Processing Prefix 'MIR:00000063' - 'aclame'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5925e, MIR:00000063, aclame)\n",
      "---> Processing Prefix 'MIR:00000064' - 'isbn'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5925f, MIR:00000064, isbn)\n",
      "---> Processing Prefix 'MIR:00000065' - 'pride'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59260, MIR:00000065, pride)\n",
      "---> Processing Prefix 'MIR:00000066' - '3dmet'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59261, MIR:00000066, 3dmet)\n",
      "---> Processing Prefix 'MIR:00000067' - 'fma'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59262, MIR:00000067, fma)\n",
      "---> Processing Prefix 'MIR:00000068' - 'matrixdb.association'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59263, MIR:00000068, matrixdb.association)\n",
      "---> Processing Prefix 'MIR:00000069' - 'ncbigene'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59264, MIR:00000069, ncbigene)\n",
      "---> Processing Prefix 'MIR:00000070' - 'kegg.genes'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59265, MIR:00000070, kegg.genes)\n",
      "---> Processing Prefix 'MIR:00000071' - 'brenda'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59266, MIR:00000071, brenda)\n",
      "---> Processing Prefix 'MIR:00000072' - 'pubchem.bioassay'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59267, MIR:00000072, pubchem.bioassay)\n",
      "---> Processing Prefix 'MIR:00000073' - 'pathwaycommons'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59268, MIR:00000073, pathwaycommons)\n",
      "---> Processing Prefix 'MIR:00000074' - 'hovergen'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59269, MIR:00000074, hovergen)\n",
      "---> Processing Prefix 'MIR:00000075' - 'mmmp:biomaps'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5926a, MIR:00000075, mmmp:biomaps)\n",
      "---> Processing Prefix 'MIR:00000076' - 'wikipathways'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5926b, MIR:00000076, wikipathways)\n",
      "---> Processing Prefix 'MIR:00000077' - 'macie'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5926c, MIR:00000077, macie)\n",
      "---> Processing Prefix 'MIR:00000078' - 'mirbase'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5926d, MIR:00000078, mirbase)\n",
      "---> Processing Prefix 'MIR:00000079' - 'zfin'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5926e, MIR:00000079, zfin)\n",
      "---> Processing Prefix 'MIR:00000080' - 'hgnc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5926f, MIR:00000080, hgnc)\n",
      "---> Processing Prefix 'MIR:00000081' - 'so'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59270, MIR:00000081, so)\n",
      "---> Processing Prefix 'MIR:00000082' - 'rhea'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59271, MIR:00000082, rhea)\n",
      "---> Processing Prefix 'MIR:00000083' - 'unipathway'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59272, MIR:00000083, unipathway)\n",
      "---> Processing Prefix 'MIR:00000084' - 'chembl.compound'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59273, MIR:00000084, chembl.compound)\n",
      "---> Processing Prefix 'MIR:00000085' - 'chembl.target'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59274, MIR:00000085, chembl.target)\n",
      "---> Processing Prefix 'MIR:00000086' - 'sabiork.kineticrecord'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59275, MIR:00000086, sabiork.kineticrecord)\n",
      "---> Processing Prefix 'MIR:00000087' - 'lgic'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59276, MIR:00000087, lgic)\n",
      "---> Processing Prefix 'MIR:00000088' - 'atc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59277, MIR:00000088, atc)\n",
      "---> Processing Prefix 'MIR:00000089' - 'pharmgkb.pathways'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59278, MIR:00000089, pharmgkb.pathways)\n",
      "---> Processing Prefix 'MIR:00000090' - 'pharmgkb.disease'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59279, MIR:00000090, pharmgkb.disease)\n",
      "---> Processing Prefix 'MIR:00000091' - 'pharmgkb.drug'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5927a, MIR:00000091, pharmgkb.drug)\n",
      "---> Processing Prefix 'MIR:00000092' - 'ttd.drug'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5927b, MIR:00000092, ttd.drug)\n",
      "---> Processing Prefix 'MIR:00000093' - 'ttd.target'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5927c, MIR:00000093, ttd.target)\n",
      "---> Processing Prefix 'MIR:00000094' - 'neurondb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5927d, MIR:00000094, neurondb)\n",
      "---> Processing Prefix 'MIR:00000095' - 'neuromorpho'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5927e, MIR:00000095, neuromorpho)\n",
      "---> Processing Prefix 'MIR:00000096' - 'chemidplus'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5927f, MIR:00000096, chemidplus)\n",
      "---> Processing Prefix 'MIR:00000097' - 'biosystems'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59280, MIR:00000097, biosystems)\n",
      "---> Processing Prefix 'MIR:00000098' - 'ctd.chemical'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59281, MIR:00000098, ctd.chemical)\n",
      "---> Processing Prefix 'MIR:00000099' - 'ctd.disease'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59282, MIR:00000099, ctd.disease)\n",
      "---> Processing Prefix 'MIR:00000100' - 'ctd.gene'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59283, MIR:00000100, ctd.gene)\n",
      "---> Processing Prefix 'MIR:00000101' - 'bionumbers'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59284, MIR:00000101, bionumbers)\n",
      "---> Processing Prefix 'MIR:00000102' - 'drugbank'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59285, MIR:00000102, drugbank)\n",
      "---> Processing Prefix 'MIR:00000103' - 't3db'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59286, MIR:00000103, t3db)\n",
      "---> Processing Prefix 'MIR:00000104' - 'smpdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59287, MIR:00000104, smpdb)\n",
      "---> Processing Prefix 'MIR:00000105' - 'phosphosite.protein'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59288, MIR:00000105, phosphosite.protein)\n",
      "---> Processing Prefix 'MIR:00000106' - 'genedb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59289, MIR:00000106, genedb)\n",
      "---> Processing Prefix 'MIR:00000107' - 'biomodels.teddy'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5928a, MIR:00000107, biomodels.teddy)\n",
      "---> Processing Prefix 'MIR:00000108' - 'biomodels.kisao'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5928b, MIR:00000108, biomodels.kisao)\n",
      "---> Processing Prefix 'MIR:00000109' - 'psimi'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5928c, MIR:00000109, psimi)\n",
      "---> Processing Prefix 'MIR:00000110' - 'cl'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5928d, MIR:00000110, cl)\n",
      "---> Processing Prefix 'MIR:00000111' - 'bto'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5928e, MIR:00000111, bto)\n",
      "---> Processing Prefix 'MIR:00000112' - 'pato'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5928f, MIR:00000112, pato)\n",
      "---> Processing Prefix 'MIR:00000113' - 'pdb-ccd'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59290, MIR:00000113, pdb-ccd)\n",
      "---> Processing Prefix 'MIR:00000114' - 'glycomedb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59291, MIR:00000114, glycomedb)\n",
      "---> Processing Prefix 'MIR:00000115' - 'lipidbank'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59292, MIR:00000115, lipidbank)\n",
      "---> Processing Prefix 'MIR:00000116' - 'kegg.orthology'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59293, MIR:00000116, kegg.orthology)\n",
      "---> Processing Prefix 'MIR:00000117' - 'prodom'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59294, MIR:00000117, prodom)\n",
      "---> Processing Prefix 'MIR:00000118' - 'smart'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59295, MIR:00000118, smart)\n",
      "---> Processing Prefix 'MIR:00000119' - 'cdd'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59296, MIR:00000119, cdd)\n",
      "---> Processing Prefix 'MIR:00000120' - 'ro'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59297, MIR:00000120, ro)\n",
      "---> Processing Prefix 'MIR:00000121' - 'mmdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59298, MIR:00000121, mmdb)\n",
      "---> Processing Prefix 'MIR:00000122' - 'imex'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59299, MIR:00000122, imex)\n",
      "---> Processing Prefix 'MIR:00000123' - 'irefweb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5929a, MIR:00000123, irefweb)\n",
      "---> Processing Prefix 'MIR:00000124' - 'mpid'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5929b, MIR:00000124, mpid)\n",
      "---> Processing Prefix 'MIR:00000125' - 'phosphosite.residue'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5929c, MIR:00000125, phosphosite.residue)\n",
      "---> Processing Prefix 'MIR:00000126' - 'neurolex'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5929d, MIR:00000126, neurolex)\n",
      "---> Processing Prefix 'MIR:00000127' - 'obi'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5929e, MIR:00000127, obi)\n",
      "---> Processing Prefix 'MIR:00000128' - 'sabiork.ec'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5929f, MIR:00000128, sabiork.ec)\n",
      "---> Processing Prefix 'MIR:00000129' - 'opb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592a0, MIR:00000129, opb)\n",
      "---> Processing Prefix 'MIR:00000130' - 'jws'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592a1, MIR:00000130, jws)\n",
      "---> Processing Prefix 'MIR:00000131' - 'modeldb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592a2, MIR:00000131, modeldb)\n",
      "---> Processing Prefix 'MIR:00000132' - 'subtiwiki'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592a3, MIR:00000132, subtiwiki)\n",
      "---> Processing Prefix 'MIR:00000133' - 'pid.pathway'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592a4, MIR:00000133, pid.pathway)\n",
      "---> Processing Prefix 'MIR:00000134' - 'doqcs.model'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592a5, MIR:00000134, doqcs.model)\n",
      "---> Processing Prefix 'MIR:00000135' - 'doqcs.pathway'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592a6, MIR:00000135, doqcs.pathway)\n",
      "---> Processing Prefix 'MIR:00000136' - 'uo'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592a7, MIR:00000136, uo)\n",
      "---> Processing Prefix 'MIR:00000137' - 'clinicaltrials'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592a8, MIR:00000137, clinicaltrials)\n",
      "---> Processing Prefix 'MIR:00000138' - 'chemspider'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592a9, MIR:00000138, chemspider)\n",
      "---> Processing Prefix 'MIR:00000139' - 'ncit'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592aa, MIR:00000139, ncit)\n",
      "---> Processing Prefix 'MIR:00000140' - 'biocatalogue.service'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592ab, MIR:00000140, biocatalogue.service)\n",
      "---> Processing Prefix 'MIR:00000141' - 'pr'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592ac, MIR:00000141, pr)\n",
      "---> Processing Prefix 'MIR:00000142' - 'omia'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592ad, MIR:00000142, omia)\n",
      "---> Processing Prefix 'MIR:00000143' - 'chembank'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592ae, MIR:00000143, chembank)\n",
      "---> Processing Prefix 'MIR:00000144' - 'csa'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592af, MIR:00000144, csa)\n",
      "---> Processing Prefix 'MIR:00000145' - 'cgd'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592b0, MIR:00000145, cgd)\n",
      "---> Processing Prefix 'MIR:00000146' - 'antweb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592b1, MIR:00000146, antweb)\n",
      "---> Processing Prefix 'MIR:00000147' - 'pmc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592b2, MIR:00000147, pmc)\n",
      "---> Processing Prefix 'MIR:00000148' - 'amoebadb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592b3, MIR:00000148, amoebadb)\n",
      "---> Processing Prefix 'MIR:00000149' - 'cryptodb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592b4, MIR:00000149, cryptodb)\n",
      "---> Processing Prefix 'MIR:00000150' - 'plasmodb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592b5, MIR:00000150, plasmodb)\n",
      "---> Processing Prefix 'MIR:00000151' - 'giardiadb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592b6, MIR:00000151, giardiadb)\n",
      "---> Processing Prefix 'MIR:00000152' - 'microsporidia'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592b7, MIR:00000152, microsporidia)\n",
      "---> Processing Prefix 'MIR:00000153' - 'toxoplasma'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592b8, MIR:00000153, toxoplasma)\n",
      "---> Processing Prefix 'MIR:00000154' - 'trichdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592b9, MIR:00000154, trichdb)\n",
      "---> Processing Prefix 'MIR:00000155' - 'tritrypdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592ba, MIR:00000155, tritrypdb)\n",
      "---> Processing Prefix 'MIR:00000156' - 'bdgp.insertion'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592bb, MIR:00000156, bdgp.insertion)\n",
      "---> Processing Prefix 'MIR:00000157' - 'beetlebase'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592bc, MIR:00000157, beetlebase)\n",
      "---> Processing Prefix 'MIR:00000158' - 'bold.taxonomy'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592bd, MIR:00000158, bold.taxonomy)\n",
      "---> Processing Prefix 'MIR:00000159' - 'dbest'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592be, MIR:00000159, dbest)\n",
      "---> Processing Prefix 'MIR:00000160' - 'dbprobe'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592bf, MIR:00000160, dbprobe)\n",
      "---> Processing Prefix 'MIR:00000161' - 'dbsnp'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592c0, MIR:00000161, dbsnp)\n",
      "---> Processing Prefix 'MIR:00000162' - 'unists'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592c1, MIR:00000162, unists)\n",
      "---> Processing Prefix 'MIR:00000163' - 'ecogene'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592c2, MIR:00000163, ecogene)\n",
      "---> Processing Prefix 'MIR:00000164' - 'gabi'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592c3, MIR:00000164, gabi)\n",
      "---> Processing Prefix 'MIR:00000165' - 'greengenes'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592c4, MIR:00000165, greengenes)\n",
      "---> Processing Prefix 'MIR:00000166' - 'grin.taxonomy'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592c5, MIR:00000166, grin.taxonomy)\n",
      "---> Processing Prefix 'MIR:00000167' - 'hinv.locus'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592c6, MIR:00000167, hinv.locus)\n",
      "---> Processing Prefix 'MIR:00000168' - 'hinv.transcript'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592c7, MIR:00000168, hinv.transcript)\n",
      "---> Processing Prefix 'MIR:00000169' - 'hinv.protein'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592c8, MIR:00000169, hinv.protein)\n",
      "---> Processing Prefix 'MIR:00000170' - 'homd.seq'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592c9, MIR:00000170, homd.seq)\n",
      "---> Processing Prefix 'MIR:00000171' - 'homd.taxon'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592ca, MIR:00000171, homd.taxon)\n",
      "---> Processing Prefix 'MIR:00000172' - 'ird.segment'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592cb, MIR:00000172, ird.segment)\n",
      "---> Processing Prefix 'MIR:00000173' - 'isfinder'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592cc, MIR:00000173, isfinder)\n",
      "---> Processing Prefix 'MIR:00000174' - 'jcm'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592cd, MIR:00000174, jcm)\n",
      "---> Processing Prefix 'MIR:00000175' - 'img.taxon'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592ce, MIR:00000175, img.taxon)\n",
      "---> Processing Prefix 'MIR:00000176' - 'img.gene'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592cf, MIR:00000176, img.gene)\n",
      "---> Processing Prefix 'MIR:00000177' - 'maizegdb.locus'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592d0, MIR:00000177, maizegdb.locus)\n",
      "---> Processing Prefix 'MIR:00000178' - 'mycobank'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592d1, MIR:00000178, mycobank)\n",
      "---> Processing Prefix 'MIR:00000179' - 'nbrc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592d2, MIR:00000179, nbrc)\n",
      "---> Processing Prefix 'MIR:00000180' - 'pseudomonas'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592d3, MIR:00000180, pseudomonas)\n",
      "---> Processing Prefix 'MIR:00000181' - 'gramene.protein'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592d4, MIR:00000181, gramene.protein)\n",
      "---> Processing Prefix 'MIR:00000182' - 'gramene.gene'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592d5, MIR:00000182, gramene.gene)\n",
      "---> Processing Prefix 'MIR:00000183' - 'gramene.taxonomy'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592d6, MIR:00000183, gramene.taxonomy)\n",
      "---> Processing Prefix 'MIR:00000184' - 'gramene.qtl'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592d7, MIR:00000184, gramene.qtl)\n",
      "---> Processing Prefix 'MIR:00000185' - 'sgn'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592d8, MIR:00000185, sgn)\n",
      "---> Processing Prefix 'MIR:00000186' - 'xenbase'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592d9, MIR:00000186, xenbase)\n",
      "---> Processing Prefix 'MIR:00000187' - 'bioportal'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592da, MIR:00000187, bioportal)\n",
      "---> Processing Prefix 'MIR:00000188' - 'miriam.resource'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592db, MIR:00000188, miriam.resource)\n",
      "---> Processing Prefix 'MIR:00000189' - 'edam'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592dc, MIR:00000189, edam)\n",
      "---> Processing Prefix 'MIR:00000190' - 'pmdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592dd, MIR:00000190, pmdb)\n",
      "---> Processing Prefix 'MIR:00000193' - 'arachnoserver'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592de, MIR:00000193, arachnoserver)\n",
      "---> Processing Prefix 'MIR:00000194' - 'biocyc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592df, MIR:00000194, biocyc)\n",
      "---> Processing Prefix 'MIR:00000195' - 'cazy'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592e0, MIR:00000195, cazy)\n",
      "---> Processing Prefix 'MIR:00000196' - 'goa'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592e1, MIR:00000196, goa)\n",
      "---> Processing Prefix 'MIR:00000197' - 'paleodb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592e2, MIR:00000197, paleodb)\n",
      "---> Processing Prefix 'MIR:00000198' - 'compulyeast'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592e3, MIR:00000198, compulyeast)\n",
      "---> Processing Prefix 'MIR:00000199' - 'disprot'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592e4, MIR:00000199, disprot)\n",
      "---> Processing Prefix 'MIR:00000200' - 'echobase'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592e5, MIR:00000200, echobase)\n",
      "---> Processing Prefix 'MIR:00000201' - 'eggnog'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592e6, MIR:00000201, eggnog)\n",
      "---> Processing Prefix 'MIR:00000202' - 'ensembl.bacteria'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592e7, MIR:00000202, ensembl.bacteria)\n",
      "---> Processing Prefix 'MIR:00000203' - 'ensembl.protist'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592e8, MIR:00000203, ensembl.protist)\n",
      "---> Processing Prefix 'MIR:00000204' - 'ensembl.metazoa'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592e9, MIR:00000204, ensembl.metazoa)\n",
      "---> Processing Prefix 'MIR:00000205' - 'ensembl.plant'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592ea, MIR:00000205, ensembl.plant)\n",
      "---> Processing Prefix 'MIR:00000206' - 'ensembl.fungi'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592eb, MIR:00000206, ensembl.fungi)\n",
      "---> Processing Prefix 'MIR:00000207' - 'hcvdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592ec, MIR:00000207, hcvdb)\n",
      "---> Processing Prefix 'MIR:00000208' - 'genatlas'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592ed, MIR:00000208, genatlas)\n",
      "---> Processing Prefix 'MIR:00000209' - 'cath.superfamily'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592ee, MIR:00000209, cath.superfamily)\n",
      "---> Processing Prefix 'MIR:00000210' - 'cath.domain'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592ef, MIR:00000210, cath.domain)\n",
      "---> Processing Prefix 'MIR:00000211' - 'genefarm'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592f0, MIR:00000211, genefarm)\n",
      "---> Processing Prefix 'MIR:00000212' - 'gpcrdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592f1, MIR:00000212, gpcrdb)\n",
      "---> Processing Prefix 'MIR:00000213' - 'hogenom'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592f2, MIR:00000213, hogenom)\n",
      "---> Processing Prefix 'MIR:00000214' - 'genetree'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592f3, MIR:00000214, genetree)\n",
      "---> Processing Prefix 'MIR:00000215' - 'hssp'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592f4, MIR:00000215, hssp)\n",
      "---> Processing Prefix 'MIR:00000216' - 'myco.tuber'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592f5, MIR:00000216, myco.tuber)\n",
      "---> Processing Prefix 'MIR:00000217' - 'myco.lepra'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592f6, MIR:00000217, myco.lepra)\n",
      "---> Processing Prefix 'MIR:00000218' - 'myco.marinum'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592f7, MIR:00000218, myco.marinum)\n",
      "---> Processing Prefix 'MIR:00000219' - 'myco.smeg'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592f8, MIR:00000219, myco.smeg)\n",
      "---> Processing Prefix 'MIR:00000220' - 'orphanet'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592f9, MIR:00000220, orphanet)\n",
      "---> Processing Prefix 'MIR:00000221' - 'orthodb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592fa, MIR:00000221, orthodb)\n",
      "---> Processing Prefix 'MIR:00000222' - 'peroxibase'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592fb, MIR:00000222, peroxibase)\n",
      "---> Processing Prefix 'MIR:00000223' - 'phylomedb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592fc, MIR:00000223, phylomedb)\n",
      "---> Processing Prefix 'MIR:00000224' - 'pmap.substratedb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592fd, MIR:00000224, pmap.substratedb)\n",
      "---> Processing Prefix 'MIR:00000225' - 'pmap.cutdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592fe, MIR:00000225, pmap.cutdb)\n",
      "---> Processing Prefix 'MIR:00000226' - 'protclustdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c592ff, MIR:00000226, protclustdb)\n",
      "---> Processing Prefix 'MIR:00000227' - 'pmp'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59300, MIR:00000227, pmp)\n",
      "---> Processing Prefix 'MIR:00000228' - 'protonet.proteincard'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59301, MIR:00000228, protonet.proteincard)\n",
      "---> Processing Prefix 'MIR:00000229' - 'protonet.cluster'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59302, MIR:00000229, protonet.cluster)\n",
      "---> Processing Prefix 'MIR:00000230' - 'rebase'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59303, MIR:00000230, rebase)\n",
      "---> Processing Prefix 'MIR:00000231' - 'swiss-model'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59304, MIR:00000231, swiss-model)\n",
      "---> Processing Prefix 'MIR:00000232' - 'vectorbase'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59305, MIR:00000232, vectorbase)\n",
      "---> Processing Prefix 'MIR:00000233' - 'doid'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59306, MIR:00000233, doid)\n",
      "---> Processing Prefix 'MIR:00000234' - 'cco'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59307, MIR:00000234, cco)\n",
      "---> Processing Prefix 'MIR:00000235' - 'mirbase.mature'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59308, MIR:00000235, mirbase.mature)\n",
      "---> Processing Prefix 'MIR:00000236' - 'nextprot'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59309, MIR:00000236, nextprot)\n",
      "---> Processing Prefix 'MIR:00000237' - 'cas'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5930a, MIR:00000237, cas)\n",
      "---> Processing Prefix 'MIR:00000238' - 'kegg.genome'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5930b, MIR:00000238, kegg.genome)\n",
      "---> Processing Prefix 'MIR:00000239' - 'kegg.metagenome'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5930c, MIR:00000239, kegg.metagenome)\n",
      "---> Processing Prefix 'MIR:00000240' - 'narcis'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5930d, MIR:00000240, narcis)\n",
      "---> Processing Prefix 'MIR:00000241' - 'jcsd'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5930e, MIR:00000241, jcsd)\n",
      "---> Processing Prefix 'MIR:00000242' - 'pw'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5930f, MIR:00000242, pw)\n",
      "---> Processing Prefix 'MIR:00000243' - 'insdc.sra'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59310, MIR:00000243, insdc.sra)\n",
      "---> Processing Prefix 'MIR:00000244' - 'scretf'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59311, MIR:00000244, scretf)\n",
      "---> Processing Prefix 'MIR:00000245' - 'pharmgkb.gene'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59312, MIR:00000245, pharmgkb.gene)\n",
      "---> Processing Prefix 'MIR:00000246' - 'mirnest'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59313, MIR:00000246, mirnest)\n",
      "---> Processing Prefix 'MIR:00000247' - 'napp'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59314, MIR:00000247, napp)\n",
      "---> Processing Prefix 'MIR:00000248' - 'noncodev3'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59315, MIR:00000248, noncodev3)\n",
      "---> Processing Prefix 'MIR:00000249' - 'virsirna'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59316, MIR:00000249, virsirna)\n",
      "---> Processing Prefix 'MIR:00000250' - 'elm'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59317, MIR:00000250, elm)\n",
      "---> Processing Prefix 'MIR:00000251' - 'mimodb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59318, MIR:00000251, mimodb)\n",
      "---> Processing Prefix 'MIR:00000252' - 'sitex'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59319, MIR:00000252, sitex)\n",
      "---> Processing Prefix 'MIR:00000253' - 'bykdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5931a, MIR:00000253, bykdb)\n",
      "---> Processing Prefix 'MIR:00000254' - 'conoserver'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5931b, MIR:00000254, conoserver)\n",
      "---> Processing Prefix 'MIR:00000255' - 'topfind'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5931c, MIR:00000255, topfind)\n",
      "---> Processing Prefix 'MIR:00000256' - 'mipmod'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5931d, MIR:00000256, mipmod)\n",
      "---> Processing Prefix 'MIR:00000257' - 'cellimage'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5931e, MIR:00000257, cellimage)\n",
      "---> Processing Prefix 'MIR:00000258' - 'combine.specifications'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5931f, MIR:00000258, combine.specifications)\n",
      "---> Processing Prefix 'MIR:00000261' - 'cabri'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59320, MIR:00000261, cabri)\n",
      "---> Processing Prefix 'MIR:00000262' - 'cygd'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59321, MIR:00000262, cygd)\n",
      "---> Processing Prefix 'MIR:00000263' - 'huge'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59322, MIR:00000263, huge)\n",
      "---> Processing Prefix 'MIR:00000264' - 'bindingDB'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59323, MIR:00000264, bindingDB)\n",
      "---> Processing Prefix 'MIR:00000265' - 'string'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59324, MIR:00000265, string)\n",
      "---> Processing Prefix 'MIR:00000266' - 'stitch'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59325, MIR:00000266, stitch)\n",
      "---> Processing Prefix 'MIR:00000267' - 'atcvet'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59326, MIR:00000267, atcvet)\n",
      "---> Processing Prefix 'MIR:00000268' - 'phenolexplorer'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59327, MIR:00000268, phenolexplorer)\n",
      "---> Processing Prefix 'MIR:00000269' - 'snomedct'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59328, MIR:00000269, snomedct)\n",
      "---> Processing Prefix 'MIR:00000270' - 'mesh.2012'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59329, MIR:00000270, mesh.2012)\n",
      "---> Processing Prefix 'MIR:00000271' - 'knapsack'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5932a, MIR:00000271, knapsack)\n",
      "---> Processing Prefix 'MIR:00000272' - 'cdpd'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5932b, MIR:00000272, cdpd)\n",
      "---> Processing Prefix 'MIR:00000273' - 'massbank'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5932c, MIR:00000273, massbank)\n",
      "---> Processing Prefix 'MIR:00000274' - 'gmd'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5932d, MIR:00000274, gmd)\n",
      "---> Processing Prefix 'MIR:00000275' - 'homologene'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5932e, MIR:00000275, homologene)\n",
      "---> Processing Prefix 'MIR:00000276' - 'umbbd.compound'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5932f, MIR:00000276, umbbd.compound)\n",
      "---> Processing Prefix 'MIR:00000277' - 'abs'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59330, MIR:00000277, abs)\n",
      "---> Processing Prefix 'MIR:00000278' - 'apd'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59331, MIR:00000278, apd)\n",
      "---> Processing Prefix 'MIR:00000279' - 'chemdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59332, MIR:00000279, chemdb)\n",
      "---> Processing Prefix 'MIR:00000280' - 'dpv'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59333, MIR:00000280, dpv)\n",
      "---> Processing Prefix 'MIR:00000281' - 'iuphar.receptor'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59334, MIR:00000281, iuphar.receptor)\n",
      "---> Processing Prefix 'MIR:00000282' - 'aceview.worm'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59335, MIR:00000282, aceview.worm)\n",
      "---> Processing Prefix 'MIR:00000283' - 'asap'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59336, MIR:00000283, asap)\n",
      "---> Processing Prefix 'MIR:00000284' - 'atcc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59337, MIR:00000284, atcc)\n",
      "---> Processing Prefix 'MIR:00000285' - 'bdgp.est'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59338, MIR:00000285, bdgp.est)\n",
      "---> Processing Prefix 'MIR:00000286' - 'dictybase.gene'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59339, MIR:00000286, dictybase.gene)\n",
      "---> Processing Prefix 'MIR:00000287' - 'imgt.ligm'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5933a, MIR:00000287, imgt.ligm)\n",
      "---> Processing Prefix 'MIR:00000288' - 'worfdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5933b, MIR:00000288, worfdb)\n",
      "---> Processing Prefix 'MIR:00000289' - 'nextdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5933c, MIR:00000289, nextdb)\n",
      "---> Processing Prefix 'MIR:00000291' - 'soybase'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5933d, MIR:00000291, soybase)\n",
      "---> Processing Prefix 'MIR:00000292' - 'hamap'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5933e, MIR:00000292, hamap)\n",
      "---> Processing Prefix 'MIR:00000293' - 'rouge'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5933f, MIR:00000293, rouge)\n",
      "---> Processing Prefix 'MIR:00000294' - 'arrayexpress.platform'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59340, MIR:00000294, arrayexpress.platform)\n",
      "---> Processing Prefix 'MIR:00000295' - 'cgsc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59341, MIR:00000295, cgsc)\n",
      "---> Processing Prefix 'MIR:00000297' - 'dragondb.dna'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59342, MIR:00000297, dragondb.dna)\n",
      "---> Processing Prefix 'MIR:00000298' - 'dragondb.protein'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59343, MIR:00000298, dragondb.protein)\n",
      "---> Processing Prefix 'MIR:00000299' - 'dragondb.locus'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59344, MIR:00000299, dragondb.locus)\n",
      "---> Processing Prefix 'MIR:00000300' - 'dragondb.allele'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59345, MIR:00000300, dragondb.allele)\n",
      "---> Processing Prefix 'MIR:00000301' - 'issn'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59346, MIR:00000301, issn)\n",
      "---> Processing Prefix 'MIR:00000302' - 'merops.family'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59347, MIR:00000302, merops.family)\n",
      "---> Processing Prefix 'MIR:00000303' - 'mo'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59348, MIR:00000303, mo)\n",
      "---> Processing Prefix 'MIR:00000304' - 'nasc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59349, MIR:00000304, nasc)\n",
      "---> Processing Prefix 'MIR:00000305' - 'niaest'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5934a, MIR:00000305, niaest)\n",
      "---> Processing Prefix 'MIR:00000306' - 'pazar'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5934b, MIR:00000306, pazar)\n",
      "---> Processing Prefix 'MIR:00000307' - 'po'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5934c, MIR:00000307, po)\n",
      "---> Processing Prefix 'MIR:00000308' - 'rnamods'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5934d, MIR:00000308, rnamods)\n",
      "---> Processing Prefix 'MIR:00000312' - 'treebase'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5934e, MIR:00000312, treebase)\n",
      "---> Processing Prefix 'MIR:00000313' - 'tgd'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5934f, MIR:00000313, tgd)\n",
      "---> Processing Prefix 'MIR:00000315' - 'tigrfam'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59350, MIR:00000315, tigrfam)\n",
      "---> Processing Prefix 'MIR:00000316' - 'atfdb.family'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59351, MIR:00000316, atfdb.family)\n",
      "---> Processing Prefix 'MIR:00000317' - 'iuphar.family'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59352, MIR:00000317, iuphar.family)\n",
      "---> Processing Prefix 'MIR:00000318' - 'dbg2introns'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59353, MIR:00000318, dbg2introns)\n",
      "---> Processing Prefix 'MIR:00000319' - 'sdbs'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59354, MIR:00000319, sdbs)\n",
      "---> Processing Prefix 'MIR:00000320' - 'vbase2'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59355, MIR:00000320, vbase2)\n",
      "---> Processing Prefix 'MIR:00000321' - 'spike.map'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59356, MIR:00000321, spike.map)\n",
      "---> Processing Prefix 'MIR:00000322' - 'metlin'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59357, MIR:00000322, metlin)\n",
      "---> Processing Prefix 'MIR:00000323' - 'genecards'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59358, MIR:00000323, genecards)\n",
      "---> Processing Prefix 'MIR:00000324' - 'mmrrc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59359, MIR:00000324, mmrrc)\n",
      "---> Processing Prefix 'MIR:00000325' - 'umbbd.reaction'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5935a, MIR:00000325, umbbd.reaction)\n",
      "---> Processing Prefix 'MIR:00000326' - 'umbbd.enzyme'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5935b, MIR:00000326, umbbd.enzyme)\n",
      "---> Processing Prefix 'MIR:00000327' - 'umbbd.pathway'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5935c, MIR:00000327, umbbd.pathway)\n",
      "---> Processing Prefix 'MIR:00000328' - 'umbbd.rule'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5935d, MIR:00000328, umbbd.rule)\n",
      "---> Processing Prefix 'MIR:00000329' - 'mirex'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5935e, MIR:00000329, mirex)\n",
      "---> Processing Prefix 'MIR:00000330' - 'dictybase.est'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5935f, MIR:00000330, dictybase.est)\n",
      "---> Processing Prefix 'MIR:00000331' - 'imgt.hla'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59360, MIR:00000331, imgt.hla)\n",
      "---> Processing Prefix 'MIR:00000332' - 'flystock'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59361, MIR:00000332, flystock)\n",
      "---> Processing Prefix 'MIR:00000333' - 'opm'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59362, MIR:00000333, opm)\n",
      "---> Processing Prefix 'MIR:00000334' - 'allergome'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59363, MIR:00000334, allergome)\n",
      "---> Processing Prefix 'MIR:00000335' - 'pombase'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59364, MIR:00000335, pombase)\n",
      "---> Processing Prefix 'MIR:00000336' - 'hpa'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59365, MIR:00000336, hpa)\n",
      "---> Processing Prefix 'MIR:00000337' - 'jaxmice'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59366, MIR:00000337, jaxmice)\n",
      "---> Processing Prefix 'MIR:00000338' - 'ubio.namebank'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59367, MIR:00000338, ubio.namebank)\n",
      "---> Processing Prefix 'MIR:00000339' - 'yetfasco'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59368, MIR:00000339, yetfasco)\n",
      "---> Processing Prefix 'MIR:00000340' - 'tarbase'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59369, MIR:00000340, tarbase)\n",
      "---> Processing Prefix 'MIR:00000341' - 'charprot'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5936a, MIR:00000341, charprot)\n",
      "---> Processing Prefix 'MIR:00000342' - 'oma.protein'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5936b, MIR:00000342, oma.protein)\n",
      "---> Processing Prefix 'MIR:00000343' - 'oma.grp'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5936c, MIR:00000343, oma.grp)\n",
      "---> Processing Prefix 'MIR:00000344' - 'ncbiprotein'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5936d, MIR:00000344, ncbiprotein)\n",
      "---> Processing Prefix 'MIR:00000345' - 'genpept'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5936e, MIR:00000345, genpept)\n",
      "---> Processing Prefix 'MIR:00000346' - 'unigene'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5936f, MIR:00000346, unigene)\n",
      "---> Processing Prefix 'MIR:00000347' - 'bitterdb.rec'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59370, MIR:00000347, bitterdb.rec)\n",
      "---> Processing Prefix 'MIR:00000348' - 'bitterdb.cpd'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59371, MIR:00000348, bitterdb.cpd)\n",
      "---> Processing Prefix 'MIR:00000349' - 'bioproject'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59372, MIR:00000349, bioproject)\n",
      "---> Processing Prefix 'MIR:00000350' - 'biosample'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59373, MIR:00000350, biosample)\n",
      "---> Processing Prefix 'MIR:00000351' - 'piroplasma'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59374, MIR:00000351, piroplasma)\n",
      "---> Processing Prefix 'MIR:00000352' - 'unite'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59375, MIR:00000352, unite)\n",
      "---> Processing Prefix 'MIR:00000353' - 'ncim'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59376, MIR:00000353, ncim)\n",
      "---> Processing Prefix 'MIR:00000354' - 'proglyc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59377, MIR:00000354, proglyc)\n",
      "---> Processing Prefix 'MIR:00000355' - 'polbase'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59378, MIR:00000355, polbase)\n",
      "---> Processing Prefix 'MIR:00000356' - 'nuclearbd'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59379, MIR:00000356, nuclearbd)\n",
      "---> Processing Prefix 'MIR:00000357' - 'supfam'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5937a, MIR:00000357, supfam)\n",
      "---> Processing Prefix 'MIR:00000358' - 'ricegap'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5937b, MIR:00000358, ricegap)\n",
      "---> Processing Prefix 'MIR:00000359' - 'pina'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5937c, MIR:00000359, pina)\n",
      "---> Processing Prefix 'MIR:00000360' - 'tissuelist'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5937d, MIR:00000360, tissuelist)\n",
      "---> Processing Prefix 'MIR:00000361' - 'bacmap.biog'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5937e, MIR:00000361, bacmap.biog)\n",
      "---> Processing Prefix 'MIR:00000362' - 'hgnc.symbol'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5937f, MIR:00000362, hgnc.symbol)\n",
      "---> Processing Prefix 'MIR:00000363' - 'panther.pathway'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59380, MIR:00000363, panther.pathway)\n",
      "---> Processing Prefix 'MIR:00000364' - 'fairsharing'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59381, MIR:00000364, fairsharing)\n",
      "---> Processing Prefix 'MIR:00000365' - 'fungidb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59382, MIR:00000365, fungidb)\n",
      "---> Processing Prefix 'MIR:00000366' - 'darc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59383, MIR:00000366, darc)\n",
      "---> Processing Prefix 'MIR:00000367' - 'drsc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59384, MIR:00000367, drsc)\n",
      "---> Processing Prefix 'MIR:00000368' - 'oridb.schizo'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59385, MIR:00000368, oridb.schizo)\n",
      "---> Processing Prefix 'MIR:00000369' - 'oridb.sacch'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59386, MIR:00000369, oridb.sacch)\n",
      "---> Processing Prefix 'MIR:00000370' - 'pscdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59387, MIR:00000370, pscdb)\n",
      "---> Processing Prefix 'MIR:00000371' - 'scop'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59388, MIR:00000371, scop)\n",
      "---> Processing Prefix 'MIR:00000372' - 'ena.embl'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59389, MIR:00000372, ena.embl)\n",
      "---> Processing Prefix 'MIR:00000373' - 'dommino'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5938a, MIR:00000373, dommino)\n",
      "---> Processing Prefix 'MIR:00000374' - 'panther.node'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5938b, MIR:00000374, panther.node)\n",
      "---> Processing Prefix 'MIR:00000375' - 'ccds'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5938c, MIR:00000375, ccds)\n",
      "---> Processing Prefix 'MIR:00000376' - 'lrg'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5938d, MIR:00000376, lrg)\n",
      "---> Processing Prefix 'MIR:00000377' - 'hprd'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5938e, MIR:00000377, hprd)\n",
      "---> Processing Prefix 'MIR:00000378' - 'gxa.gene'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5938f, MIR:00000378, gxa.gene)\n",
      "---> Processing Prefix 'MIR:00000379' - 'gxa.expt'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59390, MIR:00000379, gxa.expt)\n",
      "---> Processing Prefix 'MIR:00000380' - 'metabolights'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59391, MIR:00000380, metabolights)\n",
      "---> Processing Prefix 'MIR:00000381' - 'nbn'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59392, MIR:00000381, nbn)\n",
      "---> Processing Prefix 'MIR:00000382' - 'orcid'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59393, MIR:00000382, orcid)\n",
      "---> Processing Prefix 'MIR:00000383' - 'inchi'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59394, MIR:00000383, inchi)\n",
      "---> Processing Prefix 'MIR:00000384' - 'wikipedia.en'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59395, MIR:00000384, wikipedia.en)\n",
      "---> Processing Prefix 'MIR:00000385' - 'phosphopoint.kinase'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59396, MIR:00000385, phosphopoint.kinase)\n",
      "---> Processing Prefix 'MIR:00000386' - 'phosphopoint.protein'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59397, MIR:00000386, phosphopoint.protein)\n",
      "---> Processing Prefix 'MIR:00000387' - 'inchikey'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59398, MIR:00000387, inchikey)\n",
      "---> Processing Prefix 'MIR:00000388' - 'uniprot.isoform'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59399, MIR:00000388, uniprot.isoform)\n",
      "---> Processing Prefix 'MIR:00000389' - 'kegg.environ'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5939a, MIR:00000389, kegg.environ)\n",
      "---> Processing Prefix 'MIR:00000390' - 'cldb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5939b, MIR:00000390, cldb)\n",
      "---> Processing Prefix 'MIR:00000391' - 'efo'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5939c, MIR:00000391, efo)\n",
      "---> Processing Prefix 'MIR:00000392' - 'hgmd'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5939d, MIR:00000392, hgmd)\n",
      "---> Processing Prefix 'MIR:00000393' - 'aphidbase.transcript'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5939e, MIR:00000393, aphidbase.transcript)\n",
      "---> Processing Prefix 'MIR:00000394' - 'affy.probeset'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5939f, MIR:00000394, affy.probeset)\n",
      "---> Processing Prefix 'MIR:00000395' - 'treefam'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593a0, MIR:00000395, treefam)\n",
      "---> Processing Prefix 'MIR:00000396' - 'caps'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593a1, MIR:00000396, caps)\n",
      "---> Processing Prefix 'MIR:00000397' - 'cubedb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593a2, MIR:00000397, cubedb)\n",
      "---> Processing Prefix 'MIR:00000398' - 'ideal'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593a3, MIR:00000398, ideal)\n",
      "---> Processing Prefix 'MIR:00000399' - 'stap'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593a4, MIR:00000399, stap)\n",
      "---> Processing Prefix 'MIR:00000400' - 'pocketome'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593a5, MIR:00000400, pocketome)\n",
      "---> Processing Prefix 'MIR:00000401' - 'gold.genome'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593a6, MIR:00000401, gold.genome)\n",
      "---> Processing Prefix 'MIR:00000402' - 'gold.meta'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593a7, MIR:00000402, gold.meta)\n",
      "---> Processing Prefix 'MIR:00000403' - 'bugbase.protocol'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593a8, MIR:00000403, bugbase.protocol)\n",
      "---> Processing Prefix 'MIR:00000404' - 'bugbase.expt'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593a9, MIR:00000404, bugbase.expt)\n",
      "---> Processing Prefix 'MIR:00000405' - 'tol'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593aa, MIR:00000405, tol)\n",
      "---> Processing Prefix 'MIR:00000406' - 'vario'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593ab, MIR:00000406, vario)\n",
      "---> Processing Prefix 'MIR:00000407' - 'vipr'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593ac, MIR:00000407, vipr)\n",
      "---> Processing Prefix 'MIR:00000408' - 'epd'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593ad, MIR:00000408, epd)\n",
      "---> Processing Prefix 'MIR:00000409' - 'rfam'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593ae, MIR:00000409, rfam)\n",
      "---> Processing Prefix 'MIR:00000410' - 'fbol'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593af, MIR:00000410, fbol)\n",
      "---> Processing Prefix 'MIR:00000411' - 'aftol.taxonomy'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593b0, MIR:00000411, aftol.taxonomy)\n",
      "---> Processing Prefix 'MIR:00000412' - 'aspgd.locus'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593b1, MIR:00000412, aspgd.locus)\n",
      "---> Processing Prefix 'MIR:00000413' - 'aspgd.protein'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593b2, MIR:00000413, aspgd.protein)\n",
      "---> Processing Prefix 'MIR:00000415' - 'autdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593b3, MIR:00000415, autdb)\n",
      "---> Processing Prefix 'MIR:00000416' - 'bacmap.map'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593b4, MIR:00000416, bacmap.map)\n",
      "---> Processing Prefix 'MIR:00000417' - 'bgee.family'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593b5, MIR:00000417, bgee.family)\n",
      "---> Processing Prefix 'MIR:00000418' - 'bgee.gene'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593b6, MIR:00000418, bgee.gene)\n",
      "---> Processing Prefix 'MIR:00000419' - 'bgee.stage'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593b7, MIR:00000419, bgee.stage)\n",
      "---> Processing Prefix 'MIR:00000420' - 'bgee.organ'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593b8, MIR:00000420, bgee.organ)\n",
      "---> Processing Prefix 'MIR:00000421' - 'biocarta.pathway'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593b9, MIR:00000421, biocarta.pathway)\n",
      "---> Processing Prefix 'MIR:00000422' - 'panther.pthcmp'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593ba, MIR:00000422, panther.pthcmp)\n",
      "---> Processing Prefix 'MIR:00000423' - 'gmd.profile'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593bb, MIR:00000423, gmd.profile)\n",
      "---> Processing Prefix 'MIR:00000424' - 'gmd.gcms'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593bc, MIR:00000424, gmd.gcms)\n",
      "---> Processing Prefix 'MIR:00000425' - 'gmd.ref'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593bd, MIR:00000425, gmd.ref)\n",
      "---> Processing Prefix 'MIR:00000426' - 'gmd.analyte'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593be, MIR:00000426, gmd.analyte)\n",
      "---> Processing Prefix 'MIR:00000427' - 'intact.molecule'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593bf, MIR:00000427, intact.molecule)\n",
      "---> Processing Prefix 'MIR:00000428' - 'depod'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593c0, MIR:00000428, depod)\n",
      "---> Processing Prefix 'MIR:00000429' - 'cst'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593c1, MIR:00000429, cst)\n",
      "---> Processing Prefix 'MIR:00000430' - 'cst.ab'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593c2, MIR:00000430, cst.ab)\n",
      "---> Processing Prefix 'MIR:00000431' - 'ndc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593c3, MIR:00000431, ndc)\n",
      "---> Processing Prefix 'MIR:00000432' - 'phytozome.locus'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593c4, MIR:00000432, phytozome.locus)\n",
      "---> Processing Prefix 'MIR:00000433' - 'subtilist'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593c5, MIR:00000433, subtilist)\n",
      "---> Processing Prefix 'MIR:00000434' - 'dailymed'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593c6, MIR:00000434, dailymed)\n",
      "---> Processing Prefix 'MIR:00000435' - 'sider.drug'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593c7, MIR:00000435, sider.drug)\n",
      "---> Processing Prefix 'MIR:00000436' - 'sider.effect'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593c8, MIR:00000436, sider.effect)\n",
      "---> Processing Prefix 'MIR:00000437' - 'wikigenes'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593c9, MIR:00000437, wikigenes)\n",
      "---> Processing Prefix 'MIR:00000438' - 'broad'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593ca, MIR:00000438, broad)\n",
      "---> Processing Prefix 'MIR:00000439' - 'coriell'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593cb, MIR:00000439, coriell)\n",
      "---> Processing Prefix 'MIR:00000440' - 'corum'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593cc, MIR:00000440, corum)\n",
      "---> Processing Prefix 'MIR:00000442' - 'ecoliwiki'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593cd, MIR:00000442, ecoliwiki)\n",
      "---> Processing Prefix 'MIR:00000443' - 'genprop'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593ce, MIR:00000443, genprop)\n",
      "---> Processing Prefix 'MIR:00000444' - 'jstor'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593cf, MIR:00000444, jstor)\n",
      "---> Processing Prefix 'MIR:00000445' - 'ma'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593d0, MIR:00000445, ma)\n",
      "---> Processing Prefix 'MIR:00000446' - 'uberon'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593d1, MIR:00000446, uberon)\n",
      "---> Processing Prefix 'MIR:00000447' - 'unimod'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593d2, MIR:00000447, unimod)\n",
      "---> Processing Prefix 'MIR:00000448' - 'vbrc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593d3, MIR:00000448, vbrc)\n",
      "---> Processing Prefix 'MIR:00000449' - 'viralzone'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593d4, MIR:00000449, viralzone)\n",
      "---> Processing Prefix 'MIR:00000450' - 'go.ref'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593d5, MIR:00000450, go.ref)\n",
      "---> Processing Prefix 'MIR:00000451' - 'rgd.qtl'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593d6, MIR:00000451, rgd.qtl)\n",
      "---> Processing Prefix 'MIR:00000452' - 'rgd.strain'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593d7, MIR:00000452, rgd.strain)\n",
      "---> Processing Prefix 'MIR:00000453' - 'door'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593d8, MIR:00000453, door)\n",
      "---> Processing Prefix 'MIR:00000454' - 'degradome'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593d9, MIR:00000454, degradome)\n",
      "---> Processing Prefix 'MIR:00000455' - 'dbd'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593da, MIR:00000455, dbd)\n",
      "---> Processing Prefix 'MIR:00000456' - 'datf'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593db, MIR:00000456, datf)\n",
      "---> Processing Prefix 'MIR:00000457' - 'iuphar.ligand'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593dc, MIR:00000457, iuphar.ligand)\n",
      "---> Processing Prefix 'MIR:00000458' - 'molbase'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593dd, MIR:00000458, molbase)\n",
      "---> Processing Prefix 'MIR:00000459' - 'yrcpdr'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593de, MIR:00000459, yrcpdr)\n",
      "---> Processing Prefix 'MIR:00000460' - 'yid'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593df, MIR:00000460, yid)\n",
      "---> Processing Prefix 'MIR:00000461' - 'funcbase.fly'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593e0, MIR:00000461, funcbase.fly)\n",
      "---> Processing Prefix 'MIR:00000462' - 'funcbase.human'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593e1, MIR:00000462, funcbase.human)\n",
      "---> Processing Prefix 'MIR:00000463' - 'funcbase.mouse'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593e2, MIR:00000463, funcbase.mouse)\n",
      "---> Processing Prefix 'MIR:00000464' - 'funcbase.yeast'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593e3, MIR:00000464, funcbase.yeast)\n",
      "---> Processing Prefix 'MIR:00000465' - 'ydpm'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593e4, MIR:00000465, ydpm)\n",
      "---> Processing Prefix 'MIR:00000466' - 'wb.rnai'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593e5, MIR:00000466, wb.rnai)\n",
      "---> Processing Prefix 'MIR:00000467' - 'funcat'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593e6, MIR:00000467, funcat)\n",
      "---> Processing Prefix 'MIR:00000468' - 'pass2'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593e7, MIR:00000468, pass2)\n",
      "---> Processing Prefix 'MIR:00000469' - 'iceberg.element'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593e8, MIR:00000469, iceberg.element)\n",
      "---> Processing Prefix 'MIR:00000470' - 'iceberg.family'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593e9, MIR:00000470, iceberg.family)\n",
      "---> Processing Prefix 'MIR:00000471' - 'vfdb.genus'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593ea, MIR:00000471, vfdb.genus)\n",
      "---> Processing Prefix 'MIR:00000472' - 'vfdb.gene'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593eb, MIR:00000472, vfdb.gene)\n",
      "---> Processing Prefix 'MIR:00000473' - 'mesh.2013'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593ec, MIR:00000473, mesh.2013)\n",
      "---> Processing Prefix 'MIR:00000474' - 'kegg.module'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593ed, MIR:00000474, kegg.module)\n",
      "---> Processing Prefix 'MIR:00000475' - 'kegg.disease'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593ee, MIR:00000475, kegg.disease)\n",
      "---> Processing Prefix 'MIR:00000476' - 'medlineplus'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593ef, MIR:00000476, medlineplus)\n",
      "---> Processing Prefix 'MIR:00000477' - 'ligandbox'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593f0, MIR:00000477, ligandbox)\n",
      "---> Processing Prefix 'MIR:00000478' - 'glycoepitope'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593f1, MIR:00000478, glycoepitope)\n",
      "---> Processing Prefix 'MIR:00000479' - 'jcggdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593f2, MIR:00000479, jcggdb)\n",
      "---> Processing Prefix 'MIR:00000480' - 'noncodev4.gene'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593f3, MIR:00000480, noncodev4.gene)\n",
      "---> Processing Prefix 'MIR:00000481' - 'noncodev4.rna'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593f4, MIR:00000481, noncodev4.rna)\n",
      "---> Processing Prefix 'MIR:00000482' - 'oryzabase.gene'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593f5, MIR:00000482, oryzabase.gene)\n",
      "---> Processing Prefix 'MIR:00000483' - 'oryzabase.mutant'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593f6, MIR:00000483, oryzabase.mutant)\n",
      "---> Processing Prefix 'MIR:00000484' - 'oryzabase.strain'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593f7, MIR:00000484, oryzabase.strain)\n",
      "---> Processing Prefix 'MIR:00000485' - 'oryzabase.stage'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593f8, MIR:00000485, oryzabase.stage)\n",
      "---> Processing Prefix 'MIR:00000486' - 'otl'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593f9, MIR:00000486, otl)\n",
      "---> Processing Prefix 'MIR:00000487' - 'genewiki'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593fa, MIR:00000487, genewiki)\n",
      "---> Processing Prefix 'MIR:00000488' - 'paxdb.organism'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593fb, MIR:00000488, paxdb.organism)\n",
      "---> Processing Prefix 'MIR:00000489' - 'paxdb.protein'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593fc, MIR:00000489, paxdb.protein)\n",
      "---> Processing Prefix 'MIR:00000490' - 'pdb.ligand'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593fd, MIR:00000490, pdb.ligand)\n",
      "---> Processing Prefix 'MIR:00000491' - 'merops.inhibitor'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593fe, MIR:00000491, merops.inhibitor)\n",
      "---> Processing Prefix 'MIR:00000492' - 'adw'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c593ff, MIR:00000492, adw)\n",
      "---> Processing Prefix 'MIR:00000493' - 'glida.gpcr'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59400, MIR:00000493, glida.gpcr)\n",
      "---> Processing Prefix 'MIR:00000494' - 'glida.ligand'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59401, MIR:00000494, glida.ligand)\n",
      "---> Processing Prefix 'MIR:00000495' - 'grsdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59402, MIR:00000495, grsdb)\n",
      "---> Processing Prefix 'MIR:00000496' - 'fsnp'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59403, MIR:00000496, fsnp)\n",
      "---> Processing Prefix 'MIR:00000497' - 'hdr'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59404, MIR:00000497, hdr)\n",
      "---> Processing Prefix 'MIR:00000498' - 'norine'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59405, MIR:00000498, norine)\n",
      "---> Processing Prefix 'MIR:00000499' - 'ordb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59406, MIR:00000499, ordb)\n",
      "---> Processing Prefix 'MIR:00000500' - 'odor'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59407, MIR:00000500, odor)\n",
      "---> Processing Prefix 'MIR:00000501' - 'p3db.protein'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59408, MIR:00000501, p3db.protein)\n",
      "---> Processing Prefix 'MIR:00000502' - 'p3db.site'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59409, MIR:00000502, p3db.site)\n",
      "---> Processing Prefix 'MIR:00000503' - 'topdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5940a, MIR:00000503, topdb)\n",
      "---> Processing Prefix 'MIR:00000504' - 'cattleqtldb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5940b, MIR:00000504, cattleqtldb)\n",
      "---> Processing Prefix 'MIR:00000505' - 'chickenqtldb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5940c, MIR:00000505, chickenqtldb)\n",
      "---> Processing Prefix 'MIR:00000506' - 'pigqtldb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5940d, MIR:00000506, pigqtldb)\n",
      "---> Processing Prefix 'MIR:00000507' - 'sheepqtldb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5940e, MIR:00000507, sheepqtldb)\n",
      "---> Processing Prefix 'MIR:00000508' - 'gramene.growthstage'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5940f, MIR:00000508, gramene.growthstage)\n",
      "---> Processing Prefix 'MIR:00000510' - 'mgnify.samp'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59410, MIR:00000510, mgnify.samp)\n",
      "---> Processing Prefix 'MIR:00000511' - 'ega.study'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59411, MIR:00000511, ega.study)\n",
      "---> Processing Prefix 'MIR:00000512' - 'ega.dataset'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59412, MIR:00000512, ega.dataset)\n",
      "---> Processing Prefix 'MIR:00000513' - 'px'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59413, MIR:00000513, px)\n",
      "---> Processing Prefix 'MIR:00000514' - 'biomodels.vocabulary'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59414, MIR:00000514, biomodels.vocabulary)\n",
      "---> Processing Prefix 'MIR:00000515' - 'pride.project'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59415, MIR:00000515, pride.project)\n",
      "---> Processing Prefix 'MIR:00000516' - 'antibodyregistry'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59416, MIR:00000516, antibodyregistry)\n",
      "---> Processing Prefix 'MIR:00000517' - 'mamo'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59417, MIR:00000517, mamo)\n",
      "---> Processing Prefix 'MIR:00000518' - 'eo'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59418, MIR:00000518, eo)\n",
      "---> Processing Prefix 'MIR:00000519' - 'idot'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59419, MIR:00000519, idot)\n",
      "---> Processing Prefix 'MIR:00000520' - 'hgnc.family'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5941a, MIR:00000520, hgnc.family)\n",
      "---> Processing Prefix 'MIR:00000521' - 'yeastintron'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5941b, MIR:00000521, yeastintron)\n",
      "---> Processing Prefix 'MIR:00000522' - 'ardb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5941c, MIR:00000522, ardb)\n",
      "---> Processing Prefix 'MIR:00000524' - 'proteomicsdb.protein'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5941d, MIR:00000524, proteomicsdb.protein)\n",
      "---> Processing Prefix 'MIR:00000525' - 'proteomicsdb.peptide'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5941e, MIR:00000525, proteomicsdb.peptide)\n",
      "---> Processing Prefix 'MIR:00000526' - 'hpm.protein'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5941f, MIR:00000526, hpm.protein)\n",
      "---> Processing Prefix 'MIR:00000527' - 'hpm.peptide'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59420, MIR:00000527, hpm.peptide)\n",
      "---> Processing Prefix 'MIR:00000528' - 'drugbankv4.target'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59421, MIR:00000528, drugbankv4.target)\n",
      "---> Processing Prefix 'MIR:00000529' - 'zinc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59422, MIR:00000529, zinc)\n",
      "---> Processing Prefix 'MIR:00000530' - 'foodb.compound'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59423, MIR:00000530, foodb.compound)\n",
      "---> Processing Prefix 'MIR:00000531' - 'unii'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59424, MIR:00000531, unii)\n",
      "---> Processing Prefix 'MIR:00000532' - 'orphanet.ordo'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59425, MIR:00000532, orphanet.ordo)\n",
      "---> Processing Prefix 'MIR:00000533' - 'psipar'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59426, MIR:00000533, psipar)\n",
      "---> Processing Prefix 'MIR:00000534' - 'clinvar.record'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59427, MIR:00000534, clinvar.record)\n",
      "---> Processing Prefix 'MIR:00000535' - 'mgnify.proj'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59428, MIR:00000535, mgnify.proj)\n",
      "---> Processing Prefix 'MIR:00000536' - 'euclinicaltrials'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59429, MIR:00000536, euclinicaltrials)\n",
      "---> Processing Prefix 'MIR:00000537' - 'google.patent'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5942a, MIR:00000537, google.patent)\n",
      "---> Processing Prefix 'MIR:00000538' - 'uspto'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5942b, MIR:00000538, uspto)\n",
      "---> Processing Prefix 'MIR:00000539' - 'cpc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5942c, MIR:00000539, cpc)\n",
      "---> Processing Prefix 'MIR:00000540' - 'gwascentral.study'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5942d, MIR:00000540, gwascentral.study)\n",
      "---> Processing Prefix 'MIR:00000541' - 'exac.variant'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5942e, MIR:00000541, exac.variant)\n",
      "---> Processing Prefix 'MIR:00000542' - 'gwascentral.marker'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5942f, MIR:00000542, gwascentral.marker)\n",
      "---> Processing Prefix 'MIR:00000543' - 'gwascentral.phenotype'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59430, MIR:00000543, gwascentral.phenotype)\n",
      "---> Processing Prefix 'MIR:00000544' - 'lincs.cell'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59431, MIR:00000544, lincs.cell)\n",
      "---> Processing Prefix 'MIR:00000545' - 'lincs.protein'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59432, MIR:00000545, lincs.protein)\n",
      "---> Processing Prefix 'MIR:00000546' - 'lincs.smallmolecule'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59433, MIR:00000546, lincs.smallmolecule)\n",
      "---> Processing Prefix 'MIR:00000547' - 'exac.transcript'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59434, MIR:00000547, exac.transcript)\n",
      "---> Processing Prefix 'MIR:00000548' - 'exac.gene'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59435, MIR:00000548, exac.gene)\n",
      "---> Processing Prefix 'MIR:00000549' - 'wikidata'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59436, MIR:00000549, wikidata)\n",
      "---> Processing Prefix 'MIR:00000550' - 'swisslipid'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59437, MIR:00000550, swisslipid)\n",
      "---> Processing Prefix 'MIR:00000551' - 'unipathway.compound'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59438, MIR:00000551, unipathway.compound)\n",
      "---> Processing Prefix 'MIR:00000552' - 'seed'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59439, MIR:00000552, seed)\n",
      "---> Processing Prefix 'MIR:00000553' - 'seed.compound'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5943a, MIR:00000553, seed.compound)\n",
      "---> Processing Prefix 'MIR:00000554' - 'bigg.model'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5943b, MIR:00000554, bigg.model)\n",
      "---> Processing Prefix 'MIR:00000555' - 'bigg.compartment'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5943c, MIR:00000555, bigg.compartment)\n",
      "---> Processing Prefix 'MIR:00000556' - 'bigg.metabolite'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5943d, MIR:00000556, bigg.metabolite)\n",
      "---> Processing Prefix 'MIR:00000557' - 'bigg.reaction'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5943e, MIR:00000557, bigg.reaction)\n",
      "---> Processing Prefix 'MIR:00000558' - 'rrid'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5943f, MIR:00000558, rrid)\n",
      "---> Processing Prefix 'MIR:00000559' - 'umls'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59440, MIR:00000559, umls)\n",
      "---> Processing Prefix 'MIR:00000560' - 'mesh'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59441, MIR:00000560, mesh)\n",
      "---> Processing Prefix 'MIR:00000561' - 'emdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59442, MIR:00000561, emdb)\n",
      "---> Processing Prefix 'MIR:00000562' - 'mirtarbase'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59443, MIR:00000562, mirtarbase)\n",
      "---> Processing Prefix 'MIR:00000563' - 'meddra'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59444, MIR:00000563, meddra)\n",
      "---> Processing Prefix 'MIR:00000564' - 'dashr'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59445, MIR:00000564, dashr)\n",
      "---> Processing Prefix 'MIR:00000565' - 'dashr.expression'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59446, MIR:00000565, dashr.expression)\n",
      "---> Processing Prefix 'MIR:00000566' - 'splash'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59447, MIR:00000566, splash)\n",
      "---> Processing Prefix 'MIR:00000567' - 'metanetx.chemical'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59448, MIR:00000567, metanetx.chemical)\n",
      "---> Processing Prefix 'MIR:00000568' - 'metanetx.reaction'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59449, MIR:00000568, metanetx.reaction)\n",
      "---> Processing Prefix 'MIR:00000569' - 'metanetx.compartment'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5944a, MIR:00000569, metanetx.compartment)\n",
      "---> Processing Prefix 'MIR:00000570' - 'unipathway.reaction'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5944b, MIR:00000570, unipathway.reaction)\n",
      "---> Processing Prefix 'MIR:00000571' - 'hp'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5944c, MIR:00000571, hp)\n",
      "---> Processing Prefix 'MIR:00000572' - 'sasbdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5944d, MIR:00000572, sasbdb)\n",
      "---> Processing Prefix 'MIR:00000573' - 'hgnc.genefamily'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5944e, MIR:00000573, hgnc.genefamily)\n",
      "---> Processing Prefix 'MIR:00000574' - 'mdm'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5944f, MIR:00000574, mdm)\n",
      "---> Processing Prefix 'MIR:00000575' - 'probonto'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59450, MIR:00000575, probonto)\n",
      "---> Processing Prefix 'MIR:00000576' - 'apid.interactions'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59451, MIR:00000576, apid.interactions)\n",
      "---> Processing Prefix 'MIR:00000577' - 'storedb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59452, MIR:00000577, storedb)\n",
      "---> Processing Prefix 'MIR:00000578' - 'kegg'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59453, MIR:00000578, kegg)\n",
      "---> Processing Prefix 'MIR:00000579' - 'planttfdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59454, MIR:00000579, planttfdb)\n",
      "---> Processing Prefix 'MIR:00000580' - 'mp'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59455, MIR:00000580, mp)\n",
      "---> Processing Prefix 'MIR:00000581' - 'worms'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59456, MIR:00000581, worms)\n",
      "---> Processing Prefix 'MIR:00000582' - 'multicellds.collection'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59457, MIR:00000582, multicellds.collection)\n",
      "---> Processing Prefix 'MIR:00000583' - 'multicellds.snapshot'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59458, MIR:00000583, multicellds.snapshot)\n",
      "---> Processing Prefix 'MIR:00000584' - 'multicellds.cell_line'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59459, MIR:00000584, multicellds.cell_line)\n",
      "---> Processing Prefix 'MIR:00000585' - 'ecyano.entity'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5945a, MIR:00000585, ecyano.entity)\n",
      "---> Processing Prefix 'MIR:00000586' - 'ecyano.model'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5945b, MIR:00000586, ecyano.model)\n",
      "---> Processing Prefix 'MIR:00000587' - 'ecyano.rule'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5945c, MIR:00000587, ecyano.rule)\n",
      "---> Processing Prefix 'MIR:00000588' - 'lincs.data'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5945d, MIR:00000588, lincs.data)\n",
      "---> Processing Prefix 'MIR:00000589' - 'agricola'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5945e, MIR:00000589, agricola)\n",
      "---> Processing Prefix 'MIR:00000590' - 'ms'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5945f, MIR:00000590, ms)\n",
      "---> Processing Prefix 'MIR:00000591' - 'envo'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59460, MIR:00000591, envo)\n",
      "---> Processing Prefix 'MIR:00000592' - 'ark'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59461, MIR:00000592, ark)\n",
      "---> Processing Prefix 'MIR:00000593' - 'grid'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59462, MIR:00000593, grid)\n",
      "---> Processing Prefix 'MIR:00000594' - 'medgen'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59463, MIR:00000594, medgen)\n",
      "---> Processing Prefix 'MIR:00000595' - 'clinvar.submission'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59464, MIR:00000595, clinvar.submission)\n",
      "---> Processing Prefix 'MIR:00000596' - 'clinvar'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59465, MIR:00000596, clinvar)\n",
      "---> Processing Prefix 'MIR:00000597' - 'bao'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59466, MIR:00000597, bao)\n",
      "---> Processing Prefix 'MIR:00000598' - 'cellosaurus'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59467, MIR:00000598, cellosaurus)\n",
      "---> Processing Prefix 'MIR:00000599' - 'mir'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59468, MIR:00000599, mir)\n",
      "---> Processing Prefix 'MIR:00000600' - 'pgx'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59469, MIR:00000600, pgx)\n",
      "---> Processing Prefix 'MIR:00000601' - 'cosmic'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5946a, MIR:00000601, cosmic)\n",
      "---> Processing Prefix 'MIR:00000602' - 'dbgap'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5946b, MIR:00000602, dbgap)\n",
      "---> Processing Prefix 'MIR:00000603' - 'iao'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5946c, MIR:00000603, iao)\n",
      "---> Processing Prefix 'MIR:00000604' - 'gdc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5946d, MIR:00000604, gdc)\n",
      "---> Processing Prefix 'MIR:00000605' - 'omit'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5946e, MIR:00000605, omit)\n",
      "---> Processing Prefix 'MIR:00000606' - 'insdc.cds'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5946f, MIR:00000606, insdc.cds)\n",
      "---> Processing Prefix 'MIR:00000607' - 'insdc.gca'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59470, MIR:00000607, insdc.gca)\n",
      "---> Processing Prefix 'MIR:00000608' - 'glytoucan'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59471, MIR:00000608, glytoucan)\n",
      "---> Processing Prefix 'MIR:00000609' - 'metacyc.compound'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59472, MIR:00000609, metacyc.compound)\n",
      "---> Processing Prefix 'MIR:00000610' - 'metacyc.reaction'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59473, MIR:00000610, metacyc.reaction)\n",
      "---> Processing Prefix 'MIR:00000611' - 'd1id'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59474, MIR:00000611, d1id)\n",
      "---> Processing Prefix 'MIR:00000612' - 'cath'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59475, MIR:00000612, cath)\n",
      "---> Processing Prefix 'MIR:00000613' - 'mmp.ref'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59476, MIR:00000613, mmp.ref)\n",
      "---> Processing Prefix 'MIR:00000614' - 'mmp.db'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59477, MIR:00000614, mmp.db)\n",
      "---> Processing Prefix 'MIR:00000615' - 'mmp.cat'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59478, MIR:00000615, mmp.cat)\n",
      "---> Processing Prefix 'MIR:00000616' - 'biotools'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59479, MIR:00000616, biotools)\n",
      "---> Processing Prefix 'MIR:00000617' - 'aop'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5947a, MIR:00000617, aop)\n",
      "---> Processing Prefix 'MIR:00000618' - 'ngl'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5947b, MIR:00000618, ngl)\n",
      "---> Processing Prefix 'MIR:00000619' - 'erv'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5947c, MIR:00000619, erv)\n",
      "---> Processing Prefix 'MIR:00000620' - 'napdi'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5947d, MIR:00000620, napdi)\n",
      "---> Processing Prefix 'MIR:00000621' - 'arraymap'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5947e, MIR:00000621, arraymap)\n",
      "---> Processing Prefix 'MIR:00000622' - 'begdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5947f, MIR:00000622, begdb)\n",
      "---> Processing Prefix 'MIR:00000623' - 'cameo'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59480, MIR:00000623, cameo)\n",
      "---> Processing Prefix 'MIR:00000624' - 'sisu'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59481, MIR:00000624, sisu)\n",
      "---> Processing Prefix 'MIR:00000625' - 'mzspec'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59482, MIR:00000625, mzspec)\n",
      "---> Processing Prefix 'MIR:00000626' - 'crisprdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59483, MIR:00000626, crisprdb)\n",
      "---> Processing Prefix 'MIR:00000627' - 'gnpis'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59484, MIR:00000627, gnpis)\n",
      "---> Processing Prefix 'MIR:00000628' - 'licebase'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59485, MIR:00000628, licebase)\n",
      "---> Processing Prefix 'MIR:00000629' - 'sugarbind'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59486, MIR:00000629, sugarbind)\n",
      "---> Processing Prefix 'MIR:00000630' - 'rnacentral'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59487, MIR:00000630, rnacentral)\n",
      "---> Processing Prefix 'MIR:00000631' - 'microscope'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59488, MIR:00000631, microscope)\n",
      "---> Processing Prefix 'MIR:00000632' - 'swissregulon'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59489, MIR:00000632, swissregulon)\n",
      "---> Processing Prefix 'MIR:00000633' - 'validatordb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5948a, MIR:00000633, validatordb)\n",
      "---> Processing Prefix 'MIR:00000634' - 'biominder'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5948b, MIR:00000634, biominder)\n",
      "---> Processing Prefix 'MIR:00000635' - 'neurovault.collection'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5948c, MIR:00000635, neurovault.collection)\n",
      "---> Processing Prefix 'MIR:00000636' - 'vmhmetabolite'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5948d, MIR:00000636, vmhmetabolite)\n",
      "---> Processing Prefix 'MIR:00000637' - 'mobidb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5948e, MIR:00000637, mobidb)\n",
      "---> Processing Prefix 'MIR:00000638' - 'neurovault.image'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5948f, MIR:00000638, neurovault.image)\n",
      "---> Processing Prefix 'MIR:00000639' - 'nmr'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59490, MIR:00000639, nmr)\n",
      "---> Processing Prefix 'MIR:00000640' - 'vmhreaction'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59491, MIR:00000640, vmhreaction)\n",
      "---> Processing Prefix 'MIR:00000641' - 'ascl'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59492, MIR:00000641, ascl)\n",
      "---> Processing Prefix 'MIR:00000642' - 'facebase'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59493, MIR:00000642, facebase)\n",
      "---> Processing Prefix 'MIR:00000643' - 'gudmap'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59494, MIR:00000643, gudmap)\n",
      "---> Processing Prefix 'MIR:00000644' - 'eu89h'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59495, MIR:00000644, eu89h)\n",
      "---> Processing Prefix 'MIR:00000645' - 'minid'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59496, MIR:00000645, minid)\n",
      "---> Processing Prefix 'MIR:00000646' - 'mw.study'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59497, MIR:00000646, mw.study)\n",
      "---> Processing Prefix 'MIR:00000647' - 'mw.project'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59498, MIR:00000647, mw.project)\n",
      "---> Processing Prefix 'MIR:00000648' - 'mex'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c59499, MIR:00000648, mex)\n",
      "---> Processing Prefix 'MIR:00000649' - 'gpmdb'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5949a, MIR:00000649, gpmdb)\n",
      "---> Processing Prefix 'MIR:00000650' - 'massive'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5949b, MIR:00000650, massive)\n",
      "---> Processing Prefix 'MIR:00000651' - 'fplx'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5949c, MIR:00000651, fplx)\n",
      "---> Processing Prefix 'MIR:00000652' - 'aop.events'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5949d, MIR:00000652, aop.events)\n",
      "---> Processing Prefix 'MIR:00000653' - 'aop.relationships'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5949e, MIR:00000653, aop.relationships)\n",
      "---> Processing Prefix 'MIR:00000654' - 'aop.stressor'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c5949f, MIR:00000654, aop.stressor)\n",
      "---> Processing Prefix 'MIR:00000655' - 'swh'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594a0, MIR:00000655, swh)\n",
      "---> Processing Prefix 'MIR:00000656' - 'dev.ga4ghdos'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594a1, MIR:00000656, dev.ga4ghdos)\n",
      "---> Processing Prefix 'MIR:00000657' - 'complexportal'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594a2, MIR:00000657, complexportal)\n",
      "---> Processing Prefix 'MIR:00000658' - 'comptox'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594a3, MIR:00000658, comptox)\n",
      "---> Processing Prefix 'MIR:00000659' - 'rbk'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594a4, MIR:00000659, rbk)\n",
      "---> Processing Prefix 'MIR:00000660' - 'oci'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594a5, MIR:00000660, oci)\n",
      "---> Processing Prefix 'MIR:00000661' - 'foodon'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594a6, MIR:00000661, foodon)\n",
      "---> Processing Prefix 'MIR:00000662' - 'encode'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594a7, MIR:00000662, encode)\n",
      "---> Processing Prefix 'MIR:00000663' - 'gtex'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594a8, MIR:00000663, gtex)\n",
      "---> Processing Prefix 'MIR:00000664' - 'ricenetdb.gene'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594a9, MIR:00000664, ricenetdb.gene)\n",
      "---> Processing Prefix 'MIR:00000665' - 'ricenetdb.protein'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594aa, MIR:00000665, ricenetdb.protein)\n",
      "---> Processing Prefix 'MIR:00000666' - 'ricenetdb.compound'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594ab, MIR:00000666, ricenetdb.compound)\n",
      "---> Processing Prefix 'MIR:00000667' - 'ricenetdb.reaction'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594ac, MIR:00000667, ricenetdb.reaction)\n",
      "---> Processing Prefix 'MIR:00000668' - 'ricenetdb.miRNA'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594ad, MIR:00000668, ricenetdb.miRNA)\n",
      "---> Processing Prefix 'MIR:00000669' - 'oclc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594ae, MIR:00000669, oclc)\n",
      "---> Processing Prefix 'MIR:00000670' - 'asin'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594af, MIR:00000670, asin)\n",
      "---> Processing Prefix 'MIR:00000671' - 'ga4ghdos'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594b0, MIR:00000671, ga4ghdos)\n",
      "---> Processing Prefix 'MIR:00000672' - 'flowrepository'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594b1, MIR:00000672, flowrepository)\n",
      "---> Processing Prefix 'MIR:00000673' - 'oid'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594b2, MIR:00000673, oid)\n",
      "---> Processing Prefix 'MIR:00000674' - 'hpscreg'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594b3, MIR:00000674, hpscreg)\n",
      "---> Processing Prefix 'MIR:00000675' - 'addgene'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594b4, MIR:00000675, addgene)\n",
      "---> Processing Prefix 'MIR:00000676' - 'bacdive'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594b5, MIR:00000676, bacdive)\n",
      "---> Processing Prefix 'MIR:00000677' - 'ido'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594b6, MIR:00000677, ido)\n",
      "---> Processing Prefix 'MIR:00000678' - 'gdsc'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594b7, MIR:00000678, gdsc)\n",
      "---> Processing Prefix 'MIR:00000679' - 'cbioportal'\n",
      "\tNamespace inserted, (5bc0b686bc79c2dd14c594b8, MIR:00000679, cbioportal)\n"
     ]
    }
   ],
   "source": [
    "# Process PID entries\n",
    "mongodb_collection = dbregistry[collection_name_namespace]\n",
    "for pid_entry in resolution_dataset_source:\n",
    "    print(\"---> Processing Prefix '{}' - '{}'\".format(pid_entry['id'], pid_entry['prefix']))\n",
    "    # Translate PID Entry to Namespace model\n",
    "    namespace = hqregistry.Namespace()\n",
    "    namespace.prefix = pid_entry.get('prefix', None)\n",
    "    namespace.mirId = pid_entry.get('id', None)\n",
    "    namespace.name = pid_entry.get('name', None)\n",
    "    namespace.pattern = pid_entry.get('pattern', None)\n",
    "    namespace.description = pid_entry.get('definition', None)\n",
    "    # TODO creation and modification dates are missing from the dataset\n",
    "    # There is no deprecation date for now\n",
    "    # Flag all namespaces as 'NOT Deprecated' for now\n",
    "    namespace.deprecated = False\n",
    "    inserted_object = mongodb_collection.insert_one(namespace.__dict__)\n",
    "    print(\"\\tNamespace inserted, ({}, {}, {})\".format(inserted_object.inserted_id, namespace.mirId, namespace.prefix))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
