{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# This Helper fetches de resolver dataset from the EBI identifiers.org platform, to transform it and put it into\n",
    "# the new MongoDB based data backend.\n",
    "# Author: Manuel Bernal Llinares <mbdebian@gmail.com>\n",
    "import time\n",
    "import random\n",
    "import requests\n",
    "import threading\n",
    "import matplotlib.pyplot as plt\n",
    "import data.models.hq.registry as hqregistry\n",
    "\n",
    "%matplotlib inline"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Set up the environment\n",
    "identifiersorg_url_resolver_data_source = 'https://identifiers.org/rest/collections/expand'\n",
    "# TODO Maybe get this from environment variables later on when everything is working\n",
    "mongodb_username = 'devusername'\n",
    "mongodb_password = 'devpassword'\n",
    "mongodb_database = 'devdbregistry'\n",
    "mongodb_endpoint = 'mongodb://localhost:27017'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Helpers\n",
    "def make_rest_request_content_type_json(url):\n",
    "    # TODO - Magic number here!!!\n",
    "    n_attempts = 42\n",
    "    response = None\n",
    "    while n_attempts:\n",
    "        n_attempts -= 1\n",
    "        try:\n",
    "            response = requests.get(url, headers={\"Content-Type\": \"application/json\"})\n",
    "        except Exception as e:\n",
    "            # Any possible exception counts towards the attempt counter\n",
    "            # Random wait - TODO - Another magic number!!!\n",
    "            time.sleep(random.randint(30))\n",
    "            continue\n",
    "        if response.ok:\n",
    "            return response.json()\n",
    "        # Random wait - TODO - Another magic number!!!\n",
    "        time.sleep(random.randint(10))\n",
    "    response.raise_for_status()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Initialise pseudo-random number generator\n",
    "random.seed(time.time())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Retrieve the Resolution Source dataset\n",
    "resolution_dataset_source = make_rest_request_content_type_json(identifiersorg_url_resolver_data_source)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "---> Processing Prefix 'MIR:00000002' - 'chebi'\n",
      "---> Processing Prefix 'MIR:00000003' - 'ensembl'\n",
      "---> Processing Prefix 'MIR:00000004' - 'ec-code'\n",
      "---> Processing Prefix 'MIR:00000005' - 'uniprot'\n",
      "---> Processing Prefix 'MIR:00000006' - 'taxonomy'\n",
      "---> Processing Prefix 'MIR:00000007' - 'biomodels.db'\n",
      "---> Processing Prefix 'MIR:00000008' - 'miriam.collection'\n",
      "---> Processing Prefix 'MIR:00000009' - 'icd'\n",
      "---> Processing Prefix 'MIR:00000010' - 'intact'\n",
      "---> Processing Prefix 'MIR:00000011' - 'interpro'\n",
      "---> Processing Prefix 'MIR:00000012' - 'kegg.pathway'\n",
      "---> Processing Prefix 'MIR:00000013' - 'kegg.compound'\n",
      "---> Processing Prefix 'MIR:00000014' - 'kegg.reaction'\n",
      "---> Processing Prefix 'MIR:00000015' - 'pubmed'\n",
      "---> Processing Prefix 'MIR:00000016' - 'mim'\n",
      "---> Processing Prefix 'MIR:00000017' - 'pirsf'\n",
      "---> Processing Prefix 'MIR:00000018' - 'reactome'\n",
      "---> Processing Prefix 'MIR:00000019' - 'doi'\n",
      "---> Processing Prefix 'MIR:00000020' - 'pdb'\n",
      "---> Processing Prefix 'MIR:00000022' - 'go'\n",
      "---> Processing Prefix 'MIR:00000023' - 'sgd'\n",
      "---> Processing Prefix 'MIR:00000024' - 'sbo'\n",
      "---> Processing Prefix 'MIR:00000025' - 'kegg.drug'\n",
      "---> Processing Prefix 'MIR:00000026' - 'kegg.glycan'\n",
      "---> Processing Prefix 'MIR:00000027' - 'wb'\n",
      "---> Processing Prefix 'MIR:00000028' - 'pfam'\n",
      "---> Processing Prefix 'MIR:00000029' - 'insdc'\n",
      "---> Processing Prefix 'MIR:00000030' - 'fb'\n",
      "---> Processing Prefix 'MIR:00000031' - 'wormpep'\n",
      "---> Processing Prefix 'MIR:00000032' - 'prosite'\n",
      "---> Processing Prefix 'MIR:00000033' - 'pubchem.substance'\n",
      "---> Processing Prefix 'MIR:00000034' - 'pubchem.compound'\n",
      "---> Processing Prefix 'MIR:00000035' - 'arxiv'\n",
      "---> Processing Prefix 'MIR:00000036' - 'arrayexpress'\n",
      "---> Processing Prefix 'MIR:00000037' - 'mgi'\n",
      "---> Processing Prefix 'MIR:00000038' - 'sabiork.reaction'\n",
      "---> Processing Prefix 'MIR:00000039' - 'refseq'\n",
      "---> Processing Prefix 'MIR:00000040' - 'tcdb'\n",
      "---> Processing Prefix 'MIR:00000041' - 'uniparc'\n",
      "---> Processing Prefix 'MIR:00000042' - 'mint'\n",
      "---> Processing Prefix 'MIR:00000044' - 'dip'\n",
      "---> Processing Prefix 'MIR:00000045' - 'signaling-gateway'\n",
      "---> Processing Prefix 'MIR:00000046' - 'resid'\n",
      "---> Processing Prefix 'MIR:00000047' - 'rgd'\n",
      "---> Processing Prefix 'MIR:00000048' - 'tair.protein'\n",
      "---> Processing Prefix 'MIR:00000049' - 'tair.gene'\n",
      "---> Processing Prefix 'MIR:00000050' - 'tair.locus'\n",
      "---> Processing Prefix 'MIR:00000051' - 'hmdb'\n",
      "---> Processing Prefix 'MIR:00000052' - 'lipidmaps'\n",
      "---> Processing Prefix 'MIR:00000053' - 'peptideatlas'\n",
      "---> Processing Prefix 'MIR:00000054' - 'geo'\n",
      "---> Processing Prefix 'MIR:00000055' - 'eco'\n",
      "---> Processing Prefix 'MIR:00000056' - 'mod'\n",
      "---> Processing Prefix 'MIR:00000057' - 'sgd.pathways'\n",
      "---> Processing Prefix 'MIR:00000058' - 'biogrid'\n",
      "---> Processing Prefix 'MIR:00000059' - 'merops'\n",
      "---> Processing Prefix 'MIR:00000060' - 'panther.family'\n",
      "---> Processing Prefix 'MIR:00000061' - 'prints'\n",
      "---> Processing Prefix 'MIR:00000062' - 'ligandexpo'\n",
      "---> Processing Prefix 'MIR:00000063' - 'aclame'\n",
      "---> Processing Prefix 'MIR:00000064' - 'isbn'\n",
      "---> Processing Prefix 'MIR:00000065' - 'pride'\n",
      "---> Processing Prefix 'MIR:00000066' - '3dmet'\n",
      "---> Processing Prefix 'MIR:00000067' - 'fma'\n",
      "---> Processing Prefix 'MIR:00000068' - 'matrixdb.association'\n",
      "---> Processing Prefix 'MIR:00000069' - 'ncbigene'\n",
      "---> Processing Prefix 'MIR:00000070' - 'kegg.genes'\n",
      "---> Processing Prefix 'MIR:00000071' - 'brenda'\n",
      "---> Processing Prefix 'MIR:00000072' - 'pubchem.bioassay'\n",
      "---> Processing Prefix 'MIR:00000073' - 'pathwaycommons'\n",
      "---> Processing Prefix 'MIR:00000074' - 'hovergen'\n",
      "---> Processing Prefix 'MIR:00000075' - 'mmmp:biomaps'\n",
      "---> Processing Prefix 'MIR:00000076' - 'wikipathways'\n",
      "---> Processing Prefix 'MIR:00000077' - 'macie'\n",
      "---> Processing Prefix 'MIR:00000078' - 'mirbase'\n",
      "---> Processing Prefix 'MIR:00000079' - 'zfin'\n",
      "---> Processing Prefix 'MIR:00000080' - 'hgnc'\n",
      "---> Processing Prefix 'MIR:00000081' - 'so'\n",
      "---> Processing Prefix 'MIR:00000082' - 'rhea'\n",
      "---> Processing Prefix 'MIR:00000083' - 'unipathway'\n",
      "---> Processing Prefix 'MIR:00000084' - 'chembl.compound'\n",
      "---> Processing Prefix 'MIR:00000085' - 'chembl.target'\n",
      "---> Processing Prefix 'MIR:00000086' - 'sabiork.kineticrecord'\n",
      "---> Processing Prefix 'MIR:00000087' - 'lgic'\n",
      "---> Processing Prefix 'MIR:00000088' - 'atc'\n",
      "---> Processing Prefix 'MIR:00000089' - 'pharmgkb.pathways'\n",
      "---> Processing Prefix 'MIR:00000090' - 'pharmgkb.disease'\n",
      "---> Processing Prefix 'MIR:00000091' - 'pharmgkb.drug'\n",
      "---> Processing Prefix 'MIR:00000092' - 'ttd.drug'\n",
      "---> Processing Prefix 'MIR:00000093' - 'ttd.target'\n",
      "---> Processing Prefix 'MIR:00000094' - 'neurondb'\n",
      "---> Processing Prefix 'MIR:00000095' - 'neuromorpho'\n",
      "---> Processing Prefix 'MIR:00000096' - 'chemidplus'\n",
      "---> Processing Prefix 'MIR:00000097' - 'biosystems'\n",
      "---> Processing Prefix 'MIR:00000098' - 'ctd.chemical'\n",
      "---> Processing Prefix 'MIR:00000099' - 'ctd.disease'\n",
      "---> Processing Prefix 'MIR:00000100' - 'ctd.gene'\n",
      "---> Processing Prefix 'MIR:00000101' - 'bionumbers'\n",
      "---> Processing Prefix 'MIR:00000102' - 'drugbank'\n",
      "---> Processing Prefix 'MIR:00000103' - 't3db'\n",
      "---> Processing Prefix 'MIR:00000104' - 'smpdb'\n",
      "---> Processing Prefix 'MIR:00000105' - 'phosphosite.protein'\n",
      "---> Processing Prefix 'MIR:00000106' - 'genedb'\n",
      "---> Processing Prefix 'MIR:00000107' - 'biomodels.teddy'\n",
      "---> Processing Prefix 'MIR:00000108' - 'biomodels.kisao'\n",
      "---> Processing Prefix 'MIR:00000109' - 'psimi'\n",
      "---> Processing Prefix 'MIR:00000110' - 'cl'\n",
      "---> Processing Prefix 'MIR:00000111' - 'bto'\n",
      "---> Processing Prefix 'MIR:00000112' - 'pato'\n",
      "---> Processing Prefix 'MIR:00000113' - 'pdb-ccd'\n",
      "---> Processing Prefix 'MIR:00000114' - 'glycomedb'\n",
      "---> Processing Prefix 'MIR:00000115' - 'lipidbank'\n",
      "---> Processing Prefix 'MIR:00000116' - 'kegg.orthology'\n",
      "---> Processing Prefix 'MIR:00000117' - 'prodom'\n",
      "---> Processing Prefix 'MIR:00000118' - 'smart'\n",
      "---> Processing Prefix 'MIR:00000119' - 'cdd'\n",
      "---> Processing Prefix 'MIR:00000120' - 'ro'\n",
      "---> Processing Prefix 'MIR:00000121' - 'mmdb'\n",
      "---> Processing Prefix 'MIR:00000122' - 'imex'\n",
      "---> Processing Prefix 'MIR:00000123' - 'irefweb'\n",
      "---> Processing Prefix 'MIR:00000124' - 'mpid'\n",
      "---> Processing Prefix 'MIR:00000125' - 'phosphosite.residue'\n",
      "---> Processing Prefix 'MIR:00000126' - 'neurolex'\n",
      "---> Processing Prefix 'MIR:00000127' - 'obi'\n",
      "---> Processing Prefix 'MIR:00000128' - 'sabiork.ec'\n",
      "---> Processing Prefix 'MIR:00000129' - 'opb'\n",
      "---> Processing Prefix 'MIR:00000130' - 'jws'\n",
      "---> Processing Prefix 'MIR:00000131' - 'modeldb'\n",
      "---> Processing Prefix 'MIR:00000132' - 'subtiwiki'\n",
      "---> Processing Prefix 'MIR:00000133' - 'pid.pathway'\n",
      "---> Processing Prefix 'MIR:00000134' - 'doqcs.model'\n",
      "---> Processing Prefix 'MIR:00000135' - 'doqcs.pathway'\n",
      "---> Processing Prefix 'MIR:00000136' - 'uo'\n",
      "---> Processing Prefix 'MIR:00000137' - 'clinicaltrials'\n",
      "---> Processing Prefix 'MIR:00000138' - 'chemspider'\n",
      "---> Processing Prefix 'MIR:00000139' - 'ncit'\n",
      "---> Processing Prefix 'MIR:00000140' - 'biocatalogue.service'\n",
      "---> Processing Prefix 'MIR:00000141' - 'pr'\n",
      "---> Processing Prefix 'MIR:00000142' - 'omia'\n",
      "---> Processing Prefix 'MIR:00000143' - 'chembank'\n",
      "---> Processing Prefix 'MIR:00000144' - 'csa'\n",
      "---> Processing Prefix 'MIR:00000145' - 'cgd'\n",
      "---> Processing Prefix 'MIR:00000146' - 'antweb'\n",
      "---> Processing Prefix 'MIR:00000147' - 'pmc'\n",
      "---> Processing Prefix 'MIR:00000148' - 'amoebadb'\n",
      "---> Processing Prefix 'MIR:00000149' - 'cryptodb'\n",
      "---> Processing Prefix 'MIR:00000150' - 'plasmodb'\n",
      "---> Processing Prefix 'MIR:00000151' - 'giardiadb'\n",
      "---> Processing Prefix 'MIR:00000152' - 'microsporidia'\n",
      "---> Processing Prefix 'MIR:00000153' - 'toxoplasma'\n",
      "---> Processing Prefix 'MIR:00000154' - 'trichdb'\n",
      "---> Processing Prefix 'MIR:00000155' - 'tritrypdb'\n",
      "---> Processing Prefix 'MIR:00000156' - 'bdgp.insertion'\n",
      "---> Processing Prefix 'MIR:00000157' - 'beetlebase'\n",
      "---> Processing Prefix 'MIR:00000158' - 'bold.taxonomy'\n",
      "---> Processing Prefix 'MIR:00000159' - 'dbest'\n",
      "---> Processing Prefix 'MIR:00000160' - 'dbprobe'\n",
      "---> Processing Prefix 'MIR:00000161' - 'dbsnp'\n",
      "---> Processing Prefix 'MIR:00000162' - 'unists'\n",
      "---> Processing Prefix 'MIR:00000163' - 'ecogene'\n",
      "---> Processing Prefix 'MIR:00000164' - 'gabi'\n",
      "---> Processing Prefix 'MIR:00000165' - 'greengenes'\n",
      "---> Processing Prefix 'MIR:00000166' - 'grin.taxonomy'\n",
      "---> Processing Prefix 'MIR:00000167' - 'hinv.locus'\n",
      "---> Processing Prefix 'MIR:00000168' - 'hinv.transcript'\n",
      "---> Processing Prefix 'MIR:00000169' - 'hinv.protein'\n",
      "---> Processing Prefix 'MIR:00000170' - 'homd.seq'\n",
      "---> Processing Prefix 'MIR:00000171' - 'homd.taxon'\n",
      "---> Processing Prefix 'MIR:00000172' - 'ird.segment'\n",
      "---> Processing Prefix 'MIR:00000173' - 'isfinder'\n",
      "---> Processing Prefix 'MIR:00000174' - 'jcm'\n",
      "---> Processing Prefix 'MIR:00000175' - 'img.taxon'\n",
      "---> Processing Prefix 'MIR:00000176' - 'img.gene'\n",
      "---> Processing Prefix 'MIR:00000177' - 'maizegdb.locus'\n",
      "---> Processing Prefix 'MIR:00000178' - 'mycobank'\n",
      "---> Processing Prefix 'MIR:00000179' - 'nbrc'\n",
      "---> Processing Prefix 'MIR:00000180' - 'pseudomonas'\n",
      "---> Processing Prefix 'MIR:00000181' - 'gramene.protein'\n",
      "---> Processing Prefix 'MIR:00000182' - 'gramene.gene'\n",
      "---> Processing Prefix 'MIR:00000183' - 'gramene.taxonomy'\n",
      "---> Processing Prefix 'MIR:00000184' - 'gramene.qtl'\n",
      "---> Processing Prefix 'MIR:00000185' - 'sgn'\n",
      "---> Processing Prefix 'MIR:00000186' - 'xenbase'\n",
      "---> Processing Prefix 'MIR:00000187' - 'bioportal'\n",
      "---> Processing Prefix 'MIR:00000188' - 'miriam.resource'\n",
      "---> Processing Prefix 'MIR:00000189' - 'edam'\n",
      "---> Processing Prefix 'MIR:00000190' - 'pmdb'\n",
      "---> Processing Prefix 'MIR:00000193' - 'arachnoserver'\n",
      "---> Processing Prefix 'MIR:00000194' - 'biocyc'\n",
      "---> Processing Prefix 'MIR:00000195' - 'cazy'\n",
      "---> Processing Prefix 'MIR:00000196' - 'goa'\n",
      "---> Processing Prefix 'MIR:00000197' - 'paleodb'\n",
      "---> Processing Prefix 'MIR:00000198' - 'compulyeast'\n",
      "---> Processing Prefix 'MIR:00000199' - 'disprot'\n",
      "---> Processing Prefix 'MIR:00000200' - 'echobase'\n",
      "---> Processing Prefix 'MIR:00000201' - 'eggnog'\n",
      "---> Processing Prefix 'MIR:00000202' - 'ensembl.bacteria'\n",
      "---> Processing Prefix 'MIR:00000203' - 'ensembl.protist'\n",
      "---> Processing Prefix 'MIR:00000204' - 'ensembl.metazoa'\n",
      "---> Processing Prefix 'MIR:00000205' - 'ensembl.plant'\n",
      "---> Processing Prefix 'MIR:00000206' - 'ensembl.fungi'\n",
      "---> Processing Prefix 'MIR:00000207' - 'hcvdb'\n",
      "---> Processing Prefix 'MIR:00000208' - 'genatlas'\n",
      "---> Processing Prefix 'MIR:00000209' - 'cath.superfamily'\n",
      "---> Processing Prefix 'MIR:00000210' - 'cath.domain'\n",
      "---> Processing Prefix 'MIR:00000211' - 'genefarm'\n",
      "---> Processing Prefix 'MIR:00000212' - 'gpcrdb'\n",
      "---> Processing Prefix 'MIR:00000213' - 'hogenom'\n",
      "---> Processing Prefix 'MIR:00000214' - 'genetree'\n",
      "---> Processing Prefix 'MIR:00000215' - 'hssp'\n",
      "---> Processing Prefix 'MIR:00000216' - 'myco.tuber'\n",
      "---> Processing Prefix 'MIR:00000217' - 'myco.lepra'\n",
      "---> Processing Prefix 'MIR:00000218' - 'myco.marinum'\n",
      "---> Processing Prefix 'MIR:00000219' - 'myco.smeg'\n",
      "---> Processing Prefix 'MIR:00000220' - 'orphanet'\n",
      "---> Processing Prefix 'MIR:00000221' - 'orthodb'\n",
      "---> Processing Prefix 'MIR:00000222' - 'peroxibase'\n",
      "---> Processing Prefix 'MIR:00000223' - 'phylomedb'\n",
      "---> Processing Prefix 'MIR:00000224' - 'pmap.substratedb'\n",
      "---> Processing Prefix 'MIR:00000225' - 'pmap.cutdb'\n",
      "---> Processing Prefix 'MIR:00000226' - 'protclustdb'\n",
      "---> Processing Prefix 'MIR:00000227' - 'pmp'\n",
      "---> Processing Prefix 'MIR:00000228' - 'protonet.proteincard'\n",
      "---> Processing Prefix 'MIR:00000229' - 'protonet.cluster'\n",
      "---> Processing Prefix 'MIR:00000230' - 'rebase'\n",
      "---> Processing Prefix 'MIR:00000231' - 'swiss-model'\n",
      "---> Processing Prefix 'MIR:00000232' - 'vectorbase'\n",
      "---> Processing Prefix 'MIR:00000233' - 'doid'\n",
      "---> Processing Prefix 'MIR:00000234' - 'cco'\n",
      "---> Processing Prefix 'MIR:00000235' - 'mirbase.mature'\n",
      "---> Processing Prefix 'MIR:00000236' - 'nextprot'\n",
      "---> Processing Prefix 'MIR:00000237' - 'cas'\n",
      "---> Processing Prefix 'MIR:00000238' - 'kegg.genome'\n",
      "---> Processing Prefix 'MIR:00000239' - 'kegg.metagenome'\n",
      "---> Processing Prefix 'MIR:00000240' - 'narcis'\n",
      "---> Processing Prefix 'MIR:00000241' - 'jcsd'\n",
      "---> Processing Prefix 'MIR:00000242' - 'pw'\n",
      "---> Processing Prefix 'MIR:00000243' - 'insdc.sra'\n",
      "---> Processing Prefix 'MIR:00000244' - 'scretf'\n",
      "---> Processing Prefix 'MIR:00000245' - 'pharmgkb.gene'\n",
      "---> Processing Prefix 'MIR:00000246' - 'mirnest'\n",
      "---> Processing Prefix 'MIR:00000247' - 'napp'\n",
      "---> Processing Prefix 'MIR:00000248' - 'noncodev3'\n",
      "---> Processing Prefix 'MIR:00000249' - 'virsirna'\n",
      "---> Processing Prefix 'MIR:00000250' - 'elm'\n",
      "---> Processing Prefix 'MIR:00000251' - 'mimodb'\n",
      "---> Processing Prefix 'MIR:00000252' - 'sitex'\n",
      "---> Processing Prefix 'MIR:00000253' - 'bykdb'\n",
      "---> Processing Prefix 'MIR:00000254' - 'conoserver'\n",
      "---> Processing Prefix 'MIR:00000255' - 'topfind'\n",
      "---> Processing Prefix 'MIR:00000256' - 'mipmod'\n",
      "---> Processing Prefix 'MIR:00000257' - 'cellimage'\n",
      "---> Processing Prefix 'MIR:00000258' - 'combine.specifications'\n",
      "---> Processing Prefix 'MIR:00000261' - 'cabri'\n",
      "---> Processing Prefix 'MIR:00000262' - 'cygd'\n",
      "---> Processing Prefix 'MIR:00000263' - 'huge'\n",
      "---> Processing Prefix 'MIR:00000264' - 'bindingDB'\n",
      "---> Processing Prefix 'MIR:00000265' - 'string'\n",
      "---> Processing Prefix 'MIR:00000266' - 'stitch'\n",
      "---> Processing Prefix 'MIR:00000267' - 'atcvet'\n",
      "---> Processing Prefix 'MIR:00000268' - 'phenolexplorer'\n",
      "---> Processing Prefix 'MIR:00000269' - 'snomedct'\n",
      "---> Processing Prefix 'MIR:00000270' - 'mesh.2012'\n",
      "---> Processing Prefix 'MIR:00000271' - 'knapsack'\n",
      "---> Processing Prefix 'MIR:00000272' - 'cdpd'\n",
      "---> Processing Prefix 'MIR:00000273' - 'massbank'\n",
      "---> Processing Prefix 'MIR:00000274' - 'gmd'\n",
      "---> Processing Prefix 'MIR:00000275' - 'homologene'\n",
      "---> Processing Prefix 'MIR:00000276' - 'umbbd.compound'\n",
      "---> Processing Prefix 'MIR:00000277' - 'abs'\n",
      "---> Processing Prefix 'MIR:00000278' - 'apd'\n",
      "---> Processing Prefix 'MIR:00000279' - 'chemdb'\n",
      "---> Processing Prefix 'MIR:00000280' - 'dpv'\n",
      "---> Processing Prefix 'MIR:00000281' - 'iuphar.receptor'\n",
      "---> Processing Prefix 'MIR:00000282' - 'aceview.worm'\n",
      "---> Processing Prefix 'MIR:00000283' - 'asap'\n",
      "---> Processing Prefix 'MIR:00000284' - 'atcc'\n",
      "---> Processing Prefix 'MIR:00000285' - 'bdgp.est'\n",
      "---> Processing Prefix 'MIR:00000286' - 'dictybase.gene'\n",
      "---> Processing Prefix 'MIR:00000287' - 'imgt.ligm'\n",
      "---> Processing Prefix 'MIR:00000288' - 'worfdb'\n",
      "---> Processing Prefix 'MIR:00000289' - 'nextdb'\n",
      "---> Processing Prefix 'MIR:00000291' - 'soybase'\n",
      "---> Processing Prefix 'MIR:00000292' - 'hamap'\n",
      "---> Processing Prefix 'MIR:00000293' - 'rouge'\n",
      "---> Processing Prefix 'MIR:00000294' - 'arrayexpress.platform'\n",
      "---> Processing Prefix 'MIR:00000295' - 'cgsc'\n",
      "---> Processing Prefix 'MIR:00000297' - 'dragondb.dna'\n",
      "---> Processing Prefix 'MIR:00000298' - 'dragondb.protein'\n",
      "---> Processing Prefix 'MIR:00000299' - 'dragondb.locus'\n",
      "---> Processing Prefix 'MIR:00000300' - 'dragondb.allele'\n",
      "---> Processing Prefix 'MIR:00000301' - 'issn'\n",
      "---> Processing Prefix 'MIR:00000302' - 'merops.family'\n",
      "---> Processing Prefix 'MIR:00000303' - 'mo'\n",
      "---> Processing Prefix 'MIR:00000304' - 'nasc'\n",
      "---> Processing Prefix 'MIR:00000305' - 'niaest'\n",
      "---> Processing Prefix 'MIR:00000306' - 'pazar'\n",
      "---> Processing Prefix 'MIR:00000307' - 'po'\n",
      "---> Processing Prefix 'MIR:00000308' - 'rnamods'\n",
      "---> Processing Prefix 'MIR:00000312' - 'treebase'\n",
      "---> Processing Prefix 'MIR:00000313' - 'tgd'\n",
      "---> Processing Prefix 'MIR:00000315' - 'tigrfam'\n",
      "---> Processing Prefix 'MIR:00000316' - 'atfdb.family'\n",
      "---> Processing Prefix 'MIR:00000317' - 'iuphar.family'\n",
      "---> Processing Prefix 'MIR:00000318' - 'dbg2introns'\n",
      "---> Processing Prefix 'MIR:00000319' - 'sdbs'\n",
      "---> Processing Prefix 'MIR:00000320' - 'vbase2'\n",
      "---> Processing Prefix 'MIR:00000321' - 'spike.map'\n",
      "---> Processing Prefix 'MIR:00000322' - 'metlin'\n",
      "---> Processing Prefix 'MIR:00000323' - 'genecards'\n",
      "---> Processing Prefix 'MIR:00000324' - 'mmrrc'\n",
      "---> Processing Prefix 'MIR:00000325' - 'umbbd.reaction'\n",
      "---> Processing Prefix 'MIR:00000326' - 'umbbd.enzyme'\n",
      "---> Processing Prefix 'MIR:00000327' - 'umbbd.pathway'\n",
      "---> Processing Prefix 'MIR:00000328' - 'umbbd.rule'\n",
      "---> Processing Prefix 'MIR:00000329' - 'mirex'\n",
      "---> Processing Prefix 'MIR:00000330' - 'dictybase.est'\n",
      "---> Processing Prefix 'MIR:00000331' - 'imgt.hla'\n",
      "---> Processing Prefix 'MIR:00000332' - 'flystock'\n",
      "---> Processing Prefix 'MIR:00000333' - 'opm'\n",
      "---> Processing Prefix 'MIR:00000334' - 'allergome'\n",
      "---> Processing Prefix 'MIR:00000335' - 'pombase'\n",
      "---> Processing Prefix 'MIR:00000336' - 'hpa'\n",
      "---> Processing Prefix 'MIR:00000337' - 'jaxmice'\n",
      "---> Processing Prefix 'MIR:00000338' - 'ubio.namebank'\n",
      "---> Processing Prefix 'MIR:00000339' - 'yetfasco'\n",
      "---> Processing Prefix 'MIR:00000340' - 'tarbase'\n",
      "---> Processing Prefix 'MIR:00000341' - 'charprot'\n",
      "---> Processing Prefix 'MIR:00000342' - 'oma.protein'\n",
      "---> Processing Prefix 'MIR:00000343' - 'oma.grp'\n",
      "---> Processing Prefix 'MIR:00000344' - 'ncbiprotein'\n",
      "---> Processing Prefix 'MIR:00000345' - 'genpept'\n",
      "---> Processing Prefix 'MIR:00000346' - 'unigene'\n",
      "---> Processing Prefix 'MIR:00000347' - 'bitterdb.rec'\n",
      "---> Processing Prefix 'MIR:00000348' - 'bitterdb.cpd'\n",
      "---> Processing Prefix 'MIR:00000349' - 'bioproject'\n",
      "---> Processing Prefix 'MIR:00000350' - 'biosample'\n",
      "---> Processing Prefix 'MIR:00000351' - 'piroplasma'\n",
      "---> Processing Prefix 'MIR:00000352' - 'unite'\n",
      "---> Processing Prefix 'MIR:00000353' - 'ncim'\n",
      "---> Processing Prefix 'MIR:00000354' - 'proglyc'\n",
      "---> Processing Prefix 'MIR:00000355' - 'polbase'\n",
      "---> Processing Prefix 'MIR:00000356' - 'nuclearbd'\n",
      "---> Processing Prefix 'MIR:00000357' - 'supfam'\n",
      "---> Processing Prefix 'MIR:00000358' - 'ricegap'\n",
      "---> Processing Prefix 'MIR:00000359' - 'pina'\n",
      "---> Processing Prefix 'MIR:00000360' - 'tissuelist'\n",
      "---> Processing Prefix 'MIR:00000361' - 'bacmap.biog'\n",
      "---> Processing Prefix 'MIR:00000362' - 'hgnc.symbol'\n",
      "---> Processing Prefix 'MIR:00000363' - 'panther.pathway'\n",
      "---> Processing Prefix 'MIR:00000364' - 'fairsharing'\n",
      "---> Processing Prefix 'MIR:00000365' - 'fungidb'\n",
      "---> Processing Prefix 'MIR:00000366' - 'darc'\n",
      "---> Processing Prefix 'MIR:00000367' - 'drsc'\n",
      "---> Processing Prefix 'MIR:00000368' - 'oridb.schizo'\n",
      "---> Processing Prefix 'MIR:00000369' - 'oridb.sacch'\n",
      "---> Processing Prefix 'MIR:00000370' - 'pscdb'\n",
      "---> Processing Prefix 'MIR:00000371' - 'scop'\n",
      "---> Processing Prefix 'MIR:00000372' - 'ena.embl'\n",
      "---> Processing Prefix 'MIR:00000373' - 'dommino'\n",
      "---> Processing Prefix 'MIR:00000374' - 'panther.node'\n",
      "---> Processing Prefix 'MIR:00000375' - 'ccds'\n",
      "---> Processing Prefix 'MIR:00000376' - 'lrg'\n",
      "---> Processing Prefix 'MIR:00000377' - 'hprd'\n",
      "---> Processing Prefix 'MIR:00000378' - 'gxa.gene'\n",
      "---> Processing Prefix 'MIR:00000379' - 'gxa.expt'\n",
      "---> Processing Prefix 'MIR:00000380' - 'metabolights'\n",
      "---> Processing Prefix 'MIR:00000381' - 'nbn'\n",
      "---> Processing Prefix 'MIR:00000382' - 'orcid'\n",
      "---> Processing Prefix 'MIR:00000383' - 'inchi'\n",
      "---> Processing Prefix 'MIR:00000384' - 'wikipedia.en'\n",
      "---> Processing Prefix 'MIR:00000385' - 'phosphopoint.kinase'\n",
      "---> Processing Prefix 'MIR:00000386' - 'phosphopoint.protein'\n",
      "---> Processing Prefix 'MIR:00000387' - 'inchikey'\n",
      "---> Processing Prefix 'MIR:00000388' - 'uniprot.isoform'\n",
      "---> Processing Prefix 'MIR:00000389' - 'kegg.environ'\n",
      "---> Processing Prefix 'MIR:00000390' - 'cldb'\n",
      "---> Processing Prefix 'MIR:00000391' - 'efo'\n",
      "---> Processing Prefix 'MIR:00000392' - 'hgmd'\n",
      "---> Processing Prefix 'MIR:00000393' - 'aphidbase.transcript'\n",
      "---> Processing Prefix 'MIR:00000394' - 'affy.probeset'\n",
      "---> Processing Prefix 'MIR:00000395' - 'treefam'\n",
      "---> Processing Prefix 'MIR:00000396' - 'caps'\n",
      "---> Processing Prefix 'MIR:00000397' - 'cubedb'\n",
      "---> Processing Prefix 'MIR:00000398' - 'ideal'\n",
      "---> Processing Prefix 'MIR:00000399' - 'stap'\n",
      "---> Processing Prefix 'MIR:00000400' - 'pocketome'\n",
      "---> Processing Prefix 'MIR:00000401' - 'gold.genome'\n",
      "---> Processing Prefix 'MIR:00000402' - 'gold.meta'\n",
      "---> Processing Prefix 'MIR:00000403' - 'bugbase.protocol'\n",
      "---> Processing Prefix 'MIR:00000404' - 'bugbase.expt'\n",
      "---> Processing Prefix 'MIR:00000405' - 'tol'\n",
      "---> Processing Prefix 'MIR:00000406' - 'vario'\n",
      "---> Processing Prefix 'MIR:00000407' - 'vipr'\n",
      "---> Processing Prefix 'MIR:00000408' - 'epd'\n",
      "---> Processing Prefix 'MIR:00000409' - 'rfam'\n",
      "---> Processing Prefix 'MIR:00000410' - 'fbol'\n",
      "---> Processing Prefix 'MIR:00000411' - 'aftol.taxonomy'\n",
      "---> Processing Prefix 'MIR:00000412' - 'aspgd.locus'\n",
      "---> Processing Prefix 'MIR:00000413' - 'aspgd.protein'\n",
      "---> Processing Prefix 'MIR:00000415' - 'autdb'\n",
      "---> Processing Prefix 'MIR:00000416' - 'bacmap.map'\n",
      "---> Processing Prefix 'MIR:00000417' - 'bgee.family'\n",
      "---> Processing Prefix 'MIR:00000418' - 'bgee.gene'\n",
      "---> Processing Prefix 'MIR:00000419' - 'bgee.stage'\n",
      "---> Processing Prefix 'MIR:00000420' - 'bgee.organ'\n",
      "---> Processing Prefix 'MIR:00000421' - 'biocarta.pathway'\n",
      "---> Processing Prefix 'MIR:00000422' - 'panther.pthcmp'\n",
      "---> Processing Prefix 'MIR:00000423' - 'gmd.profile'\n",
      "---> Processing Prefix 'MIR:00000424' - 'gmd.gcms'\n",
      "---> Processing Prefix 'MIR:00000425' - 'gmd.ref'\n",
      "---> Processing Prefix 'MIR:00000426' - 'gmd.analyte'\n",
      "---> Processing Prefix 'MIR:00000427' - 'intact.molecule'\n",
      "---> Processing Prefix 'MIR:00000428' - 'depod'\n",
      "---> Processing Prefix 'MIR:00000429' - 'cst'\n",
      "---> Processing Prefix 'MIR:00000430' - 'cst.ab'\n",
      "---> Processing Prefix 'MIR:00000431' - 'ndc'\n",
      "---> Processing Prefix 'MIR:00000432' - 'phytozome.locus'\n",
      "---> Processing Prefix 'MIR:00000433' - 'subtilist'\n",
      "---> Processing Prefix 'MIR:00000434' - 'dailymed'\n",
      "---> Processing Prefix 'MIR:00000435' - 'sider.drug'\n",
      "---> Processing Prefix 'MIR:00000436' - 'sider.effect'\n",
      "---> Processing Prefix 'MIR:00000437' - 'wikigenes'\n",
      "---> Processing Prefix 'MIR:00000438' - 'broad'\n",
      "---> Processing Prefix 'MIR:00000439' - 'coriell'\n",
      "---> Processing Prefix 'MIR:00000440' - 'corum'\n",
      "---> Processing Prefix 'MIR:00000442' - 'ecoliwiki'\n",
      "---> Processing Prefix 'MIR:00000443' - 'genprop'\n",
      "---> Processing Prefix 'MIR:00000444' - 'jstor'\n",
      "---> Processing Prefix 'MIR:00000445' - 'ma'\n",
      "---> Processing Prefix 'MIR:00000446' - 'uberon'\n",
      "---> Processing Prefix 'MIR:00000447' - 'unimod'\n",
      "---> Processing Prefix 'MIR:00000448' - 'vbrc'\n",
      "---> Processing Prefix 'MIR:00000449' - 'viralzone'\n",
      "---> Processing Prefix 'MIR:00000450' - 'go.ref'\n",
      "---> Processing Prefix 'MIR:00000451' - 'rgd.qtl'\n",
      "---> Processing Prefix 'MIR:00000452' - 'rgd.strain'\n",
      "---> Processing Prefix 'MIR:00000453' - 'door'\n",
      "---> Processing Prefix 'MIR:00000454' - 'degradome'\n",
      "---> Processing Prefix 'MIR:00000455' - 'dbd'\n",
      "---> Processing Prefix 'MIR:00000456' - 'datf'\n",
      "---> Processing Prefix 'MIR:00000457' - 'iuphar.ligand'\n",
      "---> Processing Prefix 'MIR:00000458' - 'molbase'\n",
      "---> Processing Prefix 'MIR:00000459' - 'yrcpdr'\n",
      "---> Processing Prefix 'MIR:00000460' - 'yid'\n",
      "---> Processing Prefix 'MIR:00000461' - 'funcbase.fly'\n",
      "---> Processing Prefix 'MIR:00000462' - 'funcbase.human'\n",
      "---> Processing Prefix 'MIR:00000463' - 'funcbase.mouse'\n",
      "---> Processing Prefix 'MIR:00000464' - 'funcbase.yeast'\n",
      "---> Processing Prefix 'MIR:00000465' - 'ydpm'\n",
      "---> Processing Prefix 'MIR:00000466' - 'wb.rnai'\n",
      "---> Processing Prefix 'MIR:00000467' - 'funcat'\n",
      "---> Processing Prefix 'MIR:00000468' - 'pass2'\n",
      "---> Processing Prefix 'MIR:00000469' - 'iceberg.element'\n",
      "---> Processing Prefix 'MIR:00000470' - 'iceberg.family'\n",
      "---> Processing Prefix 'MIR:00000471' - 'vfdb.genus'\n",
      "---> Processing Prefix 'MIR:00000472' - 'vfdb.gene'\n",
      "---> Processing Prefix 'MIR:00000473' - 'mesh.2013'\n",
      "---> Processing Prefix 'MIR:00000474' - 'kegg.module'\n",
      "---> Processing Prefix 'MIR:00000475' - 'kegg.disease'\n",
      "---> Processing Prefix 'MIR:00000476' - 'medlineplus'\n",
      "---> Processing Prefix 'MIR:00000477' - 'ligandbox'\n",
      "---> Processing Prefix 'MIR:00000478' - 'glycoepitope'\n",
      "---> Processing Prefix 'MIR:00000479' - 'jcggdb'\n",
      "---> Processing Prefix 'MIR:00000480' - 'noncodev4.gene'\n",
      "---> Processing Prefix 'MIR:00000481' - 'noncodev4.rna'\n",
      "---> Processing Prefix 'MIR:00000482' - 'oryzabase.gene'\n",
      "---> Processing Prefix 'MIR:00000483' - 'oryzabase.mutant'\n",
      "---> Processing Prefix 'MIR:00000484' - 'oryzabase.strain'\n",
      "---> Processing Prefix 'MIR:00000485' - 'oryzabase.stage'\n",
      "---> Processing Prefix 'MIR:00000486' - 'otl'\n",
      "---> Processing Prefix 'MIR:00000487' - 'genewiki'\n",
      "---> Processing Prefix 'MIR:00000488' - 'paxdb.organism'\n",
      "---> Processing Prefix 'MIR:00000489' - 'paxdb.protein'\n",
      "---> Processing Prefix 'MIR:00000490' - 'pdb.ligand'\n",
      "---> Processing Prefix 'MIR:00000491' - 'merops.inhibitor'\n",
      "---> Processing Prefix 'MIR:00000492' - 'adw'\n",
      "---> Processing Prefix 'MIR:00000493' - 'glida.gpcr'\n",
      "---> Processing Prefix 'MIR:00000494' - 'glida.ligand'\n",
      "---> Processing Prefix 'MIR:00000495' - 'grsdb'\n",
      "---> Processing Prefix 'MIR:00000496' - 'fsnp'\n",
      "---> Processing Prefix 'MIR:00000497' - 'hdr'\n",
      "---> Processing Prefix 'MIR:00000498' - 'norine'\n",
      "---> Processing Prefix 'MIR:00000499' - 'ordb'\n",
      "---> Processing Prefix 'MIR:00000500' - 'odor'\n",
      "---> Processing Prefix 'MIR:00000501' - 'p3db.protein'\n",
      "---> Processing Prefix 'MIR:00000502' - 'p3db.site'\n",
      "---> Processing Prefix 'MIR:00000503' - 'topdb'\n",
      "---> Processing Prefix 'MIR:00000504' - 'cattleqtldb'\n",
      "---> Processing Prefix 'MIR:00000505' - 'chickenqtldb'\n",
      "---> Processing Prefix 'MIR:00000506' - 'pigqtldb'\n",
      "---> Processing Prefix 'MIR:00000507' - 'sheepqtldb'\n",
      "---> Processing Prefix 'MIR:00000508' - 'gramene.growthstage'\n",
      "---> Processing Prefix 'MIR:00000510' - 'mgnify.samp'\n",
      "---> Processing Prefix 'MIR:00000511' - 'ega.study'\n",
      "---> Processing Prefix 'MIR:00000512' - 'ega.dataset'\n",
      "---> Processing Prefix 'MIR:00000513' - 'px'\n",
      "---> Processing Prefix 'MIR:00000514' - 'biomodels.vocabulary'\n",
      "---> Processing Prefix 'MIR:00000515' - 'pride.project'\n",
      "---> Processing Prefix 'MIR:00000516' - 'antibodyregistry'\n",
      "---> Processing Prefix 'MIR:00000517' - 'mamo'\n",
      "---> Processing Prefix 'MIR:00000518' - 'eo'\n",
      "---> Processing Prefix 'MIR:00000519' - 'idot'\n",
      "---> Processing Prefix 'MIR:00000520' - 'hgnc.family'\n",
      "---> Processing Prefix 'MIR:00000521' - 'yeastintron'\n",
      "---> Processing Prefix 'MIR:00000522' - 'ardb'\n",
      "---> Processing Prefix 'MIR:00000524' - 'proteomicsdb.protein'\n",
      "---> Processing Prefix 'MIR:00000525' - 'proteomicsdb.peptide'\n",
      "---> Processing Prefix 'MIR:00000526' - 'hpm.protein'\n",
      "---> Processing Prefix 'MIR:00000527' - 'hpm.peptide'\n",
      "---> Processing Prefix 'MIR:00000528' - 'drugbankv4.target'\n",
      "---> Processing Prefix 'MIR:00000529' - 'zinc'\n",
      "---> Processing Prefix 'MIR:00000530' - 'foodb.compound'\n",
      "---> Processing Prefix 'MIR:00000531' - 'unii'\n",
      "---> Processing Prefix 'MIR:00000532' - 'orphanet.ordo'\n",
      "---> Processing Prefix 'MIR:00000533' - 'psipar'\n",
      "---> Processing Prefix 'MIR:00000534' - 'clinvar.record'\n",
      "---> Processing Prefix 'MIR:00000535' - 'mgnify.proj'\n",
      "---> Processing Prefix 'MIR:00000536' - 'euclinicaltrials'\n",
      "---> Processing Prefix 'MIR:00000537' - 'google.patent'\n",
      "---> Processing Prefix 'MIR:00000538' - 'uspto'\n",
      "---> Processing Prefix 'MIR:00000539' - 'cpc'\n",
      "---> Processing Prefix 'MIR:00000540' - 'gwascentral.study'\n",
      "---> Processing Prefix 'MIR:00000541' - 'exac.variant'\n",
      "---> Processing Prefix 'MIR:00000542' - 'gwascentral.marker'\n",
      "---> Processing Prefix 'MIR:00000543' - 'gwascentral.phenotype'\n",
      "---> Processing Prefix 'MIR:00000544' - 'lincs.cell'\n",
      "---> Processing Prefix 'MIR:00000545' - 'lincs.protein'\n",
      "---> Processing Prefix 'MIR:00000546' - 'lincs.smallmolecule'\n",
      "---> Processing Prefix 'MIR:00000547' - 'exac.transcript'\n",
      "---> Processing Prefix 'MIR:00000548' - 'exac.gene'\n",
      "---> Processing Prefix 'MIR:00000549' - 'wikidata'\n",
      "---> Processing Prefix 'MIR:00000550' - 'swisslipid'\n",
      "---> Processing Prefix 'MIR:00000551' - 'unipathway.compound'\n",
      "---> Processing Prefix 'MIR:00000552' - 'seed'\n",
      "---> Processing Prefix 'MIR:00000553' - 'seed.compound'\n",
      "---> Processing Prefix 'MIR:00000554' - 'bigg.model'\n",
      "---> Processing Prefix 'MIR:00000555' - 'bigg.compartment'\n",
      "---> Processing Prefix 'MIR:00000556' - 'bigg.metabolite'\n",
      "---> Processing Prefix 'MIR:00000557' - 'bigg.reaction'\n",
      "---> Processing Prefix 'MIR:00000558' - 'rrid'\n",
      "---> Processing Prefix 'MIR:00000559' - 'umls'\n",
      "---> Processing Prefix 'MIR:00000560' - 'mesh'\n",
      "---> Processing Prefix 'MIR:00000561' - 'emdb'\n",
      "---> Processing Prefix 'MIR:00000562' - 'mirtarbase'\n",
      "---> Processing Prefix 'MIR:00000563' - 'meddra'\n",
      "---> Processing Prefix 'MIR:00000564' - 'dashr'\n",
      "---> Processing Prefix 'MIR:00000565' - 'dashr.expression'\n",
      "---> Processing Prefix 'MIR:00000566' - 'splash'\n",
      "---> Processing Prefix 'MIR:00000567' - 'metanetx.chemical'\n",
      "---> Processing Prefix 'MIR:00000568' - 'metanetx.reaction'\n",
      "---> Processing Prefix 'MIR:00000569' - 'metanetx.compartment'\n",
      "---> Processing Prefix 'MIR:00000570' - 'unipathway.reaction'\n",
      "---> Processing Prefix 'MIR:00000571' - 'hp'\n",
      "---> Processing Prefix 'MIR:00000572' - 'sasbdb'\n",
      "---> Processing Prefix 'MIR:00000573' - 'hgnc.genefamily'\n",
      "---> Processing Prefix 'MIR:00000574' - 'mdm'\n",
      "---> Processing Prefix 'MIR:00000575' - 'probonto'\n",
      "---> Processing Prefix 'MIR:00000576' - 'apid.interactions'\n",
      "---> Processing Prefix 'MIR:00000577' - 'storedb'\n",
      "---> Processing Prefix 'MIR:00000578' - 'kegg'\n",
      "---> Processing Prefix 'MIR:00000579' - 'planttfdb'\n",
      "---> Processing Prefix 'MIR:00000580' - 'mp'\n",
      "---> Processing Prefix 'MIR:00000581' - 'worms'\n",
      "---> Processing Prefix 'MIR:00000582' - 'multicellds.collection'\n",
      "---> Processing Prefix 'MIR:00000583' - 'multicellds.snapshot'\n",
      "---> Processing Prefix 'MIR:00000584' - 'multicellds.cell_line'\n",
      "---> Processing Prefix 'MIR:00000585' - 'ecyano.entity'\n",
      "---> Processing Prefix 'MIR:00000586' - 'ecyano.model'\n",
      "---> Processing Prefix 'MIR:00000587' - 'ecyano.rule'\n",
      "---> Processing Prefix 'MIR:00000588' - 'lincs.data'\n",
      "---> Processing Prefix 'MIR:00000589' - 'agricola'\n",
      "---> Processing Prefix 'MIR:00000590' - 'ms'\n",
      "---> Processing Prefix 'MIR:00000591' - 'envo'\n",
      "---> Processing Prefix 'MIR:00000592' - 'ark'\n",
      "---> Processing Prefix 'MIR:00000593' - 'grid'\n",
      "---> Processing Prefix 'MIR:00000594' - 'medgen'\n",
      "---> Processing Prefix 'MIR:00000595' - 'clinvar.submission'\n",
      "---> Processing Prefix 'MIR:00000596' - 'clinvar'\n",
      "---> Processing Prefix 'MIR:00000597' - 'bao'\n",
      "---> Processing Prefix 'MIR:00000598' - 'cellosaurus'\n",
      "---> Processing Prefix 'MIR:00000599' - 'mir'\n",
      "---> Processing Prefix 'MIR:00000600' - 'pgx'\n",
      "---> Processing Prefix 'MIR:00000601' - 'cosmic'\n",
      "---> Processing Prefix 'MIR:00000602' - 'dbgap'\n",
      "---> Processing Prefix 'MIR:00000603' - 'iao'\n",
      "---> Processing Prefix 'MIR:00000604' - 'gdc'\n",
      "---> Processing Prefix 'MIR:00000605' - 'omit'\n",
      "---> Processing Prefix 'MIR:00000606' - 'insdc.cds'\n",
      "---> Processing Prefix 'MIR:00000607' - 'insdc.gca'\n",
      "---> Processing Prefix 'MIR:00000608' - 'glytoucan'\n",
      "---> Processing Prefix 'MIR:00000609' - 'metacyc.compound'\n",
      "---> Processing Prefix 'MIR:00000610' - 'metacyc.reaction'\n",
      "---> Processing Prefix 'MIR:00000611' - 'd1id'\n",
      "---> Processing Prefix 'MIR:00000612' - 'cath'\n",
      "---> Processing Prefix 'MIR:00000613' - 'mmp.ref'\n",
      "---> Processing Prefix 'MIR:00000614' - 'mmp.db'\n",
      "---> Processing Prefix 'MIR:00000615' - 'mmp.cat'\n",
      "---> Processing Prefix 'MIR:00000616' - 'biotools'\n",
      "---> Processing Prefix 'MIR:00000617' - 'aop'\n",
      "---> Processing Prefix 'MIR:00000618' - 'ngl'\n",
      "---> Processing Prefix 'MIR:00000619' - 'erv'\n",
      "---> Processing Prefix 'MIR:00000620' - 'napdi'\n",
      "---> Processing Prefix 'MIR:00000621' - 'arraymap'\n",
      "---> Processing Prefix 'MIR:00000622' - 'begdb'\n",
      "---> Processing Prefix 'MIR:00000623' - 'cameo'\n",
      "---> Processing Prefix 'MIR:00000624' - 'sisu'\n",
      "---> Processing Prefix 'MIR:00000625' - 'mzspec'\n",
      "---> Processing Prefix 'MIR:00000626' - 'crisprdb'\n",
      "---> Processing Prefix 'MIR:00000627' - 'gnpis'\n",
      "---> Processing Prefix 'MIR:00000628' - 'licebase'\n",
      "---> Processing Prefix 'MIR:00000629' - 'sugarbind'\n",
      "---> Processing Prefix 'MIR:00000630' - 'rnacentral'\n",
      "---> Processing Prefix 'MIR:00000631' - 'microscope'\n",
      "---> Processing Prefix 'MIR:00000632' - 'swissregulon'\n",
      "---> Processing Prefix 'MIR:00000633' - 'validatordb'\n",
      "---> Processing Prefix 'MIR:00000634' - 'biominder'\n",
      "---> Processing Prefix 'MIR:00000635' - 'neurovault.collection'\n",
      "---> Processing Prefix 'MIR:00000636' - 'vmhmetabolite'\n",
      "---> Processing Prefix 'MIR:00000637' - 'mobidb'\n",
      "---> Processing Prefix 'MIR:00000638' - 'neurovault.image'\n",
      "---> Processing Prefix 'MIR:00000639' - 'nmr'\n",
      "---> Processing Prefix 'MIR:00000640' - 'vmhreaction'\n",
      "---> Processing Prefix 'MIR:00000641' - 'ascl'\n",
      "---> Processing Prefix 'MIR:00000642' - 'facebase'\n",
      "---> Processing Prefix 'MIR:00000643' - 'gudmap'\n",
      "---> Processing Prefix 'MIR:00000644' - 'eu89h'\n",
      "---> Processing Prefix 'MIR:00000645' - 'minid'\n",
      "---> Processing Prefix 'MIR:00000646' - 'mw.study'\n",
      "---> Processing Prefix 'MIR:00000647' - 'mw.project'\n",
      "---> Processing Prefix 'MIR:00000648' - 'mex'\n",
      "---> Processing Prefix 'MIR:00000649' - 'gpmdb'\n",
      "---> Processing Prefix 'MIR:00000650' - 'massive'\n",
      "---> Processing Prefix 'MIR:00000651' - 'fplx'\n",
      "---> Processing Prefix 'MIR:00000652' - 'aop.events'\n",
      "---> Processing Prefix 'MIR:00000653' - 'aop.relationships'\n",
      "---> Processing Prefix 'MIR:00000654' - 'aop.stressor'\n",
      "---> Processing Prefix 'MIR:00000655' - 'swh'\n",
      "---> Processing Prefix 'MIR:00000656' - 'dev.ga4ghdos'\n",
      "---> Processing Prefix 'MIR:00000657' - 'complexportal'\n",
      "---> Processing Prefix 'MIR:00000658' - 'comptox'\n",
      "---> Processing Prefix 'MIR:00000659' - 'rbk'\n",
      "---> Processing Prefix 'MIR:00000660' - 'oci'\n",
      "---> Processing Prefix 'MIR:00000661' - 'foodon'\n",
      "---> Processing Prefix 'MIR:00000662' - 'encode'\n",
      "---> Processing Prefix 'MIR:00000663' - 'gtex'\n",
      "---> Processing Prefix 'MIR:00000664' - 'ricenetdb.gene'\n",
      "---> Processing Prefix 'MIR:00000665' - 'ricenetdb.protein'\n",
      "---> Processing Prefix 'MIR:00000666' - 'ricenetdb.compound'\n",
      "---> Processing Prefix 'MIR:00000667' - 'ricenetdb.reaction'\n",
      "---> Processing Prefix 'MIR:00000668' - 'ricenetdb.miRNA'\n",
      "---> Processing Prefix 'MIR:00000669' - 'oclc'\n",
      "---> Processing Prefix 'MIR:00000670' - 'asin'\n",
      "---> Processing Prefix 'MIR:00000671' - 'ga4ghdos'\n",
      "---> Processing Prefix 'MIR:00000672' - 'flowrepository'\n",
      "---> Processing Prefix 'MIR:00000673' - 'oid'\n",
      "---> Processing Prefix 'MIR:00000674' - 'hpscreg'\n",
      "---> Processing Prefix 'MIR:00000675' - 'addgene'\n",
      "---> Processing Prefix 'MIR:00000676' - 'bacdive'\n",
      "---> Processing Prefix 'MIR:00000677' - 'ido'\n",
      "---> Processing Prefix 'MIR:00000678' - 'gdsc'\n",
      "---> Processing Prefix 'MIR:00000679' - 'cbioportal'\n"
     ]
    }
   ],
   "source": [
    "# Process PID entries\n",
    "for pid_entry in resolution_dataset_source:\n",
    "    print(\"---> Processing Prefix '{}' - '{}'\".format(pid_entry['id'], pid_entry['prefix']))\n",
    "    # Translate PID Entry to Namespace model\n",
    "    namespace = hqregistry.Namespace()\n",
    "    namespace.prefix = pid_entry.get('prefix', None)\n",
    "    namespace.mirId = pid_entry.get('id', None)\n",
    "    namespace.name = pid_entry.get('name', None)\n",
    "    namespace.pattern = pid_entry.get('pattern', None)\n",
    "    namespace.description = pid_entry.get('definition', None)\n",
    "    # TODO creation and modification dates are missing from the dataset\n",
    "    # There is no deprecation date for now\n",
    "    # Flag all namespaces as 'NOT Deprecated' for now\n",
    "    namespace.deprecated = False"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
